diff -ur fontconfig-2.15.0.orig/Makefile.in fontconfig-2.15.0/Makefile.in
--- fontconfig-2.15.0.orig/Makefile.in	2023-12-22 14:55:23.000000000 +0100
+++ fontconfig-2.15.0/Makefile.in	2024-03-27 01:08:22.944364659 +0100
@@ -466,9 +466,7 @@
 top_build_prefix = @top_build_prefix@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
-SUBDIRS = fontconfig fc-case fc-lang src fc-cache fc-cat fc-conflist \
-	fc-list fc-match fc-pattern fc-query fc-scan fc-validate \
-	conf.d its po po-conf test $(am__append_1)
+SUBDIRS = fontconfig src
 ACLOCAL_AMFLAGS = -I m4
 MESON_FILES = \
 	conf.d/link_confs.py \
@@ -983,8 +981,7 @@
 
 info-am:
 
-install-data-am: install-data-local install-pkgconfigDATA \
-	install-xmlDATA
+install-data-am: install-pkgconfigDATA
 
 install-dvi: install-dvi-recursive
 
