diff -ur fontconfig-2.13.0.orig/Makefile.in fontconfig-2.13.0/Makefile.in
--- fontconfig-2.13.0.orig/Makefile.in	2018-03-06 04:31:57.000000000 +0100
+++ fontconfig-2.13.0/Makefile.in	2018-03-07 04:26:54.338869741 +0100
@@ -452,9 +452,7 @@
 top_build_prefix = @top_build_prefix@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
-SUBDIRS = fontconfig fc-case fc-lang src fc-cache fc-cat fc-conflist \
-	fc-list fc-match fc-pattern fc-query fc-scan fc-validate \
-	conf.d its po po-conf test $(am__append_1)
+SUBDIRS = fontconfig src
 ACLOCAL_AMFLAGS = -I m4
 EXTRA_DIST = config.rpath fontconfig.pc.in fonts.conf.in fonts.dtd \
 	fontconfig.spec.in fontconfig.spec fontconfig-zip.in \
@@ -945,8 +943,7 @@
 
 info-am:
 
-install-data-am: install-data-local install-pkgconfigDATA \
-	install-xmlDATA
+install-data-am: install-pkgconfigDATA
 
 install-dvi: install-dvi-recursive
 
