diff -ur fontconfig-2.13.1.orig/Makefile.in fontconfig-2.13.1/Makefile.in
--- fontconfig-2.13.1.orig/Makefile.in	2018-08-30 10:24:27.000000000 +0200
+++ fontconfig-2.13.1/Makefile.in	2018-09-03 03:16:25.811213555 +0200
@@ -454,9 +454,7 @@
 top_build_prefix = @top_build_prefix@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
-SUBDIRS = fontconfig fc-case fc-lang src fc-cache fc-cat fc-conflist \
-	fc-list fc-match fc-pattern fc-query fc-scan fc-validate \
-	conf.d its po po-conf test $(am__append_1)
+SUBDIRS = fontconfig src
 ACLOCAL_AMFLAGS = -I m4
 EXTRA_DIST = config.rpath fontconfig.pc.in fonts.conf.in fonts.dtd \
 	fontconfig.spec.in fontconfig.spec fontconfig-zip.in \
@@ -950,8 +948,7 @@
 
 info-am:
 
-install-data-am: install-data-local install-pkgconfigDATA \
-	install-xmlDATA
+install-data-am: install-pkgconfigDATA
 
 install-dvi: install-dvi-recursive
 
