diff -ur fontconfig-2.12.1.orig/Makefile.in fontconfig-2.12.1/Makefile.in
--- fontconfig-2.12.1.orig/Makefile.in	2016-08-05 07:49:13.000000000 +0200
+++ fontconfig-2.12.1/Makefile.in	2016-08-06 02:58:47.773143873 +0200
@@ -434,9 +434,7 @@
 top_build_prefix = @top_build_prefix@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
-SUBDIRS = fontconfig fc-blanks fc-case fc-lang fc-glyphname src \
-	fc-cache fc-cat fc-list fc-match fc-pattern fc-query fc-scan \
-	fc-validate conf.d test $(am__append_1)
+SUBDIRS = fontconfig src
 ACLOCAL_AMFLAGS = -I m4
 EXTRA_DIST = fontconfig.pc.in fonts.conf.in fonts.dtd \
 	fontconfig.spec.in fontconfig.spec fontconfig-zip.in \
@@ -927,8 +925,7 @@
 
 info-am:
 
-install-data-am: install-data-local install-pkgconfigDATA \
-	install-xmlDATA
+install-data-am: install-pkgconfigDATA
 
 install-dvi: install-dvi-recursive
 
