diff -ur fontconfig-2.10.2.orig/Makefile.in fontconfig-2.10.2/Makefile.in
--- fontconfig-2.10.2.orig/Makefile.in	2012-11-26 09:22:02.000000000 +0100
+++ fontconfig-2.10.2/Makefile.in	2013-02-18 06:17:31.929378994 +0100
@@ -342,9 +342,7 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 xmldir = $(XMLDIR)
-SUBDIRS = fontconfig fc-case fc-lang fc-glyphname src \
-	fc-cache fc-cat fc-list fc-match fc-pattern fc-query fc-scan \
-	conf.d $(DOCSRC) test
+SUBDIRS = fontconfig src
 
 AUTOMAKE_OPTIONS = dist-bzip2
 ACLOCAL_AMFLAGS = -I m4
@@ -862,8 +860,7 @@
 
 info-am:
 
-install-data-am: install-data-local install-pkgconfigDATA \
-	install-xmlDATA
+install-data-am: install-pkgconfigDATA
 
 install-dvi: install-dvi-recursive
 
