--- gnutls-3.3.17.orig/Makefile.in	2015-08-09 21:26:50.000000000 +0200
+++ gnutls-3.3.17/Makefile.in	2015-08-22 03:15:55.077196943 +0200
@@ -1310,7 +1310,7 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 DISTCHECK_CONFIGURE_FLAGS = --enable-gtk-doc --disable-valgrind-tests
-SUBDIRS = gl lib extra $(am__append_1) po src $(am__append_2) \
+SUBDIRS = gl lib extra $(am__append_1) po $(am__append_2) \
 	$(am__append_3) $(am__append_4)
 ACLOCAL_AMFLAGS = -I m4 -I gl/m4 -I src/libopts/m4 -I src/gl/m4
 EXTRA_DIST = cfg.mk maint.mk .clcopying
