Only in gnutls-3.6.7.orig/doc/doxygen: Doxyfile.orig
diff -ur gnutls-3.6.7.orig/lib/record.c gnutls-3.6.7/lib/record.c
--- gnutls-3.6.7.orig/lib/record.c	2019-03-15 10:10:27.000000000 +0100
+++ gnutls-3.6.7/lib/record.c	2019-04-21 17:15:13.544253607 +0200
@@ -146,9 +146,9 @@
 			  int recv_fd, int send_fd)
 {
 	session->internals.transport_send_ptr =
-	    (gnutls_transport_ptr_t) (long) send_fd;
+	    (gnutls_transport_ptr_t) (intptr_t) send_fd;
 	session->internals.transport_recv_ptr =
-	    (gnutls_transport_ptr_t) (long) recv_fd;
+	    (gnutls_transport_ptr_t) (intptr_t) recv_fd;
 }
 
 #if 0
@@ -229,8 +229,8 @@
 			  int *recv_int, int *send_int)
 {
 
-	*recv_int = (long) session->internals.transport_recv_ptr;
-	*send_int = (long) session->internals.transport_send_ptr;
+	*recv_int = (intptr_t) session->internals.transport_recv_ptr;
+	*send_int = (intptr_t) session->internals.transport_send_ptr;
 }
 
 /**
@@ -247,7 +247,7 @@
  **/
 int gnutls_transport_get_int(gnutls_session_t session)
 {
-	return (long) session->internals.transport_recv_ptr;
+	return (intptr_t) session->internals.transport_recv_ptr;
 }
 
 /**
diff -ur gnutls-3.6.7.orig/lib/system/fastopen.c gnutls-3.6.7/lib/system/fastopen.c
--- gnutls-3.6.7.orig/lib/system/fastopen.c	2019-03-15 10:10:27.000000000 +0100
+++ gnutls-3.6.7/lib/system/fastopen.c	2019-04-21 17:15:13.544253607 +0200
@@ -159,7 +159,7 @@
 {
 	tfo_st *p = ptr;
 
-	return gnutls_system_recv_timeout((gnutls_transport_ptr_t)(long)p->fd, ms);
+	return gnutls_system_recv_timeout((gnutls_transport_ptr_t)(intptr_t)p->fd, ms);
 }
 
 static ssize_t
