diff -ur gnutls-3.5.17.orig/lib/record.c gnutls-3.5.17/lib/record.c
--- gnutls-3.5.17.orig/lib/record.c	2017-12-31 13:37:48.000000000 +0100
+++ gnutls-3.5.17/lib/record.c	2018-02-04 04:00:25.949534814 +0100
@@ -144,9 +144,9 @@
 			  int recv_fd, int send_fd)
 {
 	session->internals.transport_send_ptr =
-	    (gnutls_transport_ptr_t) (long) send_fd;
+	    (gnutls_transport_ptr_t) (intptr_t) send_fd;
 	session->internals.transport_recv_ptr =
-	    (gnutls_transport_ptr_t) (long) recv_fd;
+	    (gnutls_transport_ptr_t) (intptr_t) recv_fd;
 }
 
 #if 0
@@ -227,8 +227,8 @@
 			  int *recv_int, int *send_int)
 {
 
-	*recv_int = (long) session->internals.transport_recv_ptr;
-	*send_int = (long) session->internals.transport_send_ptr;
+	*recv_int = (intptr_t) session->internals.transport_recv_ptr;
+	*send_int = (intptr_t) session->internals.transport_send_ptr;
 }
 
 /**
@@ -245,7 +245,7 @@
  **/
 int gnutls_transport_get_int(gnutls_session_t session)
 {
-	return (long) session->internals.transport_recv_ptr;
+	return (intptr_t) session->internals.transport_recv_ptr;
 }
 
 /**
diff -ur gnutls-3.5.17.orig/lib/system/fastopen.c gnutls-3.5.17/lib/system/fastopen.c
--- gnutls-3.5.17.orig/lib/system/fastopen.c	2017-12-31 13:37:48.000000000 +0100
+++ gnutls-3.5.17/lib/system/fastopen.c	2018-02-04 04:01:36.549031106 +0100
@@ -156,7 +156,7 @@
 {
 	tfo_st *p = ptr;
 
-	return gnutls_system_recv_timeout((gnutls_transport_ptr_t)(long)p->fd, ms);
+	return gnutls_system_recv_timeout((gnutls_transport_ptr_t)(intptr_t)p->fd, ms);
 }
 
 static ssize_t
