diff -ur enca-1.19.orig/Makefile.in enca-1.19/Makefile.in
--- enca-1.19.orig/Makefile.in	2016-09-05 16:19:29.000000000 +0200
+++ enca-1.19/Makefile.in	2018-02-04 05:15:01.374657094 +0100
@@ -381,7 +381,7 @@
 top_build_prefix = @top_build_prefix@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
-@MAINTAINER_MODE_FALSE@SUBDIRS = tools script lib src devel-docs test
+@MAINTAINER_MODE_FALSE@SUBDIRS = tools lib
 @MAINTAINER_MODE_TRUE@SUBDIRS = tools data script lib src devel-docs test
 man_MANS = man/enca.1
 DISTCHECK_CONFIGURE_FLAGS = --enable-maintainer-mode
@@ -908,9 +908,8 @@
 
 info-am:
 
-install-data-am: install-man install-pkgconfigDATA
+install-data-am: install-pkgconfigDATA
 	@$(NORMAL_INSTALL)
-	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
 install-dvi: install-dvi-recursive
 
 install-dvi-am:
