diff -ur enca-1.15.orig/Makefile.in enca-1.15/Makefile.in
--- enca-1.15.orig/Makefile.in	2013-09-30 09:48:22.000000000 +0200
+++ enca-1.15/Makefile.in	2013-10-07 02:21:36.564508559 +0200
@@ -368,7 +368,7 @@
 top_build_prefix = @top_build_prefix@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
-@MAINTAINER_MODE_FALSE@SUBDIRS = tools script lib src devel-docs test
+@MAINTAINER_MODE_FALSE@SUBDIRS = tools lib
 
 # @(#) $Id: Makefile.am,v 1.35 2005/11/24 10:21:32 yeti Exp $
 @MAINTAINER_MODE_TRUE@SUBDIRS = tools data script lib src devel-docs test
@@ -896,9 +896,8 @@
 
 info-am:
 
-install-data-am: install-man install-pkgconfigDATA
+install-data-am: install-pkgconfigDATA
 	@$(NORMAL_INSTALL)
-	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
 install-dvi: install-dvi-recursive
 
 install-dvi-am:
