diff -ur twolame-0.4.0.orig/Makefile.in twolame-0.4.0/Makefile.in
--- twolame-0.4.0.orig/Makefile.in	2019-10-11 18:26:49.000000000 +0200
+++ twolame-0.4.0/Makefile.in	2019-10-14 19:19:05.999665919 +0200
@@ -376,7 +376,7 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 ACLOCAL_AMFLAGS = -I build-scripts
-SUBDIRS = . libtwolame frontend simplefrontend doc tests
+SUBDIRS = . libtwolame
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = twolame.pc
 INDENT = indent -npro -kr -nsob -fca -blf -ip1 -hnl -l100 -lc100 -nut -ts4 -v
@@ -847,7 +847,7 @@
 
 info-am:
 
-install-data-am: install-pkgconfigDATA install-pkgdocDATA
+install-data-am: install-pkgconfigDATA
 
 install-dvi: install-dvi-recursive
 
