diff -ur twolame-0.4.0.orig/libtwolame/twolame.h twolame-0.4.0/libtwolame/twolame.h
--- twolame-0.4.0.orig/libtwolame/twolame.h	2019-10-11 18:26:25.000000000 +0200
+++ twolame-0.4.0/libtwolame/twolame.h	2019-10-14 19:20:25.303051104 +0200
@@ -37,6 +37,10 @@
  * the library, define LIBTWOLAME_STATIC prior to including this header.
  */
 
+#ifndef LIBTWOLAME_STATIC
+#define LIBTWOLAME_STATIC
+#endif
+
 #ifdef TL_API
 # undef TL_API
 #endif
