Only in SDL2-2.30.1: .configure.swp
diff -ur SDL2-2.30.1.orig/Makefile.in SDL2-2.30.1/Makefile.in
--- SDL2-2.30.1.orig/Makefile.in	2024-03-05 23:37:32.000000000 +0100
+++ SDL2-2.30.1/Makefile.in	2024-03-27 02:12:44.159075795 +0100
@@ -192,8 +192,6 @@
 	$(LIBTOOL) --mode=install $(INSTALL) $(objects)/$(SDLMAIN_TARGET) $(DESTDIR)$(libdir)/$(SDLMAIN_TARGET)
 	$(LIBTOOL) --mode=install $(INSTALL) $(objects)/$(SDLTEST_TARGET) $(DESTDIR)$(libdir)/$(SDLTEST_TARGET)
 install-data:
-	$(SHELL) $(auxdir)/mkinstalldirs $(DESTDIR)$(datadir)/aclocal
-	$(INSTALL) -m 644 $(srcdir)/sdl2.m4 $(DESTDIR)$(datadir)/aclocal/sdl2.m4
 	$(SHELL) $(auxdir)/mkinstalldirs $(DESTDIR)$(libdir)/pkgconfig
 	$(INSTALL) -m 644 sdl2.pc $(DESTDIR)$(libdir)/pkgconfig
 ifeq ($(INSTALL_SDL2_CONFIG),TRUE)
