diff -ur SDL2-2.0.18.orig/Makefile.in SDL2-2.0.18/Makefile.in
--- SDL2-2.0.18.orig/Makefile.in	2021-11-26 17:03:45.000000000 +0100
+++ SDL2-2.0.18/Makefile.in	2021-12-30 09:02:52.385302839 +0100
@@ -188,8 +188,6 @@
 	$(LIBTOOL) --mode=install $(INSTALL) $(objects)/$(SDLMAIN_TARGET) $(DESTDIR)$(libdir)/$(SDLMAIN_TARGET)
 	$(LIBTOOL) --mode=install $(INSTALL) $(objects)/$(SDLTEST_TARGET) $(DESTDIR)$(libdir)/$(SDLTEST_TARGET)
 install-data:
-	$(SHELL) $(auxdir)/mkinstalldirs $(DESTDIR)$(datadir)/aclocal
-	$(INSTALL) -m 644 $(srcdir)/sdl2.m4 $(DESTDIR)$(datadir)/aclocal/sdl2.m4
 	$(SHELL) $(auxdir)/mkinstalldirs $(DESTDIR)$(libdir)/pkgconfig
 	$(INSTALL) -m 644 sdl2.pc $(DESTDIR)$(libdir)/pkgconfig
 ifeq ($(INSTALL_SDL2_CONFIG),TRUE)
