diff -ur SDL2-2.0.16.orig/Makefile.in SDL2-2.0.16/Makefile.in
--- SDL2-2.0.16.orig/Makefile.in	2021-08-06 21:36:54.000000000 +0200
+++ SDL2-2.0.16/Makefile.in	2021-10-24 12:31:27.157549502 +0200
@@ -187,8 +187,6 @@
 	$(LIBTOOL) --mode=install $(INSTALL) $(objects)/$(SDLMAIN_TARGET) $(DESTDIR)$(libdir)/$(SDLMAIN_TARGET)
 	$(LIBTOOL) --mode=install $(INSTALL) $(objects)/$(SDLTEST_TARGET) $(DESTDIR)$(libdir)/$(SDLTEST_TARGET)
 install-data:
-	$(SHELL) $(auxdir)/mkinstalldirs $(DESTDIR)$(datadir)/aclocal
-	$(INSTALL) -m 644 $(srcdir)/sdl2.m4 $(DESTDIR)$(datadir)/aclocal/sdl2.m4
 	$(SHELL) $(auxdir)/mkinstalldirs $(DESTDIR)$(libdir)/pkgconfig
 	$(INSTALL) -m 644 sdl2.pc $(DESTDIR)$(libdir)/pkgconfig
 ifeq ($(INSTALL_SDL2_CONFIG),TRUE)
