diff -ur SDL2-2.0.14.orig/Makefile.in SDL2-2.0.14/Makefile.in
--- SDL2-2.0.14.orig/Makefile.in	2020-12-21 18:44:36.000000000 +0100
+++ SDL2-2.0.14/Makefile.in	2021-06-26 05:15:45.846898028 +0200
@@ -184,8 +184,6 @@
 	$(LIBTOOL) --mode=install $(INSTALL) $(objects)/$(SDLMAIN_TARGET) $(DESTDIR)$(libdir)/$(SDLMAIN_TARGET)
 	$(LIBTOOL) --mode=install $(INSTALL) $(objects)/$(SDLTEST_TARGET) $(DESTDIR)$(libdir)/$(SDLTEST_TARGET)
 install-data:
-	$(SHELL) $(auxdir)/mkinstalldirs $(DESTDIR)$(datadir)/aclocal
-	$(INSTALL) -m 644 $(srcdir)/sdl2.m4 $(DESTDIR)$(datadir)/aclocal/sdl2.m4
 	$(SHELL) $(auxdir)/mkinstalldirs $(DESTDIR)$(libdir)/pkgconfig
 	$(INSTALL) -m 644 sdl2.pc $(DESTDIR)$(libdir)/pkgconfig
 ifeq ($(INSTALL_SDL2_CONFIG),TRUE)
