diff -ur SDL2-2.26.1.orig/Makefile.in SDL2-2.26.1/Makefile.in
--- SDL2-2.26.1.orig/Makefile.in	2022-12-01 20:56:31.000000000 +0100
+++ SDL2-2.26.1/Makefile.in	2022-12-16 23:41:49.826328412 +0100
@@ -191,8 +191,6 @@
 	$(LIBTOOL) --mode=install $(INSTALL) $(objects)/$(SDLMAIN_TARGET) $(DESTDIR)$(libdir)/$(SDLMAIN_TARGET)
 	$(LIBTOOL) --mode=install $(INSTALL) $(objects)/$(SDLTEST_TARGET) $(DESTDIR)$(libdir)/$(SDLTEST_TARGET)
 install-data:
-	$(SHELL) $(auxdir)/mkinstalldirs $(DESTDIR)$(datadir)/aclocal
-	$(INSTALL) -m 644 $(srcdir)/sdl2.m4 $(DESTDIR)$(datadir)/aclocal/sdl2.m4
 	$(SHELL) $(auxdir)/mkinstalldirs $(DESTDIR)$(libdir)/pkgconfig
 	$(INSTALL) -m 644 sdl2.pc $(DESTDIR)$(libdir)/pkgconfig
 ifeq ($(INSTALL_SDL2_CONFIG),TRUE)
