diff -ur SDL2-2.0.16.orig/configure SDL2-2.0.16/configure
--- SDL2-2.0.16.orig/configure	2021-08-06 21:36:54.000000000 +0200
+++ SDL2-2.0.16/configure	2021-10-24 12:31:50.409573465 +0200
@@ -17173,11 +17173,11 @@
 #ifdef __cplusplus
 extern "C"
 #endif
-char iconv_open ();
+char libiconv_open ();
 int
 main ()
 {
-return iconv_open ();
+return libiconv_open ();
   ;
   return 0;
 }
@@ -17199,7 +17199,7 @@
 
     for ac_func in iconv
 do :
-  ac_fn_c_check_func "$LINENO" "iconv" "ac_cv_func_iconv"
+  ac_fn_c_check_func "$LINENO" "libiconv" "ac_cv_func_iconv"
 if test "x$ac_cv_func_iconv" = xyes; then :
   cat >>confdefs.h <<_ACEOF
 #define HAVE_ICONV 1
