diff -ur SDL2-2.0.12.orig/configure SDL2-2.0.12/configure
--- SDL2-2.0.12.orig/configure	2020-03-11 02:36:18.000000000 +0100
+++ SDL2-2.0.12/configure	2020-04-19 16:31:06.730769992 +0200
@@ -17060,11 +17060,11 @@
 #ifdef __cplusplus
 extern "C"
 #endif
-char iconv_open ();
+char libiconv_open ();
 int
 main ()
 {
-return iconv_open ();
+return libiconv_open ();
   ;
   return 0;
 }
@@ -17086,7 +17086,7 @@
 
     for ac_func in iconv
 do :
-  ac_fn_c_check_func "$LINENO" "iconv" "ac_cv_func_iconv"
+  ac_fn_c_check_func "$LINENO" "libiconv" "ac_cv_func_iconv"
 if test "x$ac_cv_func_iconv" = xyes; then :
   cat >>confdefs.h <<_ACEOF
 #define HAVE_ICONV 1
