diff -ur SDL2-2.0.14.orig/configure SDL2-2.0.14/configure
--- SDL2-2.0.14.orig/configure	2020-12-21 18:44:36.000000000 +0100
+++ SDL2-2.0.14/configure	2021-06-26 05:16:08.530852425 +0200
@@ -17153,11 +17153,11 @@
 #ifdef __cplusplus
 extern "C"
 #endif
-char iconv_open ();
+char libiconv_open ();
 int
 main ()
 {
-return iconv_open ();
+return libiconv_open ();
   ;
   return 0;
 }
@@ -17179,7 +17179,7 @@
 
     for ac_func in iconv
 do :
-  ac_fn_c_check_func "$LINENO" "iconv" "ac_cv_func_iconv"
+  ac_fn_c_check_func "$LINENO" "libiconv" "ac_cv_func_iconv"
 if test "x$ac_cv_func_iconv" = xyes; then :
   cat >>confdefs.h <<_ACEOF
 #define HAVE_ICONV 1
