diff -ur SDL2-2.0.18.orig/configure SDL2-2.0.18/configure
--- SDL2-2.0.18.orig/configure	2021-11-27 20:03:55.000000000 +0100
+++ SDL2-2.0.18/configure	2021-12-30 09:03:17.269382607 +0100
@@ -17176,11 +17176,11 @@
 #ifdef __cplusplus
 extern "C"
 #endif
-char iconv_open ();
+char libiconv_open ();
 int
 main ()
 {
-return iconv_open ();
+return libiconv_open ();
   ;
   return 0;
 }
@@ -17202,7 +17202,7 @@
 
     for ac_func in iconv
 do :
-  ac_fn_c_check_func "$LINENO" "iconv" "ac_cv_func_iconv"
+  ac_fn_c_check_func "$LINENO" "libiconv" "ac_cv_func_iconv"
 if test "x$ac_cv_func_iconv" = xyes; then :
   cat >>confdefs.h <<_ACEOF
 #define HAVE_ICONV 1
