diff -ur SDL2-2.0.14.orig/include/begin_code.h SDL2-2.0.14/include/begin_code.h
--- SDL2-2.0.14.orig/include/begin_code.h	2020-12-21 18:44:36.000000000 +0100
+++ SDL2-2.0.14/include/begin_code.h	2021-06-26 05:17:17.330713236 +0200
@@ -50,6 +50,7 @@
 #endif
 
 /* Some compilers use a special export keyword */
+#define DECLSPEC
 #ifndef DECLSPEC
 # if defined(__WIN32__) || defined(__WINRT__) || defined(__CYGWIN__)
 #  ifdef DLL_EXPORT
