diff --git a/libass/ass.c b/libass/ass.c
index 51fa201..7c71aca 100644
--- a/libass/ass.c
+++ b/libass/ass.c
@@ -850,7 +850,7 @@ static int decode_font(ASS_Track *track)
     size_t dsize;                  // decoded size
     unsigned char *buf = 0;
 
-    ass_msg(track->library, MSGL_V, "Font: %zu bytes encoded data",
+    ass_msg(track->library, MSGL_V, "Font: %" PRIuPTR " bytes encoded data",
             track->parser_priv->fontdata_used);
     size = track->parser_priv->fontdata_used;
     if (size % 4 == 1) {
diff --git a/libass/ass_drawing.c b/libass/ass_drawing.c
index c7f99b7..ba1e66d 100644
--- a/libass/ass_drawing.c
+++ b/libass/ass_drawing.c
@@ -24,6 +24,7 @@
 #include <math.h>
 #include <stdbool.h>
 #include <limits.h>
+#include <inttypes.h>
 
 #include "ass_utils.h"
 #include "ass_drawing.h"
@@ -257,7 +258,7 @@ bool ass_drawing_parse(ASS_Outline *outline, ASS_Rect *cbox,
 
     if (lib)
         ass_msg(lib, MSGL_V,
-                "Parsed drawing with %zu points and %zu segments",
+                "Parsed drawing with %" PRIuPTR " points and %" PRIuPTR " segments",
                 outline->n_points, outline->n_segments);
 
     drawing_free_tokens(tokens);
