diff -ur zlib-1.2.12.orig/configure zlib-1.2.12/configure
--- zlib-1.2.12.orig/configure	2022-03-28 01:39:42.000000000 +0200
+++ zlib-1.2.12/configure	2022-04-07 00:58:16.952687348 +0200
@@ -127,7 +127,7 @@
     -l* | --libdir) libdir="$2"; shift; shift ;;
     -i* | --includedir) includedir="$2"; shift; shift ;;
     -s* | --shared | --enable-shared) shared=1; shift ;;
-    -t | --static) shared=0; shift ;;
+    -t | --enable-static | --static) shared=0; shift ;;
     --solo) solo=1; shift ;;
     --cover) cover=1; shift ;;
     -z* | --zprefix) zprefix=1; shift ;;
@@ -141,8 +141,7 @@
     --sanitize) sanitize=1; shift ;;
     *)
       echo "unknown option: $1" | tee -a configure.log
-      echo "$0 --help for help" | tee -a configure.log
-      leave 1;;
+      shift ;;
     esac
 done
 
