diff -ur libogg-1.3.5.orig/Makefile.in libogg-1.3.5/Makefile.in
--- libogg-1.3.5.orig/Makefile.in	2021-06-04 05:21:09.000000000 +0200
+++ libogg-1.3.5/Makefile.in	2021-06-26 04:56:21.945141573 +0200
@@ -375,7 +375,7 @@
 #AUTOMAKE_OPTIONS = foreign 1.6 dist-zip
 AUTOMAKE_OPTIONS = foreign 1.11 dist-zip dist-xz
 ACLOCAL_AMFLAGS = -I m4
-SUBDIRS = src include doc
+SUBDIRS = src include
 m4datadir = $(datadir)/aclocal
 m4data_DATA = ogg.m4
 pkgconfigdir = $(libdir)/pkgconfig
@@ -863,7 +863,7 @@
 
 info-am:
 
-install-data-am: install-m4dataDATA install-pkgconfigDATA
+install-data-am: install-pkgconfigDATA
 
 install-dvi: install-dvi-recursive
 
