diff -ur libogg-1.3.1.orig/Makefile.in libogg-1.3.1/Makefile.in
--- libogg-1.3.1.orig/Makefile.in	2013-05-20 16:00:34.000000000 +0200
+++ libogg-1.3.1/Makefile.in	2014-05-11 20:24:22.385614783 +0200
@@ -291,7 +291,7 @@
 
 #AUTOMAKE_OPTIONS = foreign 1.6 dist-zip
 AUTOMAKE_OPTIONS = foreign 1.11 dist-zip dist-xz
-SUBDIRS = src include doc
+SUBDIRS = src include
 m4datadir = $(datadir)/aclocal
 m4data_DATA = ogg.m4
 pkgconfigdir = $(libdir)/pkgconfig
@@ -798,7 +798,7 @@
 
 info-am:
 
-install-data-am: install-m4dataDATA install-pkgconfigDATA
+install-data-am: install-pkgconfigDATA
 
 install-dvi: install-dvi-recursive
 
