diff -ur libogg-1.3.0.orig/Makefile.in libogg-1.3.0/Makefile.in
--- libogg-1.3.0.orig/Makefile.in	2011-08-04 19:37:26.000000000 +0200
+++ libogg-1.3.0/Makefile.in	2013-02-17 09:39:44.999899623 +0100
@@ -259,7 +259,7 @@
 
 #AUTOMAKE_OPTIONS = foreign 1.6 dist-zip
 AUTOMAKE_OPTIONS = foreign 1.11 dist-zip dist-xz
-SUBDIRS = src include doc
+SUBDIRS = src include
 m4datadir = $(datadir)/aclocal
 m4data_DATA = ogg.m4
 pkgconfigdir = $(libdir)/pkgconfig
@@ -749,7 +749,7 @@
 
 info-am:
 
-install-data-am: install-m4dataDATA install-pkgconfigDATA
+install-data-am: install-pkgconfigDATA
 
 install-dvi: install-dvi-recursive
 
