diff -Nur harfbuzz-1.8.8.orig/CMakeLists.txt harfbuzz-1.8.8/CMakeLists.txt
--- harfbuzz-1.8.8.orig/CMakeLists.txt	2018-07-17 17:06:41.000000000 +0200
+++ harfbuzz-1.8.8/CMakeLists.txt	2018-09-03 03:31:16.109281762 +0200
@@ -16,6 +16,7 @@
   set (BUILD_SHARED_LIBS ON)
 endif ()
 
+include(GNUInstallDirs)
 
 ## Disallow in-source builds, as CMake generated make files can collide with autotools ones
 if (NOT MSVC AND "${PROJECT_BINARY_DIR}" STREQUAL "${PROJECT_SOURCE_DIR}")
@@ -759,6 +760,13 @@
   endif ()
 endif ()
 
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/src/harfbuzz.pc.cmakein
+               ${CMAKE_CURRENT_BINARY_DIR}/harfbuzz.pc @ONLY)
+
+if (NOT SKIP_INSTALL_ALL)
+  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/harfbuzz.pc DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
+endif ()
+
 if (NOT SKIP_INSTALL_LIBRARIES AND NOT SKIP_INSTALL_ALL)
   install(TARGETS harfbuzz
     ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
diff -Nur harfbuzz-1.8.8.orig/src/harfbuzz.pc.cmakein harfbuzz-1.8.8/src/harfbuzz.pc.cmakein
--- harfbuzz-1.8.8.orig/src/harfbuzz.pc.cmakein	1970-01-01 01:00:00.000000000 +0100
+++ harfbuzz-1.8.8/src/harfbuzz.pc.cmakein	2018-09-03 03:31:16.109281762 +0200
@@ -0,0 +1,10 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+libdir=@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_LIBDIR@
+includedir=@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_INCLUDEDIR@
+
+Name: harfbuzz
+Description: HarfBuzz text shaping library
+Version: @HB_VERSION@
+
+Libs: -L${libdir} -lharfbuzz
+Cflags: -I${includedir}/harfbuzz
