diff -ur harfbuzz-2.2.0.orig/CMakeLists.txt harfbuzz-2.2.0/CMakeLists.txt
--- harfbuzz-2.2.0.orig/CMakeLists.txt	2018-10-30 08:52:22.000000000 +0100
+++ harfbuzz-2.2.0/CMakeLists.txt	2018-12-03 03:12:44.352566609 +0100
@@ -16,6 +16,7 @@
   set (BUILD_SHARED_LIBS ON)
 endif ()
 
+include(GNUInstallDirs)
 
 ## Disallow in-source builds, as CMake generated make files can collide with autotools ones
 if (NOT MSVC AND "${PROJECT_BINARY_DIR}" STREQUAL "${PROJECT_SOURCE_DIR}")
@@ -785,6 +786,13 @@
   endif ()
 endif ()
 
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/harfbuzz.pc.cmakein
+               ${CMAKE_CURRENT_BINARY_DIR}/harfbuzz.pc @ONLY)
+
+if (NOT SKIP_INSTALL_ALL)
+  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/harfbuzz.pc DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
+endif ()
+
 if (NOT SKIP_INSTALL_LIBRARIES AND NOT SKIP_INSTALL_ALL)
   install(TARGETS harfbuzz
     EXPORT harfbuzzConfig
