diff -ur harfbuzz-2.0.2.orig/util/options.cc harfbuzz-2.0.2/util/options.cc
--- harfbuzz-2.0.2.orig/util/options.cc	2018-10-18 06:45:27.000000000 +0200
+++ harfbuzz-2.0.2/util/options.cc	2018-10-25 04:44:12.546091772 +0200
@@ -145,7 +145,7 @@
 void
 option_parser_t::parse (int *argc, char ***argv)
 {
-  setlocale (LC_ALL, "");
+  //setlocale (LC_ALL, "");
 
   GError *parse_error = nullptr;
   if (!g_option_context_parse (context, argc, argv, &parse_error))
