diff -ur harfbuzz-1.8.8.orig/util/options.cc harfbuzz-1.8.8/util/options.cc
--- harfbuzz-1.8.8.orig/util/options.cc	2018-07-17 16:29:54.000000000 +0200
+++ harfbuzz-1.8.8/util/options.cc	2018-09-03 03:29:23.246005302 +0200
@@ -149,7 +149,7 @@
 void
 option_parser_t::parse (int *argc, char ***argv)
 {
-  setlocale (LC_ALL, "");
+  setlocale (LC_ALL, "C");
 
   GError *parse_error = nullptr;
   if (!g_option_context_parse (context, argc, argv, &parse_error))
