diff --git a/Makefile.am b/Makefile.am
index c85c673..5bc28e9 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -19,7 +19,7 @@ EXTRA_DIST = zimg.pc.in
 exampledir = $(docdir)/example
 examplemiscdir = $(exampledir)/misc
 
-
+if FALSE
 dist_doc_DATA = ChangeLog COPYING README.md
 
 dist_example_DATA = \
@@ -38,7 +38,7 @@ dist_examplemisc_DATA = \
 	src/testcommon/mmap.h \
 	src/testcommon/win32_bitmap.cpp \
 	src/testcommon/win32_bitmap.h
-
+endif # FALSE
 
 noinst_PROGRAMS =
 lib_LTLIBRARIES = libzimg.la
