diff -ur libcdio-0.90.orig/Makefile.in libcdio-0.90/Makefile.in
--- libcdio-0.90.orig/Makefile.in	2012-10-28 00:23:29.000000000 +0200
+++ libcdio-0.90/Makefile.in	2013-03-03 09:13:41.776915051 +0100
@@ -384,7 +384,7 @@
 	make-check-filter.rb \
 	package/libcdio.spec.in 
 
-SUBDIRS = doc include lib src test example
+SUBDIRS = include lib
 
 # pkg-config(1) related rules
 pkgconfigdir = $(libdir)/pkgconfig
