diff --git a/include/cdio/types.h b/include/cdio/types.h
index dc0f90cf..0525a969 100644
--- a/include/cdio/types.h
+++ b/include/cdio/types.h
@@ -109,7 +109,7 @@ typedef uint8_t ubyte;
 
 #if __GNUC__ > 2 || (__GNUC__ == 2 && __GNUC_MINOR__ > 4)
 #define GNUC_PRINTF( format_idx, arg_idx )              \
-  __attribute__((format (printf, format_idx, arg_idx)))
+  __attribute__((format (gnu_printf, format_idx, arg_idx)))
 #define GNUC_SCANF( format_idx, arg_idx )               \
   __attribute__((format (scanf, format_idx, arg_idx)))
 #define GNUC_FORMAT( arg_idx )                  \
