diff -ur libcdio-0.90.orig/configure libcdio-0.90/configure
--- libcdio-0.90.orig/configure	2012-10-28 00:23:27.000000000 +0200
+++ libcdio-0.90/configure	2013-03-03 09:13:02.156914109 +0100
@@ -17455,8 +17455,8 @@
 	if test "x$enable_largefile" != "xno"; then
 	  { $as_echo "$as_me:${as_lineno-$LINENO}: result: enabling support for large files (_FILE_OFFSET_BITS=64)" >&5
 $as_echo "enabling support for large files (_FILE_OFFSET_BITS=64)" >&6; }
-	  LIBCDIO_CFLAGS+=' -D_FILE_OFFSET_BITS=64 -D_OFF_T_DEFINED -D_OFF_T_ -D_off_t=off64_t -Doff_t=off64_t'
-	  LIBISO9660_CFLAGS+=' -D_FILE_OFFSET_BITS=64 -D_OFF_T_DEFINED -D_OFF_T_ -D_off_t=off64_t -Doff_t=off64_t'
+	  LIBCDIO_CFLAGS+=' -D_FILE_OFFSET_BITS=64'
+	  LIBISO9660_CFLAGS+=' -D_FILE_OFFSET_BITS=64'
 	fi
 	joliet_supported="yes"
 	if test "x$enable_cdda_player" = "xyes"; then
