diff -ur libcdio-0.90.orig/lib/driver/MSWindows/win32.c libcdio-0.90/lib/driver/MSWindows/win32.c
--- libcdio-0.90.orig/lib/driver/MSWindows/win32.c	2012-10-02 18:23:44.000000000 +0200
+++ libcdio-0.90/lib/driver/MSWindows/win32.c	2013-03-03 09:15:27.124917554 +0100
@@ -980,6 +980,7 @@
   _funcs.get_track_lba          = NULL; /* This could be done if need be. */
   _funcs.get_track_msf          = _cdio_get_track_msf;
   _funcs.get_track_preemphasis  = get_track_preemphasis_generic,
+#undef lseek
   _funcs.lseek                  = NULL;
   _funcs.read                   = NULL;
   _funcs.read_audio_sectors     = read_audio_sectors;
