diff -ur libcdio-0.90.orig/include/cdio/types.h libcdio-0.90/include/cdio/types.h
--- libcdio-0.90.orig/include/cdio/types.h	2012-09-26 18:14:57.000000000 +0200
+++ libcdio-0.90/include/cdio/types.h	2014-01-10 03:52:50.361294723 +0100
@@ -140,7 +140,7 @@
 #define GNUC_UNUSED                             \
   __attribute__((unused))
 #define GNUC_PACKED                             \
-  __attribute__((packed))
+  __attribute__((packed)) __attribute__((gcc_struct))
 #else   /* !__GNUC__ */
 #define GNUC_PRINTF( format_idx, arg_idx )
 #define GNUC_SCANF( format_idx, arg_idx )
