diff -ur flac-1.4.3.orig/Makefile.in flac-1.4.3/Makefile.in
--- flac-1.4.3.orig/Makefile.in	2023-06-22 10:50:44.205745737 +0200
+++ flac-1.4.3/Makefile.in	2024-03-27 01:05:36.444258732 +0100
@@ -396,7 +396,7 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 ACLOCAL_AMFLAGS = -I m4
-SUBDIRS = doc include m4 man src test microbench oss-fuzz \
+SUBDIRS = include src \
 	$(am__append_1)
 EXTRA_DIST = \
 	CMakeLists.txt \
diff -ur flac-1.4.3.orig/src/libFLAC/Makefile.in flac-1.4.3/src/libFLAC/Makefile.in
--- flac-1.4.3.orig/src/libFLAC/Makefile.in	2023-06-22 10:50:44.685747363 +0200
+++ flac-1.4.3/src/libFLAC/Makefile.in	2024-03-27 01:05:36.444258732 +0100
@@ -1044,7 +1044,7 @@
 
 info-am:
 
-install-data-am: install-m4dataDATA install-pkgconfigDATA
+install-data-am: install-pkgconfigDATA
 
 install-dvi: install-dvi-recursive
 
diff -ur flac-1.4.3.orig/src/libFLAC++/Makefile.in flac-1.4.3/src/libFLAC++/Makefile.in
--- flac-1.4.3.orig/src/libFLAC++/Makefile.in	2023-06-22 10:50:44.657747267 +0200
+++ flac-1.4.3/src/libFLAC++/Makefile.in	2024-03-27 01:05:36.444258732 +0100
@@ -750,7 +750,7 @@
 
 info-am:
 
-install-data-am: install-m4dataDATA install-pkgconfigDATA
+install-data-am: install-pkgconfigDATA
 
 install-dvi: install-dvi-am
 
