diff -ur flac-1.3.0.orig/Makefile.in flac-1.3.0/Makefile.in
--- flac-1.3.0.orig/Makefile.in	2013-05-27 10:11:58.639908999 +0200
+++ flac-1.3.0/Makefile.in	2014-06-09 03:28:17.689098235 +0200
@@ -323,7 +323,7 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 ACLOCAL_AMFLAGS = -I m4
-SUBDIRS = doc include m4 man src examples test build objs
+SUBDIRS = include src build objs
 EXTRA_DIST = \
 	COPYING.FDL \
 	COPYING.GPL \
diff -ur flac-1.3.0.orig/src/libFLAC/Makefile.in flac-1.3.0/src/libFLAC/Makefile.in
--- flac-1.3.0.orig/src/libFLAC/Makefile.in	2013-05-27 10:11:57.459950296 +0200
+++ flac-1.3.0/src/libFLAC/Makefile.in	2014-06-09 03:28:56.857096971 +0200
@@ -899,7 +899,7 @@
 
 info-am:
 
-install-data-am: install-m4dataDATA install-pkgconfigDATA
+install-data-am: install-pkgconfigDATA
 
 install-dvi: install-dvi-recursive
 
diff -ur flac-1.3.0.orig/src/libFLAC++/Makefile.in flac-1.3.0/src/libFLAC++/Makefile.in
--- flac-1.3.0.orig/src/libFLAC++/Makefile.in	2013-05-27 10:11:57.399952397 +0200
+++ flac-1.3.0/src/libFLAC++/Makefile.in	2014-06-09 03:29:06.325096665 +0200
@@ -646,7 +646,7 @@
 
 info-am:
 
-install-data-am: install-m4dataDATA install-pkgconfigDATA
+install-data-am: install-pkgconfigDATA
 
 install-dvi: install-dvi-am
 
diff -ur flac-1.3.0.orig/src/Makefile.in flac-1.3.0/src/Makefile.in
--- flac-1.3.0.orig/src/Makefile.in	2013-05-27 10:11:57.243957858 +0200
+++ flac-1.3.0/src/Makefile.in	2014-06-09 03:29:24.989096063 +0200
@@ -298,8 +298,6 @@
 SUBDIRS = \
 	share \
 	libFLAC \
-	flac \
-	metaflac \
 	monkeys_audio_utilities \
 	$(XMMS_DIRS) \
 	test_grabbag \
