diff -ur flac-1.4.2.orig/Makefile.in flac-1.4.2/Makefile.in
--- flac-1.4.2.orig/Makefile.in	2022-10-21 19:54:45.140608200 +0200
+++ flac-1.4.2/Makefile.in	2022-11-06 01:52:24.379908868 +0100
@@ -395,7 +395,7 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 ACLOCAL_AMFLAGS = -I m4
-SUBDIRS = doc include m4 man src test microbench oss-fuzz \
+SUBDIRS = include src \
 	$(am__append_1)
 EXTRA_DIST = \
 	CMakeLists.txt \
diff -ur flac-1.4.2.orig/src/libFLAC/Makefile.in flac-1.4.2/src/libFLAC/Makefile.in
--- flac-1.4.2.orig/src/libFLAC/Makefile.in	2022-10-21 19:54:46.738329000 +0200
+++ flac-1.4.2/src/libFLAC/Makefile.in	2022-11-06 01:53:02.803851144 +0100
@@ -1039,7 +1039,7 @@
 
 info-am:
 
-install-data-am: install-m4dataDATA install-pkgconfigDATA
+install-data-am: install-pkgconfigDATA
 
 install-dvi: install-dvi-recursive
 
diff -ur flac-1.4.2.orig/src/libFLAC++/Makefile.in flac-1.4.2/src/libFLAC++/Makefile.in
--- flac-1.4.2.orig/src/libFLAC++/Makefile.in	2022-10-21 19:54:46.640249800 +0200
+++ flac-1.4.2/src/libFLAC++/Makefile.in	2022-11-06 01:53:18.647827412 +0100
@@ -744,7 +744,7 @@
 
 info-am:
 
-install-data-am: install-m4dataDATA install-pkgconfigDATA
+install-data-am: install-pkgconfigDATA
 
 install-dvi: install-dvi-am
 
