diff -ur flac-1.2.1.orig/Makefile.in flac-1.2.1/Makefile.in
--- flac-1.2.1.orig/Makefile.in	2007-09-16 22:04:57.000000000 +0200
+++ flac-1.2.1/Makefile.in	2013-02-17 09:31:46.963888262 +0100
@@ -234,7 +234,7 @@
 
 AUTOMAKE_OPTIONS = foreign 1.7
 
-SUBDIRS = doc include m4 man src examples test build obj
+SUBDIRS = include src build obj
 
 DISTCLEANFILES = libtool-disable-static
 
diff -ur flac-1.2.1.orig/src/libFLAC/Makefile.in flac-1.2.1/src/libFLAC/Makefile.in
--- flac-1.2.1.orig/src/libFLAC/Makefile.in	2007-09-16 22:05:12.000000000 +0200
+++ flac-1.2.1/src/libFLAC/Makefile.in	2013-02-17 09:31:46.963888262 +0100
@@ -813,7 +813,7 @@
 
 info-am:
 
-install-data-am: install-m4dataDATA install-pkgconfigDATA
+install-data-am: install-pkgconfigDATA
 
 install-exec-am: install-libLTLIBRARIES
 
diff -ur flac-1.2.1.orig/src/libFLAC++/Makefile.in flac-1.2.1/src/libFLAC++/Makefile.in
--- flac-1.2.1.orig/src/libFLAC++/Makefile.in	2007-09-16 22:05:10.000000000 +0200
+++ flac-1.2.1/src/libFLAC++/Makefile.in	2013-02-17 09:31:46.967888262 +0100
@@ -556,7 +556,7 @@
 
 info-am:
 
-install-data-am: install-m4dataDATA install-pkgconfigDATA
+install-data-am: install-pkgconfigDATA
 
 install-exec-am: install-libLTLIBRARIES
 
diff -ur flac-1.2.1.orig/src/Makefile.in flac-1.2.1/src/Makefile.in
--- flac-1.2.1.orig/src/Makefile.in	2007-09-16 22:05:09.000000000 +0200
+++ flac-1.2.1/src/Makefile.in	2013-02-17 09:31:57.071888502 +0100
@@ -226,8 +226,6 @@
 SUBDIRS = \
 	libFLAC \
 	share \
-	flac \
-	metaflac \
 	monkeys_audio_utilities \
 	$(XMMS_DIRS) \
 	plugin_winamp2 \
