From dd748e887b9c26ab00ac3de8817a1dccfeb225b1 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Fri, 25 Jan 2013 01:28:35 +0100
Subject: [PATCH 01/20] configure: flavor selection for the target

---
 configure |  110 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 110 insertions(+)

diff --git a/configure b/configure
index d925d4e..1cae473 100755
--- a/configure
+++ b/configure
@@ -578,6 +578,7 @@ Miscellaneous options:
   --with-install=PATH    path to a custom install program
 
 Advanced options:
+  --flavor=system           build a specified target system
   --enable-mmx              enable MMX [autodetect]
   --enable-mmxext           enable MMX2 (Pentium III, Athlon) [autodetect]
   --enable-3dnow            enable 3DNow! [autodetect]
@@ -635,6 +636,7 @@ exit 0
 # GOTCHA: the variables below defines the default behavior for autodetection
 # and have - unless stated otherwise - at least 2 states : yes no
 # If autodetection is available then the third state is: auto
+_flavor=none
 _mmx=auto
 _3dnow=auto
 _3dnowext=auto
@@ -1030,6 +1032,9 @@ for ac_option do
   --disable-debug)
     _debug=
     ;;
+  --flavor=*)
+    _flavor=`echo $ac_option | cut -d '=' -f 2`
+    ;;
   --enable-runtime-cpudetection)    _runtime_cpudetection=yes   ;;
   --disable-runtime-cpudetection)   _runtime_cpudetection=no    ;;
   --enable-cross-compile)           _cross_compile=yes          ;;
@@ -2641,6 +2646,111 @@ EOF
   test -n "$cflags_stackrealign" && echores "yes" || echores "no"
 fi # if darwin && test "$cc_vendor" = "gnu" ; then
 
+# user selected target system
+if test "$_runtime_cpudetection" = no ; then
+    case "$host_arch-$cc_vendor-$_flavor" in
+        *-none) ;; # No flavor, standard build
+        i386-gnu-corei7|i386-gnu-i7)
+         _mmx=yes
+         _3dnow=no
+         _3dnowext=no
+         _mmxext=yes
+         _sse=yes
+         _sse2=yes
+         _sse3=yes
+         _ssse3=yes
+         _sse4_1=yes
+         _sse4_2=yes
+         _avx=no
+         _mcpu=$cpuopt=corei7
+         _march=-march=corei7
+         ;;
+        i386-gnu-pentium4|i386-gnu-p4)
+         _mmx=yes
+         _3dnow=no
+         _3dnowext=no
+         _mmxext=yes
+         _sse=yes
+         _sse2=yes
+         _sse3=no
+         _ssse3=no
+         _sse4_1=no
+         _sse4_2=no
+         _avx=no
+         _mcpu=$cpuopt=pentium4
+         _march=-march=pentium4
+         ;;
+        i386-gnu-pentium3|i386-gnu-p3)
+         _mmx=yes
+         _3dnow=no
+         _3dnowext=no
+         _mmxext=yes
+         _sse=yes
+         _sse2=no
+         _sse3=no
+         _ssse3=no
+         _sse4_1=no
+         _sse4_2=no
+         _avx=no
+         _mcpu=$cpuopt=pentium3
+         _march=-march=pentium3
+         ;;
+        i386-gnu-athlon)
+         _mmx=yes
+         _3dnow=yes
+         _3dnowext=yes
+         _mmxext=yes
+         _sse=no
+         _sse2=no
+         _sse3=no
+         _ssse3=no
+         _sse4_1=no
+         _sse4_2=no
+         _avx=no
+         _mcpu=$cpuopt=athlon
+         _march=-march=athlon
+         ;;
+        i386-gnu-athlon-xp)
+         _mmx=yes
+         _3dnow=yes
+         _3dnowext=yes
+         _mmxext=yes
+         _sse=yes
+         _sse2=no
+         _sse3=no
+         _ssse3=no
+         _sse4_1=no
+         _sse4_2=no
+         _avx=no
+         _mcpu=$cpuopt=athlon-xp
+         _march=-march=athlon-xp
+         ;;
+        i386-gnu-k8)
+         _mmx=yes
+         _3dnow=yes
+         _3dnowext=yes
+         _mmxext=yes
+         _sse=yes
+         _sse2=yes
+         _sse3=no
+         _ssse3=no
+         _sse4_1=no
+         _sse4_2=no
+         _avx=no
+         _mcpu=$cpuopt=k8
+         _march=-march=k8
+         ;;
+        *)
+         echo "Invalid/unsupported target system $host_arch-$cc_vendor-$_flavor"
+         exit 1
+         ;;
+    esac
+    if test "$_flavor" != none ; then
+        # move this to case if adding different cpu
+        iproc=686
+        _optimizing="Custom Target $_flavor"
+    fi
+fi
 
 # Checking for CFLAGS
 _install_strip="-s"
-- 
1.7.10.4

