From 18611f82be3d37780c2c8a3e157fd4cabf479220 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Sat, 2 Feb 2013 12:43:34 +0100
Subject: [PATCH 05/27] make fontconfig spam while creating font cache

---
 mplayer.c | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/mplayer.c b/mplayer.c
index 5be0e5f..c7deec1 100644
--- a/mplayer.c
+++ b/mplayer.c
@@ -139,6 +139,11 @@
 #include "stream/stream_dvd.h"
 #endif
 
+#if defined(CONFIG_FONTCONFIG) && defined(_WIN32)
+#include <fontconfig/fontconfig.h>
+extern int FcDebugVal;
+#endif
+
 int slave_mode;
 int player_idle_mode;
 int quiet;
@@ -3062,6 +3067,12 @@ int main(int argc, char *argv[])
         gui(GUI_SET_CONTEXT, mpctx);
         gui(GUI_SET_STATE, (void *)(filename ? GUI_PLAY : GUI_STOP));
     }
+#elif defined(CONFIG_FONTCONFIG) && defined(_WIN32)
+    /* force cache creation here and display fontconfig scan activity */
+    FcDebugVal = 128;
+    FcConfigEnableHome(FcFalse);
+    FcInit();
+    FcDebugVal = 0;
 #endif
 
 // ******************* Now, let's see the per-file stuff ********************
-- 
1.8.4.4

