From 1bf4cbeea819a3484d03bed6d0cf4655097ddaf6 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Sat, 2 Feb 2013 12:43:34 +0100
Subject: [PATCH 05/28] make fontconfig spam while creating font cache

---
 mplayer.c | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/mplayer.c b/mplayer.c
index 527bc40..5b8f507 100644
--- a/mplayer.c
+++ b/mplayer.c
@@ -139,6 +139,11 @@
 #include "stream/stream_dvd.h"
 #endif
 
+#if defined(CONFIG_FONTCONFIG) && defined(_WIN32)
+#include <fontconfig/fontconfig.h>
+extern int FcDebugVal;
+#endif
+
 int slave_mode;
 int player_idle_mode;
 int quiet;
@@ -3066,6 +3071,14 @@ int main(int argc, char *argv[])
         gui(GUI_SET_CONTEXT, mpctx);
         gui(GUI_SET_STATE, (void *)(intptr_t)(filename ? GUI_PLAY : GUI_STOP));
     }
+#elif defined(CONFIG_FONTCONFIG) && defined(_WIN32)
+    /* force cache creation here and display fontconfig scan activity */
+    if (font_fontconfig >= 0) {
+        FcDebugVal = 128;
+        FcConfigEnableHome(FcFalse);
+        FcInit();
+        FcDebugVal = 0;
+    }
 #endif
 
 // ******************* Now, let's see the per-file stuff ********************
-- 
2.5.0

