From 705eb0508f61971505436986c2dec24d6eb955a3 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Sat, 2 Feb 2013 12:43:34 +0100
Subject: [PATCH 05/24] make fontconfig spam while creating font cache

---
 mplayer.c |   11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/mplayer.c b/mplayer.c
index ee9a5f0..d5e9f25 100644
--- a/mplayer.c
+++ b/mplayer.c
@@ -139,6 +139,11 @@
 #include "stream/stream_dvd.h"
 #endif
 
+#if defined(CONFIG_FONTCONFIG) && defined(_WIN32)
+#include <fontconfig/fontconfig.h>
+extern int FcDebugVal;
+#endif
+
 int slave_mode;
 int player_idle_mode;
 int quiet;
@@ -3060,6 +3065,12 @@ int main(int argc, char *argv[])
         gui(GUI_SET_CONTEXT, mpctx);
         gui(GUI_SET_STATE, (void *)(filename ? GUI_PLAY : GUI_STOP));
     }
+#elif defined(CONFIG_FONTCONFIG) && defined(_WIN32)
+    /* force cache creation here and display fontconfig scan activity */
+    FcDebugVal = 128;
+    FcConfigEnableHome(FcFalse);
+    FcInit();
+    FcDebugVal = 0;
 #endif
 
 // ******************* Now, let's see the per-file stuff ********************
-- 
1.7.10.4

