From 65920b68465aa93ef6fec3409941c2a4c676134b Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Sat, 2 Feb 2013 12:45:25 +0100
Subject: [PATCH 07/24] audio: disable muting audio at exit on win32, mplayer
 somehow crashes

---
 mplayer.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/mplayer.c b/mplayer.c
index d5e9f25..8bf3b22 100644
--- a/mplayer.c
+++ b/mplayer.c
@@ -702,8 +702,10 @@ void exit_player_with_rc(enum exit_reason how, int rc)
         send_udp(udp_ip, udp_port, "bye");
 #endif /* CONFIG_NETWORKING */
 
+#ifndef _WIN32
     if (mpctx->user_muted && !mpctx->edl_muted)
         mixer_mute(&mpctx->mixer);
+#endif
     uninit_player(INITIALIZED_ALL);
 #if defined(__MINGW32__) || defined(__CYGWIN__)
     timeEndPeriod(1);
-- 
1.7.10.4

