From db143a6222885805fddf4f18629f3fe7da451120 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Sat, 2 Feb 2013 12:47:25 +0100
Subject: [PATCH 08/28] win32: color term osd

---
 mplayer.c | 28 +++++++++++++++++++++++++---
 1 file changed, 25 insertions(+), 3 deletions(-)

diff --git a/mplayer.c b/mplayer.c
index 4c0f4a3..e9288c6 100644
--- a/mplayer.c
+++ b/mplayer.c
@@ -197,6 +197,25 @@ int use_gui;
 int enqueue;
 #endif
 
+#ifdef _WIN32
+#include <io.h>
+static void term_osd_eraseline(void)
+{
+    DWORD wr;
+    COORD pos;
+    HANDLE hOut = GetStdHandle(STD_OUTPUT_HANDLE);
+    CONSOLE_SCREEN_BUFFER_INFO cinfo;
+    GetConsoleScreenBufferInfo(hOut, &cinfo);
+    pos.X = 0;
+    pos.Y = cinfo.dwCursorPosition.Y - 1;
+    FillConsoleOutputCharacter(hOut, ' ', cinfo.dwSize.X, pos, &wr);
+    FillConsoleOutputAttribute(hOut, cinfo.wAttributes, cinfo.dwSize.X, pos, &wr);
+    SetConsoleCursorPosition(hOut, pos);
+}
+#else
+#define term_osd_eraseline() printf("%s", term_osd_esc)
+#endif
+
 static int list_properties;
 
 int osd_level = 1;
@@ -1583,8 +1602,10 @@ static void update_osd_msg(void)
             strncpy((char *)osd_text, msg->msg, 127);
             if (mpctx->sh_video)
                 vo_osd_changed(OSDTYPE_OSD);
-            else if (term_osd)
-                mp_msg(MSGT_CPLAYER, MSGL_STATUS, "%s%s\n", term_osd_esc, msg->msg);
+            else if (term_osd) {
+                term_osd_eraseline();
+                mp_msg(MSGT_CPLAYER, MSGL_STATUS, "%s\n", msg->msg);
+            }
         }
         return;
     }
@@ -1655,7 +1676,8 @@ static void update_osd_msg(void)
     // Clear the term osd line
     if (term_osd && osd_text[0]) {
         osd_text[0] = 0;
-        printf("%s\n", term_osd_esc);
+        term_osd_eraseline();
+        printf("\n");
     }
 }
 
-- 
2.5.0

