From f206de67abc43e40ebc4b44cbabbfdf1953ef30a Mon Sep 17 00:00:00 2001
From: Gianluigi Tiesi <sherpya@netfarm.it>
Date: Thu, 14 Feb 2013 05:14:05 +0100
Subject: [PATCH 08/26] win32: disable_system_conf defaults to 1 on win32

---
 mpcommon.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/mpcommon.c b/mpcommon.c
index cb71306a9..2672ff240 100644
--- a/mpcommon.c
+++ b/mpcommon.c
@@ -428,7 +428,11 @@ int select_video(demuxer_t* demuxer, int video_id)
 }
 
 /* Parse -noconfig common to both programs */
+#ifdef __MINGW32__
+int disable_system_conf=1;
+#else
 int disable_system_conf=0;
+#endif
 int disable_user_conf=0;
 #ifdef CONFIG_GUI
 int disable_gui_conf=0;
-- 
2.47.2

