From f34ec2a40377adeba6ba3be406da8788869b0a27 Mon Sep 17 00:00:00 2001
From: Gianluigi Tiesi <sherpya@netfarm.it>
Date: Thu, 14 Feb 2013 05:14:05 +0100
Subject: [PATCH 09/28] win32: disable_system_conf defaults to 1 on win32


diff --git a/mpcommon.c b/mpcommon.c
index 5866d1179..d2e944d1a 100644
--- a/mpcommon.c
+++ b/mpcommon.c
@@ -428,7 +428,11 @@ int select_video(demuxer_t* demuxer, int video_id)
 }
 
 /* Parse -noconfig common to both programs */
+#ifdef __MINGW32__
+int disable_system_conf=1;
+#else
 int disable_system_conf=0;
+#endif
 int disable_user_conf=0;
 #ifdef CONFIG_GUI
 int disable_gui_conf=0;
-- 
2.33.0

