From c33b3ea7f69852906bc2e41c83d29d58178c4a12 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Thu, 14 Feb 2013 05:14:05 +0100
Subject: [PATCH 09/27] win32: disable_system_conf defaults to 1 on win32

---
 mpcommon.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/mpcommon.c b/mpcommon.c
index 211828c49..3200e2555 100644
--- a/mpcommon.c
+++ b/mpcommon.c
@@ -427,7 +427,11 @@ int select_video(demuxer_t* demuxer, int video_id)
 }
 
 /* Parse -noconfig common to both programs */
+#ifdef __MINGW32__
+int disable_system_conf=1;
+#else
 int disable_system_conf=0;
+#endif
 int disable_user_conf=0;
 #ifdef CONFIG_GUI
 int disable_gui_conf=0;
-- 
2.16.2

