From bf6cbbfe0274383a468538ec6b6015882c231af9 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Sat, 2 Feb 2013 12:44:52 +0100
Subject: [PATCH 10/22] added missing functions proto

---
 libvo/w32_common.c |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/libvo/w32_common.c b/libvo/w32_common.c
index ac90565..688b2e9 100644
--- a/libvo/w32_common.c
+++ b/libvo/w32_common.c
@@ -37,6 +37,9 @@
 static const char classname[] = "MPlayer - The Movie Player";
 int vo_vm = 0;
 
+HDC vo_w32_get_dc(HWND wnd);
+void vo_w32_release_dc(HWND wnd, HDC dc);
+
 // last non-fullscreen extends
 static int prev_width;
 static int prev_height;
-- 
1.7.10.4

