From d6729421404d11cbf9258cac04d21abbbb012cb5 Mon Sep 17 00:00:00 2001
From: Gianluigi Tiesi <sherpya@netfarm.it>
Date: Sun, 3 Mar 2013 09:34:08 +0100
Subject: [PATCH 11/28] stream_cddb: fix include path on MinGW-w64


diff --git a/stream/stream_cddb.c b/stream/stream_cddb.c
index ed319cbc2..73e1cdd37 100644
--- a/stream/stream_cddb.c
+++ b/stream/stream_cddb.c
@@ -60,7 +60,7 @@
 #elif defined(__FreeBSD__) || defined(__FreeBSD_kernel__) || defined(__NetBSD__) || defined(__OpenBSD__) || defined(__DragonFly__)
 #include <sys/cdio.h>
 #elif defined(__MINGW32__) || defined(__CYGWIN__)
-#include <ddk/ntddcdrm.h>
+#include <ntddcdrm.h>
 #elif defined(__bsdi__)
 #include <dvd.h>
 #elif defined(__APPLE__) || defined(__DARWIN__)
-- 
2.33.0

