From bc24122337fedf5aca5d3fff4088009c82b309a2 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Sun, 10 Mar 2013 05:05:55 +0100
Subject: [PATCH 12/28] vcd support on MinGW-w64

---
 configure               | 2 +-
 stream/vcd_read_win32.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure b/configure
index 21f712d..732cac3 100755
--- a/configure
+++ b/configure
@@ -6084,7 +6084,7 @@ if test "$_vcd" = auto; then
   if linux || freebsd || netbsd || openbsd || dragonfly || bsdos || darwin || sunos || os2; then
     _vcd=yes
   elif mingw32; then
-  header_check ddk/ntddcdrm.h && _vcd=yes
+  header_check_broken windows.h ntddcdrm.h && _vcd=yes
   fi
 fi
 if test "$_vcd" = yes; then
diff --git a/stream/vcd_read_win32.h b/stream/vcd_read_win32.h
index a0035ad..9b77cee 100644
--- a/stream/vcd_read_win32.h
+++ b/stream/vcd_read_win32.h
@@ -22,7 +22,7 @@
 #include <stddef.h>
 #include <stdlib.h>
 #include <string.h>
-#include <ddk/ntddcdrm.h>
+#include <ntddcdrm.h>
 #include "mp_msg.h"
 
 typedef struct mp_vcd_priv_st mp_vcd_priv_t;
-- 
2.5.0

