From 6e0427f66b1666d509f62f10188aa447fc0a1172 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Sat, 2 Feb 2013 12:49:54 +0100
Subject: [PATCH 16/22] configure: fix detection of static fontconfig library

---
 configure |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure b/configure
index 5bd0933..c6a012c 100755
--- a/configure
+++ b/configure
@@ -6160,7 +6160,7 @@ int main(void) {
 }
 EOF
   _fontconfig=no
-  for ld_tmp in "" "-lexpat -lfreetype" "-lexpat -lfreetype -lz" "-lexpat -lfreetype -lz -liconv" ; do
+  for ld_tmp in "" "-lfontconfig -lfreetype -lz -lbz2 -lexpat" "-lexpat -lfreetype" "-lexpat -lfreetype -lz" "-lexpat -lfreetype -lz -liconv" ; do
     ld_tmp="-lfontconfig $ld_tmp"
     cc_check $ld_tmp && _fontconfig=yes && extra_ldflags="$extra_ldflags $ld_tmp" && break
   done
-- 
1.7.10.4

