From e8cce40d816126d76434a65e831c9154bad8bfa6 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Thu, 14 Feb 2013 05:14:05 +0100
Subject: [PATCH 18/20] win32: disable_system_conf defaults to 1 on win32

---
 mpcommon.c |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/mpcommon.c b/mpcommon.c
index a49bbce..462005a 100644
--- a/mpcommon.c
+++ b/mpcommon.c
@@ -417,7 +417,11 @@ int select_audio(demuxer_t* demuxer, int audio_id, char* audio_lang)
 }
 
 /* Parse -noconfig common to both programs */
+#ifdef __MINGW32__
+int disable_system_conf=1;
+#else
 int disable_system_conf=0;
+#endif
 int disable_user_conf=0;
 #ifdef CONFIG_GUI
 int disable_gui_conf=0;
-- 
1.7.10.4

