From 62c20adc9e58d7851661edbb7bae427540f68a55 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Sun, 25 Feb 2018 06:53:50 +0100
Subject: [PATCH 18/28] configure: enabled support for libfdk-aac in lavc

---
 configure | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/configure b/configure
index ab592307e..4891bd108 100755
--- a/configure
+++ b/configure
@@ -420,6 +420,7 @@ Codecs:
   --enable-faad             enable FAAD2 (AAC) [autodetect]
   --disable-faac            disable support for FAAC (AAC encoder) [autodetect]
   --disable-faac-lavc       disable support for FAAC in libavcodec [autodetect]
+  --disable-libfdk-aac-lavc disable support for libfdk-aac in libavcodec [autodetect]
   --disable-ladspa          disable LADSPA plugin support [autodetect]
   --disable-libbs2b         disable libbs2b audio filter support [autodetect]
   --disable-libdv           disable libdv 0.9.5 en/decoding support [autodetect]
@@ -740,6 +741,7 @@ _libmpeg2=auto
 _libmpeg2_internal=no
 _faad=auto
 _faac=auto
+_libfdk_aac_lavc=auto
 _ladspa=auto
 _libbs2b=auto
 _libilbc=auto
@@ -1153,6 +1155,8 @@ for ac_option do
   --disable-faad)       _faad=no        ;;
   --enable-faac)        _faac=yes       ;;
   --disable-faac)       _faac=no        ;;
+  --enable-libfdk-aac-lavc)      _libfdk_aac_lavc=yes     ;;
+  --disable-libfdk-aac-lavc)     _libfdk_aac_lavc=no      ;;
   --enable-ladspa)      _ladspa=yes     ;;
   --disable-ladspa)     _ladspa=no      ;;
   --enable-libbs2b)     _libbs2b=yes    ;;
@@ -6915,6 +6919,19 @@ else
 fi
 echores "$_faac"
 
+echocheck "libfdk-aac-lavc support"
+if test "$_libfdk_aac_lavc" = auto; then
+  _libfdk_aac_lavc=no
+  ld_libfdk_aac_lavc="-lfdk-aac"
+  statement_check fdk-aac/aacenc_lib.h 'aacEncOpen(0,0,0)' $ld_libfdk_aac_lavc && _libfdk_aac_lavc=yes
+fi
+if test "$_libfdk_aac_lavc" = yes ; then
+  def_libfdk_aac_lavc='#define CONFIG_LIBFDK_AAC 1'
+  libavencoders="$libavencoders LIBFDK_AAC_ENCODER"
+else
+  def_libfdk_aac_lavc='#define CONFIG_LIBFDK_AAC 0'
+fi
+echores "$_libfdk_aac_lavc"
 
 echocheck "FAAD2 support"
 if test "$_faad" = auto ; then
@@ -9237,6 +9254,7 @@ $def_windows_h
 /* external libraries */
 $def_bzlib
 $def_crystalhd
+$def_libfdk_aac_lavc
 $def_libgsm
 $def_libopencore_amrnb
 $def_libopencore_amrwb
-- 
2.20.1

