From 880e89d4bfdfd26c171e47b77eb69173f84df17b Mon Sep 17 00:00:00 2001
From: Gianluigi Tiesi <sherpya@netfarm.it>
Date: Sun, 21 Apr 2019 20:55:21 +0200
Subject: [PATCH 19/30] configure: fixes parsing of gcc versions with dash

---
 configure | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/configure b/configure
index db0dedf4b..b726a2040 100755
--- a/configure
+++ b/configure
@@ -1693,9 +1693,9 @@ else
           cc_fail=yes
           ;;
         *)
-          _cc_major=$(echo $cc_version | cut -d '.' -f 1)
-          _cc_minor=$(echo $cc_version | cut -d '.' -f 2)
-          _cc_mini=$(echo $cc_version | cut -d '.' -f 3)
+          _cc_major=$(echo $cc_version | cut -d '.' -f 1 | cut -d '-' -f 1 )
+          _cc_minor=$(echo $cc_version | cut -d '.' -f 2 | cut -d '-' -f 1)
+          _cc_mini=$(echo $cc_version | cut -d '.' -f 3 | cut -d '-' -f 1)
           ;;
       esac
       echores "$cc_version"
-- 
2.23.0

