From d165d29cc778598ee4a9190266115828edd74f65 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Thu, 14 Feb 2013 04:05:32 +0100
Subject: [PATCH 20/27] configure: added --with-arch and --with-tune overrides

---
 configure | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/configure b/configure
index ed14617..fdbac9a 100755
--- a/configure
+++ b/configure
@@ -547,6 +547,8 @@ Available values are: all $msg_lang_all
 Miscellaneous options:
   --enable-runtime-cpudetection    enable runtime CPU detection [disable]
   --enable-cross-compile enable cross-compilation [autodetect]
+  --with-arch=CPU        Override -march detected option
+  --with-tune=CPU        Override -mtune detected option
   --cc=COMPILER          C compiler to build MPlayer [gcc]
   --host-cc=COMPILER     C compiler for tools needed while building [gcc]
   --as=ASSEMBLER         assembler to build MPlayer [as]
@@ -945,6 +947,12 @@ for ac_option do
   --target=*)
     _target=$(option_value $ac_option)
     ;;
+  --with-arch=*)
+    _arch=$(option_value $ac_option)
+    ;;
+  --with-tune=*)
+    _tune=$(option_value $ac_option)
+    ;;
   --cc=*)
     _cc=$(option_value $ac_option)
     ;;
@@ -2684,6 +2692,9 @@ EOF
     ;;
 esac # case "$host_arch" in
 
+test -z $_arch || _march="-march=$_arch"
+test -z $_tune || _mcpu="$cpuopt=$_tune"
+
 if test "$_runtime_cpudetection" = yes ; then
   if x86 ; then
     test "$_cmov"     != no && _cmov=yes
-- 
2.1.3

