From 567955e18990afb86a922fd3f7625222a26431d7 Mon Sep 17 00:00:00 2001
From: Gianluigi Tiesi <sherpya@netfarm.it>
Date: Thu, 14 Feb 2013 04:52:06 +0100
Subject: [PATCH 20/28] configure: honor disable_system_conf

except stream_dvb

diff --git a/input/input.c b/input/input.c
index 30636e219..aee14d183 100644
--- a/input/input.c
+++ b/input/input.c
@@ -36,6 +36,7 @@
 #ifdef MP_DEBUG
 #include <assert.h>
 #endif
+#include "mpcommon.h"
 #include "mp_fifo.h"
 #include "osdep/getch2.h"
 #include "osdep/keycodes.h"
@@ -1779,7 +1780,7 @@ mp_input_init(void) {
     }
     // Try global conf dir
     file = MPLAYER_CONFDIR "/input.conf";
-    if(! mp_input_parse_config(file))
+    if(disable_system_conf || !mp_input_parse_config(file))
       mp_msg(MSGT_INPUT,MSGL_V,"Falling back on default (hardcoded) input config\n");
   }
   else
diff --git a/mpcommon.c b/mpcommon.c
index d2e944d1a..3e937118e 100644
--- a/mpcommon.c
+++ b/mpcommon.c
@@ -597,7 +597,7 @@ int common_init(void)
     if (!codecs_file || !parse_codec_cfg(codecs_file)) {
         char *conf_path = get_path("codecs.conf");
         if (!parse_codec_cfg(conf_path)) {
-            if (!parse_codec_cfg(MPLAYER_CONFDIR "/codecs.conf")) {
+            if (disable_system_conf || !parse_codec_cfg(MPLAYER_CONFDIR "/codecs.conf")) {
                 if (!parse_codec_cfg(NULL)) {
                     free(conf_path);
                     return 0;
@@ -627,7 +627,7 @@ int common_init(void)
             char *desc_path = get_path("font/font.desc");
             vo_font = read_font_desc(desc_path, font_factor, verbose>1);
             free(desc_path);
-            if (!vo_font)
+            if (!vo_font && !disable_system_conf)
                 vo_font = read_font_desc(MPLAYER_DATADIR "/font/font.desc", font_factor, verbose>1);
         }
         if (sub_font_name)
diff --git a/mplayer.c b/mplayer.c
index 0d8172607..94dc6ac16 100644
--- a/mplayer.c
+++ b/mplayer.c
@@ -3053,7 +3053,7 @@ int main(int argc, char *argv[])
             if (menu_init(mpctx, menu_cfg))
                 mp_msg(MSGT_CPLAYER, MSGL_V, "Menu initialized: %s\n", menu_cfg);
             else {
-                if (menu_init(mpctx, MPLAYER_CONFDIR "/menu.conf"))
+                if (disable_system_conf || menu_init(mpctx, MPLAYER_CONFDIR "/menu.conf"))
                     mp_msg(MSGT_CPLAYER, MSGL_V,  "Menu initialized: %s\n", MPLAYER_CONFDIR "/menu.conf");
                 else {
                     mp_msg(MSGT_CPLAYER, MSGL_ERR, MSGTR_MenuInitFailed);
diff --git a/sub/font_load_ft.c b/sub/font_load_ft.c
index bd3e1c1de..d4d02e2cb 100644
--- a/sub/font_load_ft.c
+++ b/sub/font_load_ft.c
@@ -902,7 +902,8 @@ static int load_sub_face(const char *name, int face_index, FT_Face *face)
 	err = FT_New_Face(library, font_file, 0, face);
 	free(font_file);
 	if (err) {
-	    err = FT_New_Face(library, MPLAYER_DATADIR "/subfont.ttf", 0, face);
+	    if (!disable_system_conf)
+	        err = FT_New_Face(library, MPLAYER_DATADIR "/subfont.ttf", 0, face);
 	    if (err) {
 	        mp_msg(MSGT_OSD, MSGL_ERR, MSGTR_LIBVO_FONT_LOAD_FT_NewFaceFailed);
 		return -1;
-- 
2.33.0

