From 3e237b95d556cae61894278b669215a55d54a474 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Mon, 17 Jun 2013 19:06:27 +0200
Subject: [PATCH 23/28] subs: fix oom on huge number overlapping lines

---
 sub/subreader.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/sub/subreader.c b/sub/subreader.c
index 2a11931..0acba72 100644
--- a/sub/subreader.c
+++ b/sub/subreader.c
@@ -1674,6 +1674,11 @@ if ((suboverlap_enabled == 2) ||
 	    }
 	}
 
+        /* Avoid n^2 memory use for the "placeholder" data structure
+         * below with subtitles that have a huge number of
+         * consecutive overlapping lines. */
+        lines_to_add = FFMIN(lines_to_add, SUB_MAX_TEXT);
+
 	// we need a structure to keep trace of the screen lines
 	// used by the subs, a 'placeholder'
 	counter = 2 * sub_to_add + 1;  // the maximum number of subs derived
-- 
2.5.0

