From 12374e1d090192915202f8ac930258fdc9d0a80b Mon Sep 17 00:00:00 2001
From: Gianluigi Tiesi <sherpya@netfarm.it>
Date: Tue, 15 Oct 2019 00:42:27 +0200
Subject: [PATCH 23/31] configure: enable support for mbedtls

---
 configure | 22 +++++++++++++++++++++-
 1 file changed, 21 insertions(+), 1 deletion(-)

diff --git a/configure b/configure
index 88741c01a..38eafd6a3 100755
--- a/configure
+++ b/configure
@@ -869,6 +869,7 @@ _sctp=auto
 _gethostbyname2=auto
 _gnutls=auto
 _openssl=no
+_mbedtls=no
 _ftp=auto
 _musepack=no
 _vstream=auto
@@ -1377,6 +1378,8 @@ for ac_option do
   --enable-gnutls)       _gnutls=yes    ;;
   --disable-gnutls)      _gnutls=no     ;;
   --enable-openssl-nondistributable) _openssl=yes ;;
+  --enable-mbedtls)     _mbedtls=yes   ;;
+  --disable-mbedtls)    _mbedtls=no    ;;
 
   --enable-fribidi)     _fribidi=yes    ;;
   --disable-fribidi)    _fribidi=no     ;;
@@ -1652,7 +1655,7 @@ libavencoders=$(filter_out_component encoder 'LIB[A-Z0-9_]* [A-Z0-9]*_QSV [A-Z0-
 libavbsfs=$(filter_out_component bsf 'TRACE_HEADERS [A-Z0-9_]*_METADATA H264_REDUNDANT_PPS FILTER_UNITS')
 libavdemuxers=$(filter_out_component demuxer 'AVISYNTH DASH LIB[A-Z0-9_]* REDIR VAPOURSYNTH')
 libavmuxers=$(filter_out_component muxer 'CHROMAPRINT LIB[A-Z0-9_]* RTP RTSP SAP')
-libavprotocols=$(filter_out_component protocol 'BLURAY FFRTMPCRYPT HTTPS LIB[A-Z0-9_]* TLS TLS_GNUTLS TLS_OPENSSL TLS_SECURETRANSPORT TLS_SCHANNEL')
+libavprotocols=$(filter_out_component protocol 'BLURAY FFRTMPCRYPT HTTPS LIB[A-Z0-9_]* TLS TLS_GNUTLS TLS_OPENSSL TLS_MBEDTLS TLS_SECURETRANSPORT TLS_SCHANNEL')
 libavfilters=$(filter_out_component filter 'VF_FREI0R[A-Z0-9_]* LIB[A-Z0-9_]* MP VF_OCV')
 
 # second pass command line parsing for options needing local FFmpeg checkout
@@ -4601,6 +4604,21 @@ else
 fi
 echores "$_openssl"
 
+echocheck "mbedTLS"
+if test "$_mbedtls" = yes ; then
+  def_mbedtls='#define CONFIG_MBEDTLS 1'
+  libavprotocols="$libavprotocols HTTPS_PROTOCOL"
+  if contains_item "$libavprotocols_all" 'TLS_MBEDTLS_PROTOCOL' ; then
+      libavprotocols="$libavprotocols TLS_MBEDTLS_PROTOCOL"
+  else
+      libavprotocols="$libavprotocols TLS_PROTOCOL"
+  fi
+  extra_ldflags="$extra_ldflags -lmbedtls -lmbedx509 -lmbedcrypto"
+else
+  def_mbedtls='#define CONFIG_MBEDTLS 0'
+fi
+echores "$_mbedtls"
+
 echocheck "Samba support (libsmbclient)"
 if test "$_smb" = yes; then
   extra_ldflags="$extra_ldflags -lsmbclient"
@@ -8914,6 +8932,7 @@ CONFIG_CRYSTALHD= $crystalhd
 CONFIG_ENCODERS = yes
 CONFIG_GNUTLS   = $_gnutls
 CONFIG_OPENSSL  = $_openssl
+CONFIG_MBEDTLS  = $_mbedtls
 CONFIG_GPL      = yes
 CONFIG_ICONV    = $_iconv
 CONFIG_MLIB     = $_mlib
@@ -9463,6 +9482,7 @@ $def_erf
 #define CONFIG_FTRAPV 0
 $def_gnutls
 $def_openssl
+$def_mbedtls
 #define CONFIG_GPL 1
 #define CONFIG_GRAY 0
 #define CONFIG_LIBVORBIS 0
-- 
2.29.2

