From 4c65a915320c433268f9012bc11d9f447bf00f29 Mon Sep 17 00:00:00 2001
From: Gianluigi Tiesi <sherpya@netfarm.it>
Date: Wed, 5 Sep 2018 01:43:05 +0200
Subject: [PATCH 27/28] disable SIGSEGV handler when a debugger is attached


diff --git a/mplayer.c b/mplayer.c
index 94dc6ac16..3cb46d79b 100644
--- a/mplayer.c
+++ b/mplayer.c
@@ -3090,6 +3090,9 @@ int main(int argc, char *argv[])
     // fatal errors:
     signal(SIGBUS, exit_sighandler); // bus error
 #ifndef __WINE__                      // hack: the Wine executable will crash else
+#ifdef _WIN32
+    if (!IsDebuggerPresent())
+#endif
     signal(SIGSEGV, exit_sighandler); // segfault
 #endif
     signal(SIGILL, exit_sighandler); // illegal instruction
-- 
2.33.0

