From 9f47d97a64014a3bedd140701b62e6047fa30197 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Tue, 5 Jul 2016 04:18:34 +0200
Subject: [PATCH 28/29] configure: unbreak build with ffmpeg bs filters

---
 configure | 32 ++++++++++++++++++++++++++++++++
 1 file changed, 32 insertions(+)

diff --git a/configure b/configure
index 1b104890f..e479224c3 100755
--- a/configure
+++ b/configure
@@ -8854,6 +8854,22 @@ CONFIG_VP56DSP = yes
 CONFIG_VP8DSP = yes
 CONFIG_WMA_FREQS = yes
 CONFIG_WMV2DSP = yes
+CONFIG_AAC_ADTSTOASC_BSF = yes
+CONFIG_CHOMP_BSF = yes
+CONFIG_DUMP_EXTRADATA_BSF = yes
+CONFIG_DCA_CORE_BSF = yes
+CONFIG_H264_MP4TOANNEXB_BSF = yes
+CONFIG_HEVC_MP4TOANNEXB_BSF = yes
+CONFIG_IMX_DUMP_HEADER_BSF = yes
+CONFIG_MJPEG2JPEG_BSF = yes
+CONFIG_MJPEGA_DUMP_HEADER_BSF = yes
+CONFIG_MP3_HEADER_DECOMPRESS_BSF = yes
+CONFIG_MPEG4_UNPACK_BFRAMES_BSF = yes
+CONFIG_MOV2TEXTSUB_BSF = yes
+CONFIG_NOISE_BSF = yes
+CONFIG_REMOVE_EXTRADATA_BSF = yes
+CONFIG_TEXT2MOVSUB_BSF = yes
+CONFIG_VP9_SUPERFRAME_BSF = yes
 
 $mak_hardcoded_tables
 $mak_libavcodec_mpegaudio_hp
@@ -9518,6 +9534,22 @@ $def_simd_align_32
 #define CONFIG_VP8DSP 1
 #define CONFIG_WMA_FREQS 1
 #define CONFIG_WMV2DSP 1
+#define CONFIG_AAC_ADTSTOASC_BSF 1
+#define CONFIG_CHOMP_BSF 1
+#define CONFIG_DUMP_EXTRADATA_BSF 1
+#define CONFIG_DCA_CORE_BSF 1
+#define CONFIG_H264_MP4TOANNEXB_BSF 1
+#define CONFIG_HEVC_MP4TOANNEXB_BSF 1
+#define CONFIG_IMX_DUMP_HEADER_BSF 1
+#define CONFIG_MJPEG2JPEG_BSF 1
+#define CONFIG_MJPEGA_DUMP_HEADER_BSF 1
+#define CONFIG_MP3_HEADER_DECOMPRESS_BSF 1
+#define CONFIG_MPEG4_UNPACK_BFRAMES_BSF 1
+#define CONFIG_MOV2TEXTSUB_BSF 1
+#define CONFIG_NOISE_BSF 1
+#define CONFIG_REMOVE_EXTRADATA_BSF 1
+#define CONFIG_TEXT2MOVSUB_BSF 1
+#define CONFIG_VP9_SUPERFRAME_BSF 1
 
 /* Use these registers in FFmpeg x86 inline asm. No proper detection yet. */
 #ifndef MP_DEBUG
-- 
2.15.1

