From 9f9dd9b38332b751ca579429b31606ba89857bed Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Sun, 18 Feb 2018 02:49:23 +0100
Subject: [PATCH 28/29] configure: fix build with newer ffmpeg

---
 configure | 14 +++++++++-----
 1 file changed, 9 insertions(+), 5 deletions(-)

diff --git a/configure b/configure
index 1b104890f..47487972e 100755
--- a/configure
+++ b/configure
@@ -1569,9 +1569,9 @@ list_subparts_extern() {
 }
 
 echocheck "ffmpeg/libavcodec/allcodecs.c"
-libavdecoders_all=$(list_subparts  DEC      decoder  codec/allcodecs.c)
-libavencoders_all=$(list_subparts  ENC      encoder  codec/allcodecs.c)
-libavparsers_all=$(list_subparts   PARSER   parser   codec/allcodecs.c)
+libavdecoders_all=$(list_subparts_extern  AVCodec       decoder  codec/allcodecs.c)
+libavencoders_all=$(list_subparts_extern  AVCodec       encoder  codec/allcodecs.c)
+libavparsers_all=$(list_subparts_extern   AVCodecParser parser   codec/parser.c)
 test $? -eq 0 && _list_subparts=found || _list_subparts="not found"
 echores "$_list_subparts"
 
@@ -1582,8 +1582,8 @@ test $? -eq 0 && _list_subparts=found || _list_subparts="not found"
 echores "$_list_subparts"
 
 echocheck "ffmpeg/libavformat/allformats.c"
-libavdemuxers_all=$(list_subparts  DEMUX    demuxer  format/allformats.c)
-libavmuxers_all=$(list_subparts    _MUX     muxer    format/allformats.c)
+libavdemuxers_all=$(list_subparts_extern  AVInputFormat  demuxer  format/allformats.c)
+libavmuxers_all=$(list_subparts_extern    AVOutputFormat muxer    format/allformats.c)
 test $? -eq 0 && _list_subparts=found || _list_subparts="not found"
 echores "$_list_subparts"
 
@@ -9617,8 +9617,12 @@ print_enabled_components(){
     cp $TMPH ffmpeg/$file
 }
 
+print_enabled_components libavcodec/codec_list.c AVCodec codec_list "$libavencoders $libavdecoders"
+print_enabled_components libavcodec/parser_list.c AVCodecParser parser_list $libavparsers
 print_enabled_components libavcodec/bsf_list.c AVBitStreamFilter bitstream_filters $libavbsfs
 print_enabled_components libavformat/protocol_list.c URLProtocol url_protocols $libavprotocols
+print_enabled_components libavformat/muxer_list.c AVOutputFormat muxer_list $libavmuxers
+print_enabled_components libavformat/demuxer_list.c AVInputFormat demuxer_list $libavdemuxers
 
 fi
 
-- 
2.16.1

