From d554b0505b865add4727443a58721ee60461f0c0 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Sun, 31 Aug 2014 05:34:16 +0200
Subject: [PATCH 28/29] configure: probe for dvdnav instead of dvdnavmini

---
 configure | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure b/configure
index 80ab831..b89b28a 100755
--- a/configure
+++ b/configure
@@ -8171,8 +8171,8 @@ if test "$_dvdnav" = yes ; then
     cflags_libdvdnav="-Ilibdvdnav"
     inputmodules="dvdnav(internal) $inputmodules"
   else
-    extra_cflags="$extra_cflags $($_pkg_config --cflags dvdnavmini)"
-    extra_ldflags="$extra_ldflags $($_pkg_config --libs dvdnavmini)"
+    extra_cflags="$extra_cflags $($_pkg_config --cflags dvdnav)"
+    extra_ldflags="$extra_ldflags $($_pkg_config --libs dvdnav)"
     inputmodules="dvdnav $inputmodules"
   fi
 else
-- 
2.1.1

