From 87b79179b92a63a071c92209cb1b7a6e57b24415 Mon Sep 17 00:00:00 2001
From: Gianluigi Tiesi <sherpya@netfarm.it>
Date: Sat, 22 Aug 2015 03:32:39 +0200
Subject: [PATCH 29/31] avformat: call avformat_network_init() as needed

---
 av_helpers.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/av_helpers.c b/av_helpers.c
index 56dc71cf3..05f84351a 100644
--- a/av_helpers.c
+++ b/av_helpers.c
@@ -110,6 +110,7 @@ void init_avformat(void)
         show_av_version(MSGT_DEMUX, "libavformat", LIBAVFORMAT_VERSION_INT,
                         avformat_version(), avformat_configuration());
         av_register_all();
+        avformat_network_init();
         avformat_initialized = 1;
         av_log_set_callback(mp_msp_av_log_callback);
     }
-- 
2.29.2

