From ee22c69de620325e2495aee779ccff7e3d0d0db2 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Sun, 18 May 2014 04:31:04 +0200
Subject: [PATCH 29/30] giflib: compatibilty with 5.1+

---
 libmpdemux/demux_gif.c | 3 +++
 libvo/vo_gif89a.c      | 3 +++
 2 files changed, 6 insertions(+)

diff --git a/libmpdemux/demux_gif.c b/libmpdemux/demux_gif.c
index 33b5188..fd364ad 100644
--- a/libmpdemux/demux_gif.c
+++ b/libmpdemux/demux_gif.c
@@ -50,6 +50,9 @@ typedef struct {
 #define DGifOpenFileHandle(a) DGifOpenFileHandle(a, NULL)
 #define GifError() (gif ? gif->Error : 0)
 #define GifErrorString() GifErrorString(gif->Error)
+#if defined GIFLIB_MINOR && GIFLIB_MINOR >= 1
+#define DGifCloseFile(a) DGifCloseFile(a, NULL)
+#endif
 #endif
 
 /* >= 4.2 prior GIFLIB did not have MAJOR/MINOR defines */
diff --git a/libvo/vo_gif89a.c b/libvo/vo_gif89a.c
index 5061870..e8e024f 100644
--- a/libvo/vo_gif89a.c
+++ b/libvo/vo_gif89a.c
@@ -75,6 +75,9 @@ const LIBVO_EXTERN(gif89a)
 #define MakeMapObject GifMakeMapObject
 #define FreeMapObject GifFreeMapObject
 #define QuantizeBuffer GifQuantizeBuffer
+#if defined GIFLIB_MINOR && GIFLIB_MINOR >= 1
+#define EGifCloseFile(a) EGifCloseFile(a, NULL)
+#endif
 #endif
 
 // how many frames per second we are aiming for during output.
-- 
2.1.0

