diff -ur libvorbis-1.3.5.orig/Makefile.in libvorbis-1.3.5/Makefile.in
--- libvorbis-1.3.5.orig/Makefile.in	2015-03-02 23:27:56.000000000 +0100
+++ libvorbis-1.3.5/Makefile.in	2015-03-25 04:26:32.189157285 +0100
@@ -388,7 +388,7 @@
 
 #AUTOMAKE_OPTIONS = 1.7 foreign dist-zip dist-bzip2
 AUTOMAKE_OPTIONS = foreign 1.11 dist-zip dist-xz
-SUBDIRS = m4 include vq lib test doc $(am__append_1)
+SUBDIRS = include vq lib $(am__append_1)
 m4datadir = $(datadir)/aclocal
 m4data_DATA = vorbis.m4
 pkgconfigdir = $(libdir)/pkgconfig
@@ -879,7 +879,7 @@
 
 info-am:
 
-install-data-am: install-m4dataDATA install-pkgconfigDATA
+install-data-am: install-pkgconfigDATA
 
 install-dvi: install-dvi-recursive
 
