diff -ur libiconv-1.18.orig/srclib/rawmemchr.c libiconv-1.18/srclib/rawmemchr.c
--- libiconv-1.18.orig/srclib/rawmemchr.c	2024-07-12 13:55:16.000000000 +0200
+++ libiconv-1.18/srclib/rawmemchr.c	2025-02-17 19:05:19.637341894 +0100
@@ -26,6 +26,13 @@
 # include <stdint.h>
 
 
+#ifndef UINTPTR_WIDTH
+# define UINTPTR_WIDTH _GL_INTEGER_WIDTH (0, UINTPTR_MAX)
+#endif
+#ifndef UCHAR_WIDTH
+# define UCHAR_WIDTH _GL_INTEGER_WIDTH (0, UCHAR_MAX)
+#endif
+
 /* Find the first occurrence of C in S.  */
 void *
 rawmemchr (const void *s, int c_in)
