diff -u bzip2-1.0.6.orig/bzip2.c bzip2-1.0.6/bzip2.c
--- bzip2-1.0.6.orig/bzip2.c	2010-09-11 01:04:53.000000000 +0200
+++ bzip2-1.0.6/bzip2.c	2013-02-06 04:03:34.782383139 +0100
@@ -128,7 +128,7 @@
 #if BZ_LCCWIN32
 #   include <io.h>
 #   include <fcntl.h>
-#   include <sys\stat.h>
+#   include <sys/stat.h>
 
 #   define NORETURN       /**/
 #   define PATH_SEP       '\\'
diff -u bzip2-1.0.6.orig/bzlib.h bzip2-1.0.6/bzlib.h
--- bzip2-1.0.6.orig/bzlib.h	2010-09-11 01:08:42.000000000 +0200
+++ bzip2-1.0.6/bzlib.h	2013-02-06 04:03:34.782383139 +0100
@@ -82,7 +82,7 @@
 #      undef small
 #   endif
 #   ifdef BZ_EXPORT
-#   define BZ_API(func) WINAPI func
+#   define BZ_API(func) func
 #   define BZ_EXTERN extern
 #   else
    /* import windows dll dynamically */
