diff -ur libopenmpt-0.4.4+release.autotools.orig/Makefile.in libopenmpt-0.4.4+release.autotools/Makefile.in
--- libopenmpt-0.4.4+release.autotools.orig/Makefile.in	2019-04-07 15:30:40.000000000 +0200
+++ libopenmpt-0.4.4+release.autotools/Makefile.in	2019-04-21 16:16:59.437023048 +0200
@@ -6238,8 +6238,7 @@
 
 info-am:
 
-install-data-am: install-dist_docDATA install-man \
-	install-nobase_dist_docDATA install-nobase_includeHEADERS \
+install-data-am: install-nobase_includeHEADERS \
 	install-pkgconfigDATA
 
 install-dvi: install-dvi-am
