diff -ur libopenmpt-0.7.13+release.autotools.orig/Makefile.in libopenmpt-0.7.13+release.autotools/Makefile.in
--- libopenmpt-0.7.13+release.autotools.orig/Makefile.in	2025-01-06 15:54:35.000000000 +0100
+++ libopenmpt-0.7.13+release.autotools/Makefile.in	2025-02-17 19:11:09.212288783 +0100
@@ -7384,8 +7384,7 @@
 
 info-am:
 
-install-data-am: install-dist_docDATA install-includelibopenmptHEADERS \
-	install-man install-nobase_dist_docDATA install-pkgconfigDATA
+install-data-am: install-includelibopenmptHEADERS install-pkgconfigDATA
 
 install-dvi: install-dvi-am
 
