diff -ur libopenmpt-0.6.0+release.autotools.orig/Makefile.in libopenmpt-0.6.0+release.autotools/Makefile.in
--- libopenmpt-0.6.0+release.autotools.orig/Makefile.in	2021-12-23 16:14:26.000000000 +0100
+++ libopenmpt-0.6.0+release.autotools/Makefile.in	2021-12-30 08:55:11.811849341 +0100
@@ -7149,8 +7149,7 @@
 
 info-am:
 
-install-data-am: install-dist_docDATA install-man \
-	install-nobase_dist_docDATA install-nobase_includeHEADERS \
+install-data-am: install-nobase_includeHEADERS \
 	install-pkgconfigDATA
 
 install-dvi: install-dvi-am
