diff -ur libopenmpt-0.5.0+release.autotools.orig/Makefile.in libopenmpt-0.5.0+release.autotools/Makefile.in
--- libopenmpt-0.5.0+release.autotools.orig/Makefile.in	2020-05-24 17:16:41.000000000 +0200
+++ libopenmpt-0.5.0+release.autotools/Makefile.in	2020-06-02 20:44:03.958427612 +0200
@@ -6723,8 +6723,7 @@
 
 info-am:
 
-install-data-am: install-dist_docDATA install-man \
-	install-nobase_dist_docDATA install-nobase_includeHEADERS \
+install-data-am: install-nobase_includeHEADERS \
 	install-pkgconfigDATA
 
 install-dvi: install-dvi-am
