diff -ur libopenmpt-0.5.4+release.autotools.orig/Makefile.in libopenmpt-0.5.4+release.autotools/Makefile.in
--- libopenmpt-0.5.4+release.autotools.orig/Makefile.in	2020-11-29 16:07:24.000000000 +0100
+++ libopenmpt-0.5.4+release.autotools/Makefile.in	2020-12-05 19:29:38.472343621 +0100
@@ -6758,8 +6758,7 @@
 
 info-am:
 
-install-data-am: install-dist_docDATA install-man \
-	install-nobase_dist_docDATA install-nobase_includeHEADERS \
+install-data-am: install-nobase_includeHEADERS \
 	install-pkgconfigDATA
 
 install-dvi: install-dvi-am
