diff -ur libopenmpt-0.7.6+release.autotools.orig/src/mpt/base/detect_quirks.hpp libopenmpt-0.7.6+release.autotools/src/mpt/base/detect_quirks.hpp
--- libopenmpt-0.7.6+release.autotools.orig/src/mpt/base/detect_quirks.hpp	2023-11-15 09:22:56.000000000 +0100
+++ libopenmpt-0.7.6+release.autotools/src/mpt/base/detect_quirks.hpp	2024-03-27 01:59:01.346618246 +0100
@@ -119,7 +119,7 @@
 #endif
 #endif
 #endif
-#if defined(__MINGW32__) && !defined(__MINGW64__) && (MPT_OS_WINDOWS_WIN9X || MPT_OS_WINDOWS_WIN32)
+#if !defined(__MINGW32__) && !defined(__MINGW64__) && (MPT_OS_WINDOWS_WIN9X || MPT_OS_WINDOWS_WIN32)
 #ifndef MPT_COMPILER_QUIRK_NO_WCHAR
 #define MPT_COMPILER_QUIRK_NO_WCHAR
 #endif
