From 88e3ef38fb1a4d8832a33e4de66eefea27d0946e Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Fri, 25 Jan 2013 01:30:57 +0100
Subject: [PATCH 03/23] win32: display played filename in console/video window

---
 libmpcodecs/vf_vo.c |   30 ++++++++++++++++++++++++++++--
 mencoder.c          |    4 ++++
 mplayer.c           |    8 ++++++++
 3 files changed, 40 insertions(+), 2 deletions(-)

diff --git a/libmpcodecs/vf_vo.c b/libmpcodecs/vf_vo.c
index 6511460..522f2ac 100644
--- a/libmpcodecs/vf_vo.c
+++ b/libmpcodecs/vf_vo.c
@@ -30,6 +30,33 @@
 #include "libvo/video_out.h"
 #include "sub/eosd.h"
 
+#if defined(_WIN32) && !defined(CONFIG_GUI)
+#include <windows.h>
+
+extern char *filename;
+static char message[MAX_PATH + 1];
+
+char *get_window_title(void)
+{
+    char *s;
+
+    if (!filename)
+        return "MPlayer";
+
+    s = strrchr(filename, '\\');
+    if (!s) s = strrchr(filename, '/');
+    if (s) s++; else s = filename;
+
+    snprintf(message, PATH_MAX, "MPlayer: playing %s", s);
+    return message;
+}
+#else
+const char *get_windows_title(void)
+{
+    return "MPlayer";
+}
+#endif
+
 //===========================================================================//
 
 struct vf_priv_s {
@@ -70,8 +97,7 @@ static int config(struct vf_instance *vf,
     // save vo's stride capability for the wanted colorspace:
     vf->default_caps=query_format(vf,outfmt);
     vf->draw_slice = (vf->default_caps & VOCAP_NOSLICES) ? NULL : draw_slice;
-
-    if(config_video_out(video_out,width,height,d_width,d_height,flags,"MPlayer",outfmt))
+    if(config_video_out(video_out,width,height,d_width,d_height,flags,get_window_title(),outfmt))
 	return 0;
 
     ++vo_config_count;
diff --git a/mencoder.c b/mencoder.c
index 984e95a..02d02df 100644
--- a/mencoder.c
+++ b/mencoder.c
@@ -113,6 +113,10 @@ float stream_cache_min_percent=20.0;
 float stream_cache_seek_min_percent=50.0;
 #endif
 
+#if defined(_WIN32) && !defined(CONFIG_GUI)
+char *filename = NULL; /* dummy for vf_vo */
+#endif
+
 int audio_id=-1;
 int video_id=-1;
 int dvdsub_id=-1;
diff --git a/mplayer.c b/mplayer.c
index 64bec9b..fa5256b 100644
--- a/mplayer.c
+++ b/mplayer.c
@@ -3679,6 +3679,14 @@ goto_enable_cache:
         if (mpctx->stream->type == STREAMTYPE_DVDNAV)
             mp_input_set_section("dvdnav");
 
+#if defined(_WIN32) && !defined(CONFIG_GUI)
+{
+    extern HWND vo_w32_window; /* NULL for directx, the title is set in config() */
+    extern const char* get_window_title(void);
+    if (filename && vo_w32_window)
+        SetWindowTextA(vo_w32_window, get_window_title());
+}
+#endif
 //==================== START PLAYING =======================
 
         if (mpctx->loop_times > 1)
-- 
1.7.10.4

