From 7cf1c54336e292a1c6ab310419ace7e9c579f493 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Tue, 12 Feb 2013 03:55:52 +0100
Subject: [PATCH 2/4] no Werror on implicit-function-declaration and
 missing-prototypes

---
 configure | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configure b/configure
index b5889ce..6bbce81 100755
--- a/configure
+++ b/configure
@@ -3928,8 +3928,6 @@ elif enabled ccc; then
     add_cflags -msg_disable unsupieee
 elif enabled gcc; then
     check_optflags -fno-tree-vectorize
-    check_cflags -Werror=implicit-function-declaration
-    check_cflags -Werror=missing-prototypes
     check_cflags -Werror=return-type
     check_cflags -Werror=declaration-after-statement
     check_cflags -Werror=vla
-- 
1.8.3.1

