From 5cb612e588573a197c11dfeef67fba48e70d862d Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Wed, 27 Nov 2013 01:46:02 +0100
Subject: [PATCH 2/5] configure: temp hack to workaround new freetype2 headers

---
 configure                 | 2 +-
 libavfilter/vf_drawtext.c | 1 -
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/configure b/configure
index 08d8db3..c816d78 100755
--- a/configure
+++ b/configure
@@ -3889,7 +3889,7 @@ enabled frei0r            && { check_header frei0r.h || die "ERROR: frei0r.h hea
 enabled gnutls            && require_pkg_config gnutls gnutls/gnutls.h gnutls_global_init
 enabled libfaac           && check_header libfaac "stdint.h faac.h"
 enabled libfdk_aac        && require libfdk_aac fdk-aac/aacenc_lib.h aacEncOpen -lfdk-aac
-enabled libfreetype       && require_pkg_config freetype2 "ft2build.h freetype/freetype.h" FT_Init_FreeType
+enabled libfreetype       && require_pkg_config freetype2 "ft2build.h freetype.h" FT_Init_FreeType
 enabled libgsm            && { for gsm_hdr in "gsm.h" "gsm/gsm.h"; do
                                    check_lib "${gsm_hdr}" gsm_create -lgsm && break;
                                done || die "ERROR: libgsm not found"; }
diff --git a/libavfilter/vf_drawtext.c b/libavfilter/vf_drawtext.c
index 703b472..314e0fc 100644
--- a/libavfilter/vf_drawtext.c
+++ b/libavfilter/vf_drawtext.c
@@ -47,7 +47,6 @@
 #include "video.h"
 
 #include <ft2build.h>
-#include <freetype/config/ftheader.h>
 #include FT_FREETYPE_H
 #include FT_GLYPH_H
 
-- 
1.8.5.1

