From afc3edbf9376b5f5861574d55f3e0b73d815f5ce Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Tue, 12 Feb 2013 04:09:04 +0100
Subject: [PATCH 5/5] use upstream version tag, my repo is always rebased

---
 version.sh |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/version.sh b/version.sh
index 6f72b2c..95c2724 100755
--- a/version.sh
+++ b/version.sh
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 # check for git short hash
-revision=$(cd "$1" && git describe --always 2> /dev/null)
+revision=$(cd "$1" && git describe --always upstream/master 2> /dev/null)
 
 # no revision number found
 test "$revision" || revision=$(cd "$1" && cat RELEASE 2> /dev/null)
-- 
1.7.10.4

