diff --git a/Makefile.am b/Makefile.am
index bebf32d..ace04a4 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -17,7 +17,6 @@ libshine_la_LDFLAGS  = -lm -no-undefined -version-info 3:1:0 -export-symbols lib
 libshine_ladir       = ${prefix}/include/shine
 libshine_la_HEADERS  = src/lib/layer3.h
 
-bin_PROGRAMS     = shineenc
 shineenc_SOURCES = src/bin/main.c src/bin/wave.c
 shineenc_LDADD   = libshine.la
 shineenc_CFLAGS  = -Isrc/lib
