diff -ur freetype-2.10.0.orig/builds/unix/ftconfig.in freetype-2.10.0/builds/unix/ftconfig.in
--- freetype-2.10.0.orig/builds/unix/ftconfig.in	2019-02-23 10:06:06.000000000 +0100
+++ freetype-2.10.0/builds/unix/ftconfig.in	2019-04-21 15:29:21.165146226 +0200
@@ -487,7 +487,7 @@
 
 #ifdef FT2_BUILD_LIBRARY
 
-#if defined( _WIN32 ) && defined( DLL_EXPORT )
+#if 0 // defined( _WIN32 ) && defined( DLL_EXPORT )
 #define FT_EXPORT( x )  __declspec( dllexport )  x
 #elif defined( __GNUC__ ) && __GNUC__ >= 4
 #define FT_EXPORT( x )  __attribute__(( visibility( "default" ) ))  x
