diff -ur freetype-2.9.orig/builds/unix/install.mk freetype-2.9/builds/unix/install.mk
--- freetype-2.9.orig/builds/unix/install.mk	2018-01-03 19:00:18.000000000 +0100
+++ freetype-2.9/builds/unix/install.mk	2018-02-03 02:21:01.051724563 +0100
@@ -39,9 +39,7 @@
 	$(MKINSTALLDIRS) $(DESTDIR)$(libdir)                               \
                          $(DESTDIR)$(libdir)/pkgconfig                     \
                          $(DESTDIR)$(includedir)/freetype2/freetype/config \
-                         $(DESTDIR)$(bindir)                               \
-                         $(DESTDIR)$(datadir)/aclocal                      \
-                         $(DESTDIR)$(mandir)/man1
+                         $(DESTDIR)$(bindir)
 	$(LIBTOOL) --mode=install $(INSTALL)                             \
                                   $(PROJECT_LIBRARY) $(DESTDIR)$(libdir)
 	-for P in $(PUBLIC_H) ; do                           \
@@ -62,12 +60,8 @@
           $(DESTDIR)$(includedir)/freetype2/freetype/config/ftoption.h
 	$(INSTALL_SCRIPT) -m 755 $(OBJ_BUILD)/freetype-config \
           $(DESTDIR)$(bindir)/freetype-config
-	$(INSTALL_SCRIPT) -m 644 $(BUILD_DIR)/freetype2.m4 \
-          $(DESTDIR)$(datadir)/aclocal/freetype2.m4
 	$(INSTALL_SCRIPT) -m 644 $(OBJ_BUILD)/freetype2.pc \
           $(DESTDIR)$(libdir)/pkgconfig/freetype2.pc
-	$(INSTALL_DATA) $(TOP_DIR)/docs/freetype-config.1 \
-          $(DESTDIR)$(mandir)/man1/freetype-config.1
 
 
 uninstall:
