diff -ur freetype-2.10.0.orig/modules.cfg freetype-2.10.0/modules.cfg
--- freetype-2.10.0.orig/modules.cfg	2019-02-23 10:06:07.000000000 +0100
+++ freetype-2.10.0/modules.cfg	2019-04-21 15:33:22.875838022 +0200
@@ -111,7 +111,7 @@
 
 # TrueType GX/AAT table validation.  Needs `ftgxval.c' below.
 #
-# AUX_MODULES += gxvalid
+AUX_MODULES += gxvalid
 
 # Support for streams compressed with gzip (files with suffix .gz).
 #
@@ -130,7 +130,7 @@
 
 # OpenType table validation.  Needs `ftotval.c' below.
 #
-# AUX_MODULES += otvalid
+AUX_MODULES += otvalid
 
 # Auxiliary PostScript driver component to share common code.
 #
