diff -ur freetype-2.12.0.orig/modules.cfg freetype-2.12.0/modules.cfg
--- freetype-2.12.0.orig/modules.cfg	2022-01-27 08:43:19.000000000 +0100
+++ freetype-2.12.0/modules.cfg	2022-04-07 00:52:51.024506257 +0200
@@ -117,7 +117,7 @@
 
 # TrueType GX/AAT table validation.  Needs `ftgxval.c' below.
 #
-# AUX_MODULES += gxvalid
+AUX_MODULES += gxvalid
 
 # Support for streams compressed with gzip (files with suffix .gz).
 #
@@ -136,7 +136,7 @@
 
 # OpenType table validation.  Needs `ftotval.c' below.
 #
-# AUX_MODULES += otvalid
+AUX_MODULES += otvalid
 
 # Auxiliary PostScript driver component to share common code.
 #
