diff -ur freetype-2.11.0.orig/modules.cfg freetype-2.11.0/modules.cfg
--- freetype-2.11.0.orig/modules.cfg	2021-05-19 07:43:30.000000000 +0200
+++ freetype-2.11.0/modules.cfg	2021-10-24 12:21:24.405143103 +0200
@@ -114,7 +114,7 @@
 
 # TrueType GX/AAT table validation.  Needs `ftgxval.c' below.
 #
-# AUX_MODULES += gxvalid
+AUX_MODULES += gxvalid
 
 # Support for streams compressed with gzip (files with suffix .gz).
 #
@@ -133,7 +133,7 @@
 
 # OpenType table validation.  Needs `ftotval.c' below.
 #
-# AUX_MODULES += otvalid
+AUX_MODULES += otvalid
 
 # Auxiliary PostScript driver component to share common code.
 #
