diff -ur freetype-2.6.5.orig/src/truetype/ttgload.c freetype-2.6.5/src/truetype/ttgload.c
--- freetype-2.6.5.orig/src/truetype/ttgload.c	2016-06-25 17:44:05.000000000 +0200
+++ freetype-2.6.5/src/truetype/ttgload.c	2016-08-06 03:02:05.516212138 +0200
@@ -1692,7 +1692,7 @@
 
       /* check whether we already have a composite glyph with this index */
       if ( FT_List_Find( &loader->composites,
-                         (void*)(unsigned long)glyph_index ) )
+                         (void*)(uintptr_t)glyph_index ) )
       {
         FT_TRACE1(( "TT_Load_Composite_Glyph:"
                     " infinite recursion detected\n" ));
@@ -1707,7 +1707,7 @@
       {
         if ( FT_NEW( node ) )
           goto Exit;
-        node->data = (void*)(unsigned long)glyph_index;
+        node->data = (void*)(uintptr_t)glyph_index;
         FT_List_Add( &loader->composites, node );
       }
 
