diff -ur freetype-2.5.1.orig/builds/unix/install.mk freetype-2.5.1/builds/unix/install.mk
--- freetype-2.5.1.orig/builds/unix/install.mk	2013-11-13 09:38:30.170894662 +0100
+++ freetype-2.5.1/builds/unix/install.mk	2013-11-26 00:32:15.848945260 +0100
@@ -32,8 +32,7 @@
 	$(MKINSTALLDIRS) $(DESTDIR)$(libdir)                      \
                          $(DESTDIR)$(libdir)/pkgconfig            \
                          $(DESTDIR)$(includedir)/freetype2/config \
-                         $(DESTDIR)$(bindir)                      \
-                         $(DESTDIR)$(datadir)/aclocal
+                         $(DESTDIR)$(bindir)
 	$(LIBTOOL) --mode=install $(INSTALL)                             \
                                   $(PROJECT_LIBRARY) $(DESTDIR)$(libdir)
 	-for P in $(PUBLIC_H) ; do                  \
@@ -54,8 +53,6 @@
           $(DESTDIR)$(includedir)/freetype2/config/ftmodule.h
 	$(INSTALL_SCRIPT) -m 755 $(OBJ_BUILD)/freetype-config \
           $(DESTDIR)$(bindir)/freetype-config
-	$(INSTALL_SCRIPT) -m 644 $(BUILD_DIR)/freetype2.m4 \
-          $(DESTDIR)$(datadir)/aclocal/freetype2.m4
 	$(INSTALL_SCRIPT) -m 644 $(OBJ_BUILD)/freetype2.pc \
           $(DESTDIR)$(libdir)/pkgconfig/freetype2.pc
 
