diff -ur freetype-2.4.12.orig/modules.cfg freetype-2.4.12/modules.cfg
--- freetype-2.4.12.orig/modules.cfg	2011-11-26 13:19:10.000000000 +0100
+++ freetype-2.4.12/modules.cfg	2013-05-12 00:53:54.930815901 +0200
@@ -110,7 +110,7 @@
 AUX_MODULES += cache
 
 # TrueType GX/AAT table validation.  Needs ftgxval.c below.
-# AUX_MODULES += gxvalid
+AUX_MODULES += gxvalid
 
 # Support for streams compressed with gzip (files with suffix .gz).
 #
@@ -129,7 +129,7 @@
 
 # OpenType table validation.  Needs ftotval.c below.
 #
-# AUX_MODULES += otvalid
+AUX_MODULES += otvalid
 
 # Auxiliary PostScript driver component to share common code.
 #
