--- a/include/oaidl.h	2013-09-14 02:26:30.932195549 +0200
+++ b/include/oaidl.h	2013-09-14 02:26:33.108195601 +0200
@@ -355,6 +355,7 @@
 
 #define FADF_RESERVED (0xf008)
 
+#define _FORCENAMELESSUNION
 
 #if (__STDC__ && !defined(__cplusplus) && !defined(_FORCENAMELESSUNION)) || defined(NONAMELESSUNION) || (defined (_MSC_VER) && !defined(_MSC_EXTENSIONS) && !defined(_FORCENAMELESSUNION))
 #define __VARIANT_NAME_1 n1
