diff -ur fribidi-0.19.7.orig/Makefile.in fribidi-0.19.7/Makefile.in
--- fribidi-0.19.7.orig/Makefile.in	2015-08-04 16:15:25.000000000 +0200
+++ fribidi-0.19.7/Makefile.in	2015-08-22 04:14:25.861547178 +0200
@@ -376,7 +376,7 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 ACLOCAL_AMFLAGS = -I m4
-SUBDIRS = gen.tab charset lib bin doc test
+SUBDIRS = gen.tab charset lib
 EXTRA_DIST = bootstrap ChangeLog.old
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = @PACKAGE@.pc
