diff -ur fribidi-0.19.6.orig/Makefile.in fribidi-0.19.6/Makefile.in
--- fribidi-0.19.6.orig/Makefile.in	2013-12-06 21:56:59.000000000 +0100
+++ fribidi-0.19.6/Makefile.in	2013-12-08 06:32:32.576424558 +0100
@@ -361,7 +361,7 @@
 top_build_prefix = @top_build_prefix@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
-SUBDIRS = gen.tab charset lib bin doc test
+SUBDIRS = gen.tab charset lib
 EXTRA_DIST = bootstrap ChangeLog.old
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = @PACKAGE@.pc
