diff -ur fribidi-1.0.10.orig/Makefile.in fribidi-1.0.10/Makefile.in
--- fribidi-1.0.10.orig/Makefile.in	2020-07-05 21:19:23.000000000 +0200
+++ fribidi-1.0.10/Makefile.in	2020-12-05 19:46:18.593614271 +0100
@@ -376,7 +376,7 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 ACLOCAL_AMFLAGS = -I m4
-SUBDIRS = gen.tab lib bin doc test
+SUBDIRS = gen.tab lib
 EXTRA_DIST = autogen.sh ChangeLog.old \
 	meson.build meson_options.txt bin/meson.build doc/meson.build \
 	gen.tab/meson.build lib/meson.build test/meson.build \
