diff -ur fribidi-1.0.3.orig/Makefile.in fribidi-1.0.3/Makefile.in
--- fribidi-1.0.3.orig/Makefile.in	2018-05-16 19:43:42.000000000 +0200
+++ fribidi-1.0.3/Makefile.in	2018-05-17 00:18:03.618479204 +0200
@@ -375,7 +375,7 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 ACLOCAL_AMFLAGS = -I m4
-SUBDIRS = gen.tab lib bin doc test
+SUBDIRS = gen.tab lib
 EXTRA_DIST = autogen.sh ChangeLog.old \
 	meson.build meson_options.txt bin/meson.build doc/meson.build \
 	gen.tab/meson.build lib/meson.build test/meson.build \
