diff -ur fribidi-1.0.11.orig/lib/fribidi-common.h fribidi-1.0.11/lib/fribidi-common.h
--- fribidi-1.0.11.orig/lib/fribidi-common.h	2021-09-23 23:01:24.000000000 +0200
+++ fribidi-1.0.11/lib/fribidi-common.h	2021-10-24 12:25:17.109237561 +0200
@@ -42,7 +42,7 @@
 
 
 /* FRIBIDI_ENTRY is a macro used to declare library entry points. */
-#ifndef FRIBIDI_LIB_STATIC
+#if 0
 # ifndef FRIBIDI_ENTRY
 #  ifdef _WIN32
 #   ifdef FRIBIDI_BUILD
