diff -ur libcaca-0.99.beta19.orig/Makefile.in libcaca-0.99.beta19/Makefile.in
--- libcaca-0.99.beta19.orig/Makefile.in	2014-05-16 21:15:33.000000000 +0200
+++ libcaca-0.99.beta19/Makefile.in	2014-05-18 03:51:11.679707218 +0200
@@ -409,9 +409,7 @@
 top_build_prefix = @top_build_prefix@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
-SUBDIRS = kernel caca src examples tools \
-          cxx caca-sharp python ruby java \
-          doc test
+SUBDIRS = kernel caca cxx
 
 DIST_SUBDIRS = $(SUBDIRS) win32
 EXTRA_DIST = NOTES COPYING.GPL COPYING.ISC COPYING.LGPL bootstrap build-dos build-kernel build-win32 build-win64 caca-config.in libcaca.spec libcaca.sln
