diff -ur mingw-w64-8.0.0.orig/debian/changelog mingw-w64-8.0.0/debian/changelog
--- mingw-w64-8.0.0.orig/debian/changelog	2020-09-18 18:18:53.000000000 +0200
+++ mingw-w64-8.0.0/debian/changelog	2020-11-24 13:21:51.891215209 +0100
@@ -1,3 +1,9 @@
+mingw-w64 (8.0.0-1.1~sherpya) unstable; urgency=medium
+
+  * Rebuild with -march=i486
+
+ -- Gianluigi Tiesi <sherpya@netfarm.it>  Tue, 24 Nov 2020 13:21:38 +0100
+
 mingw-w64 (8.0.0-1) unstable; urgency=medium
 
   * New upstream release, merging comutil-checkerror.patch and
Only in mingw-w64-8.0.0/debian: changelog.orig
diff -ur mingw-w64-8.0.0.orig/debian/rules mingw-w64-8.0.0/debian/rules
--- mingw-w64-8.0.0.orig/debian/rules	2020-09-12 21:22:02.000000000 +0200
+++ mingw-w64-8.0.0/debian/rules	2020-11-24 13:21:35.447263744 +0100
@@ -102,7 +102,7 @@
 $(target32crt)-configure-stamp: $(target32)~$(target32)~headers-install-stamp autoreconf-stamp
 	mkdir -p $(call buildfolder,$(target32crt)) && \
 	cd $(call buildfolder,$(target32crt)) && \
-	$(call sourcefolder,$(target32crt))/configure --prefix=/usr/$(call ruletarget,$(target32crt)) --host=$(call rulehost,$(target32crt)) --target=$(call ruletarget,$(target32crt)) --enable-lib32 --disable-lib64 CPPFLAGS="$(CPPFLAGS) -I$(top_dir)/debian/tmp/usr/$(call ruletarget,$(target32crt))/include"
+	$(call sourcefolder,$(target32crt))/configure --prefix=/usr/$(call ruletarget,$(target32crt)) --host=$(call rulehost,$(target32crt)) --target=$(call ruletarget,$(target32crt)) --enable-lib32 --disable-lib64 CFLAGS="$(CFLAGS) -mtune=generic -march=i486" CPPFLAGS="$(CPPFLAGS) -I$(top_dir)/debian/tmp/usr/$(call ruletarget,$(target32crt))/include"
 	touch $@
 target64crt := $(target64)~$(target64)~crt
 $(target64crt)-configure-stamp: $(target64)~$(target64)~headers-install-stamp autoreconf-stamp
