diff -ur mingw-w64-10.0.0.orig/debian/changelog mingw-w64-10.0.0/debian/changelog
--- mingw-w64-10.0.0.orig/debian/changelog	2022-12-14 18:32:07.000000000 +0100
+++ mingw-w64-10.0.0/debian/changelog	2022-12-16 23:02:37.763708819 +0100
@@ -1,3 +1,10 @@
+mingw-w64 (10.0.0-3.1~sherpya) unstable; urgency=medium
+
+  * Rebuild with:
+    * --with-default-win32-winnt=0x502
+
+ -- Gianluigi Tiesi <sherpya@netfarm.it>  Fri, 16 Dec 2022 22:47:18 +0100
+
 mingw-w64 (10.0.0-3) unstable; urgency=medium
 
   [ Stephen Kitt ]
diff -ur mingw-w64-10.0.0.orig/debian/rules mingw-w64-10.0.0/debian/rules
--- mingw-w64-10.0.0.orig/debian/rules	2022-12-12 13:32:12.000000000 +0100
+++ mingw-w64-10.0.0/debian/rules	2022-12-16 23:02:31.827699171 +0100
@@ -117,7 +117,8 @@
 	'$(call source_folder,$(*)~headers)/configure' \
 	  --host=$(call rule_host,$(*)~headers) \
 	  --target=$(call rule_target,$(*)~headers) \
-	  --prefix=/usr/$(call rule_target,$(*)~headers)
+	  --prefix=/usr/$(call rule_target,$(*)~headers) \
+	  --with-default-win32-winnt=0x502
 	touch '$(stamp)/$(*)~headers~configure'
 
 # Override CRT configuration to avoid multilib builds
