diff -ur live.orig/config.mingw live/config.mingw
--- live.orig/config.mingw	2013-02-11 20:19:40.000000000 +0100
+++ live/config.mingw	2014-05-11 18:46:41.973475502 +0200
@@ -1,10 +1,10 @@
 COMPILE_OPTS =         $(INCLUDES) -I. -O -DSOCKLEN_T=int -DLOCALE_NOT_USED
 C =                    c
 C_COMPILER =           $(CC)
-C_FLAGS =              $(COMPILE_OPTS) -DUSE_OUR_BZERO=1 -D__MINGW32__
+C_FLAGS =              $(COMPILE_OPTS) -DUSE_OUR_BZERO=1 -D__MINGW32__ $(CFLAGS)
 CPP =                  cpp
 CPLUSPLUS_COMPILER =   $(CXX)
-CPLUSPLUS_FLAGS =      $(COMPILE_OPTS) -D__MINGW32__ -Wall -Wno-deprecated
+CPLUSPLUS_FLAGS =      $(COMPILE_OPTS) -D__MINGW32__ -Wall -Wno-deprecated $(CXXFLAGS)
 OBJ =                  o
 LINK =                 $(CXX) -o 
 LINK_OPTS =            -L.
