diff -ur libpng-1.6.43.orig/Makefile.in libpng-1.6.43/Makefile.in
--- libpng-1.6.43.orig/Makefile.in	2024-02-23 10:51:40.000000000 +0100
+++ libpng-1.6.43/Makefile.in	2024-03-27 02:04:19.866781164 +0100
@@ -2232,7 +2232,7 @@
 
 info-am:
 
-install-data-am: install-man install-nodist_pkgincludeHEADERS \
+install-data-am: install-nodist_pkgincludeHEADERS \
 	install-pkgconfigDATA install-pkgincludeHEADERS
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
@@ -2240,7 +2240,7 @@
 
 install-dvi-am:
 
-install-exec-am: install-binPROGRAMS install-binSCRIPTS \
+install-exec-am: install-binSCRIPTS \
 	install-libLTLIBRARIES
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-exec-hook
