diff -ur libpng-1.6.19.orig/Makefile.in libpng-1.6.19/Makefile.in
--- libpng-1.6.19.orig/Makefile.in	2015-11-12 14:00:23.000000000 +0100
+++ libpng-1.6.19/Makefile.in	2015-11-23 01:11:54.457685276 +0100
@@ -1931,7 +1931,7 @@
 
 info-am:
 
-install-data-am: install-man install-nodist_pkgincludeHEADERS \
+install-data-am: install-nodist_pkgincludeHEADERS \
 	install-pkgconfigDATA install-pkgincludeHEADERS
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
@@ -1939,7 +1939,7 @@
 
 install-dvi-am:
 
-install-exec-am: install-binPROGRAMS install-binSCRIPTS \
+install-exec-am: install-binSCRIPTS \
 	install-libLTLIBRARIES
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-exec-hook
