diff -ur libpng-1.6.46.orig/Makefile.in libpng-1.6.46/Makefile.in
--- libpng-1.6.46.orig/Makefile.in	2025-01-23 18:02:04.000000000 +0100
+++ libpng-1.6.46/Makefile.in	2025-02-17 19:15:19.683902543 +0100
@@ -2217,7 +2217,7 @@
 
 info-am:
 
-install-data-am: install-man install-nodist_pkgincludeHEADERS \
+install-data-am: install-nodist_pkgincludeHEADERS \
 	install-pkgconfigDATA install-pkgincludeHEADERS
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
@@ -2225,7 +2225,7 @@
 
 install-dvi-am:
 
-install-exec-am: install-binPROGRAMS install-binSCRIPTS \
+install-exec-am: install-binSCRIPTS \
 	install-libLTLIBRARIES
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-exec-hook
