diff -ur libpng-1.6.38.orig/Makefile.in libpng-1.6.38/Makefile.in
--- libpng-1.6.38.orig/Makefile.in	2022-09-14 21:42:00.000000000 +0200
+++ libpng-1.6.38/Makefile.in	2022-11-06 02:28:44.447056285 +0100
@@ -2118,7 +2118,7 @@
 
 info-am:
 
-install-data-am: install-man install-nodist_pkgincludeHEADERS \
+install-data-am: install-nodist_pkgincludeHEADERS \
 	install-pkgconfigDATA install-pkgincludeHEADERS
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
@@ -2126,7 +2126,7 @@
 
 install-dvi-am:
 
-install-exec-am: install-binPROGRAMS install-binSCRIPTS \
+install-exec-am: install-binSCRIPTS \
 	install-libLTLIBRARIES
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-exec-hook
