diff -ur libpng-1.6.23.orig/Makefile.in libpng-1.6.23/Makefile.in
--- libpng-1.6.23.orig/Makefile.in	2016-06-09 13:45:06.000000000 +0200
+++ libpng-1.6.23/Makefile.in	2016-07-03 06:50:59.431058808 +0200
@@ -1951,7 +1951,7 @@
 
 info-am:
 
-install-data-am: install-man install-nodist_pkgincludeHEADERS \
+install-data-am: install-nodist_pkgincludeHEADERS \
 	install-pkgconfigDATA install-pkgincludeHEADERS
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
@@ -1959,7 +1959,7 @@
 
 install-dvi-am:
 
-install-exec-am: install-binPROGRAMS install-binSCRIPTS \
+install-exec-am: install-binSCRIPTS \
 	install-libLTLIBRARIES
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-exec-hook
