diff -ur libpng-1.6.17.orig/Makefile.in libpng-1.6.17/Makefile.in
--- libpng-1.6.17.orig/Makefile.in	2015-03-26 14:06:35.000000000 +0100
+++ libpng-1.6.17/Makefile.in	2015-03-26 23:26:23.516374080 +0100
@@ -1993,7 +1993,7 @@
 
 info-am:
 
-install-data-am: install-man install-nodist_pkgincludeHEADERS \
+install-data-am: install-nodist_pkgincludeHEADERS \
 	install-pkgconfigDATA install-pkgincludeHEADERS
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
@@ -2001,7 +2001,7 @@
 
 install-dvi-am:
 
-install-exec-am: install-binPROGRAMS install-binSCRIPTS \
+install-exec-am: install-binSCRIPTS \
 	install-libLTLIBRARIES
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-exec-hook
