diff -ur libpng-1.6.36.orig/Makefile.in libpng-1.6.36/Makefile.in
--- libpng-1.6.36.orig/Makefile.in	2018-12-01 15:36:00.000000000 +0100
+++ libpng-1.6.36/Makefile.in	2018-12-03 03:20:17.269532887 +0100
@@ -2114,7 +2114,7 @@
 
 info-am:
 
-install-data-am: install-man install-nodist_pkgincludeHEADERS \
+install-data-am: install-nodist_pkgincludeHEADERS \
 	install-pkgconfigDATA install-pkgincludeHEADERS
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
@@ -2122,7 +2122,7 @@
 
 install-dvi-am:
 
-install-exec-am: install-binPROGRAMS install-binSCRIPTS \
+install-exec-am: install-binSCRIPTS \
 	install-libLTLIBRARIES
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-exec-hook
