diff -ur libpng-1.6.26.orig/Makefile.in libpng-1.6.26/Makefile.in
--- libpng-1.6.26.orig/Makefile.in	2016-10-20 02:07:33.000000000 +0200
+++ libpng-1.6.26/Makefile.in	2016-12-04 01:46:58.515012565 +0100
@@ -1986,7 +1986,7 @@
 
 info-am:
 
-install-data-am: install-man install-nodist_pkgincludeHEADERS \
+install-data-am: install-nodist_pkgincludeHEADERS \
 	install-pkgconfigDATA install-pkgincludeHEADERS
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
@@ -1994,7 +1994,7 @@
 
 install-dvi-am:
 
-install-exec-am: install-binPROGRAMS install-binSCRIPTS \
+install-exec-am: install-binSCRIPTS \
 	install-libLTLIBRARIES
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-exec-hook
