diff -ur libpng-1.6.10.orig/Makefile.in libpng-1.6.10/Makefile.in
--- libpng-1.6.10.orig/Makefile.in	2014-03-06 19:35:15.000000000 +0100
+++ libpng-1.6.10/Makefile.in	2014-03-09 03:26:39.215056047 +0100
@@ -1981,7 +1981,7 @@
 
 info-am:
 
-install-data-am: install-man install-nodist_pkgincludeHEADERS \
+install-data-am: install-nodist_pkgincludeHEADERS \
 	install-pkgconfigDATA install-pkgincludeHEADERS
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
@@ -1989,7 +1989,7 @@
 
 install-dvi-am:
 
-install-exec-am: install-binPROGRAMS install-binSCRIPTS \
+install-exec-am: install-binSCRIPTS \
 	install-libLTLIBRARIES
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-exec-hook
