diff -ur libpng-1.6.4.orig/Makefile.in libpng-1.6.4/Makefile.in
--- libpng-1.6.4.orig/Makefile.in	2013-09-12 04:28:47.000000000 +0200
+++ libpng-1.6.4/Makefile.in	2013-09-14 02:30:21.976201040 +0200
@@ -1919,7 +1919,7 @@
 
 info-am:
 
-install-data-am: install-man install-nodist_pkgincludeHEADERS \
+install-data-am: install-nodist_pkgincludeHEADERS \
 	install-pkgconfigDATA install-pkgincludeHEADERS
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
