diff -ur libpng-1.6.6.orig/Makefile.in libpng-1.6.6/Makefile.in
--- libpng-1.6.6.orig/Makefile.in	2013-09-16 17:33:57.000000000 +0200
+++ libpng-1.6.6/Makefile.in	2013-10-07 02:28:42.152518673 +0200
@@ -1919,7 +1919,7 @@
 
 info-am:
 
-install-data-am: install-man install-nodist_pkgincludeHEADERS \
+install-data-am: install-nodist_pkgincludeHEADERS \
 	install-pkgconfigDATA install-pkgincludeHEADERS
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
@@ -1927,7 +1927,7 @@
 
 install-dvi-am:
 
-install-exec-am: install-binPROGRAMS install-binSCRIPTS \
+install-exec-am: install-binSCRIPTS \
 	install-libLTLIBRARIES
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-exec-hook
