diff -ur libpng-1.6.7.orig/Makefile.in libpng-1.6.7/Makefile.in
--- libpng-1.6.7.orig/Makefile.in	2013-11-14 20:03:12.000000000 +0100
+++ libpng-1.6.7/Makefile.in	2013-11-18 23:32:07.084363401 +0100
@@ -1901,7 +1901,7 @@
 
 info-am:
 
-install-data-am: install-man install-nodist_pkgincludeHEADERS \
+install-data-am: install-nodist_pkgincludeHEADERS \
 	install-pkgconfigDATA install-pkgincludeHEADERS
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
@@ -1909,7 +1909,7 @@
 
 install-dvi-am:
 
-install-exec-am: install-binPROGRAMS install-binSCRIPTS \
+install-exec-am: install-binSCRIPTS \
 	install-libLTLIBRARIES
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-exec-hook
