diff -ur libpng-1.6.9.orig/Makefile.in libpng-1.6.9/Makefile.in
--- libpng-1.6.9.orig/Makefile.in	2014-02-06 05:29:07.000000000 +0100
+++ libpng-1.6.9/Makefile.in	2014-02-08 00:43:21.382293068 +0100
@@ -1952,7 +1952,7 @@
 
 info-am:
 
-install-data-am: install-man install-nodist_pkgincludeHEADERS \
+install-data-am: install-nodist_pkgincludeHEADERS \
 	install-pkgconfigDATA install-pkgincludeHEADERS
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
@@ -1960,7 +1960,7 @@
 
 install-dvi-am:
 
-install-exec-am: install-binPROGRAMS install-binSCRIPTS \
+install-exec-am: install-binSCRIPTS \
 	install-libLTLIBRARIES
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-exec-hook
