diff -ur libpng-1.5.16.orig/Makefile.in libpng-1.5.16/Makefile.in
--- libpng-1.5.16.orig/Makefile.in	2013-05-23 13:45:30.000000000 +0200
+++ libpng-1.5.16/Makefile.in	2013-05-25 01:18:29.181248850 +0200
@@ -1378,7 +1378,7 @@
 
 info-am:
 
-install-data-am: install-man install-nodist_pkgincludeHEADERS \
+install-data-am: install-nodist_pkgincludeHEADERS \
 	install-pkgconfigDATA install-pkgincludeHEADERS
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
