diff --git a/source/CMakeLists.txt b/source/CMakeLists.txt
index ab5ddfeb7..c069fa48d 100755
--- a/source/CMakeLists.txt
+++ b/source/CMakeLists.txt
@@ -784,7 +784,7 @@ if(X265_LATEST_TAG)
         # blacklist of libraries that should not be in Libs.private
         list(REMOVE_ITEM PLIBLIST "-lc" "-lpthread" "-lmingwex" "-lmingwthrd"
             "-lmingw32" "-lmoldname" "-lmsvcrt" "-ladvapi32" "-lshell32"
-            "-luser32" "-lkernel32")
+            "-luser32" "-lkernel32" "-lgcc", "-lgcc_s")
         string(REPLACE ";" " " PRIVATE_LIBS "${PLIBLIST}")
     else()
         set(PRIVATE_LIBS "")
