diff -r c5015669b6dc source/encoder/frameencoder.h
--- a/source/encoder/frameencoder.h	Thu Jun 05 21:40:38 2014 -0500
+++ b/source/encoder/frameencoder.h	Sun Jun 15 02:00:51 2014 +0200
@@ -29,7 +29,7 @@
 #include "wavefront.h"
 
 #include "TLibCommon/TComPic.h"
-#include "TLibCommon/TComBitstream.h"
+#include "TLibCommon/TComBitStream.h"
 
 #include "TLibEncoder/TEncCu.h"
 #include "TLibEncoder/TEncSearch.h"
