diff -r 65b6d13ab22a source/CMakeLists.txt
--- a/source/CMakeLists.txt	Wed Oct 28 01:00:40 2020 +0530
+++ b/source/CMakeLists.txt	Sat Jul 10 02:08:05 2021 +0200
@@ -704,7 +704,7 @@
         # blacklist of libraries that should not be in Libs.private
         list(REMOVE_ITEM PLIBLIST "-lc" "-lpthread" "-lmingwex" "-lmingwthrd"
             "-lmingw32" "-lmoldname" "-lmsvcrt" "-ladvapi32" "-lshell32"
-            "-luser32" "-lkernel32")
+            "-luser32" "-lkernel32" "-lgcc", "-lgcc_s")
         string(REPLACE ";" " " PRIVATE_LIBS "${PLIBLIST}")
     else()
         set(PRIVATE_LIBS "")
