diff -ur opus-1.5.1.orig/Makefile.in opus-1.5.1/Makefile.in
--- opus-1.5.1.orig/Makefile.in	2024-03-04 18:09:15.000000000 +0100
+++ opus-1.5.1/Makefile.in	2024-03-27 03:05:02.599892051 +0100
@@ -4426,8 +4426,7 @@
 
 info-am:
 
-install-data-am: install-data-local install-m4dataDATA \
-	install-pkgconfigDATA install-pkgincludeHEADERS
+install-data-am: install-pkgconfigDATA install-pkgincludeHEADERS
 
 install-dvi: install-dvi-recursive
 
