diff -ur opus-1.5.2.orig/Makefile.in opus-1.5.2/Makefile.in
--- opus-1.5.2.orig/Makefile.in	2024-04-12 03:52:07.000000000 +0200
+++ opus-1.5.2/Makefile.in	2025-02-17 19:29:10.615774548 +0100
@@ -4429,8 +4429,7 @@
 
 info-am:
 
-install-data-am: install-data-local install-m4dataDATA \
-	install-pkgconfigDATA install-pkgincludeHEADERS
+install-data-am: install-pkgconfigDATA install-pkgincludeHEADERS
 
 install-dvi: install-dvi-recursive
 
