diff -ur opus-1.2.1.orig/Makefile.in opus-1.2.1/Makefile.in
--- opus-1.2.1.orig/Makefile.in	2017-06-26 20:06:49.000000000 +0200
+++ opus-1.2.1/Makefile.in	2017-11-11 03:09:11.995550620 +0100
@@ -3214,8 +3214,7 @@
 
 info-am:
 
-install-data-am: install-data-local install-m4dataDATA \
-	install-pkgconfigDATA install-pkgincludeHEADERS
+install-data-am: install-pkgconfigDATA install-pkgincludeHEADERS
 
 install-dvi: install-dvi-recursive
 
