diff -ur opus-1.1.2.orig/Makefile.in opus-1.1.2/Makefile.in
--- opus-1.1.2.orig/Makefile.in	2016-01-12 19:17:49.000000000 +0100
+++ opus-1.1.2/Makefile.in	2016-02-26 02:22:05.038216518 +0100
@@ -2970,8 +2970,7 @@
 
 info-am:
 
-install-data-am: install-data-local install-m4dataDATA \
-	install-pkgconfigDATA install-pkgincludeHEADERS
+install-data-am: install-pkgconfigDATA install-pkgincludeHEADERS
 
 install-dvi: install-dvi-recursive
 
