diff -ur opus-1.3.1.orig/Makefile.in opus-1.3.1/Makefile.in
--- opus-1.3.1.orig/Makefile.in	2019-04-13 00:03:37.000000000 +0200
+++ opus-1.3.1/Makefile.in	2019-04-21 16:36:04.977467502 +0200
@@ -3286,8 +3286,7 @@
 
 info-am:
 
-install-data-am: install-data-local install-m4dataDATA \
-	install-pkgconfigDATA install-pkgincludeHEADERS
+install-data-am: install-pkgconfigDATA install-pkgincludeHEADERS
 
 install-dvi: install-dvi-recursive
 
