diff -ur opus-1.5.1.orig/include/opus_defines.h opus-1.5.1/include/opus_defines.h
--- opus-1.5.1.orig/include/opus_defines.h	2024-03-03 09:31:07.000000000 +0100
+++ opus-1.5.1/include/opus_defines.h	2024-03-27 03:06:30.723813526 +0100
@@ -62,7 +62,7 @@
 
 /** @cond OPUS_INTERNAL_DOC */
 /**Export control for opus functions */
-
+#define OPUS_EXPORT
 #ifndef OPUS_EXPORT
 # if defined(_WIN32)
 #  if defined(OPUS_BUILD) && defined(DLL_EXPORT)
