diff -ur opus-1.5.1.orig/configure opus-1.5.1/configure
--- opus-1.5.1.orig/configure	2024-03-04 18:09:15.000000000 +0100
+++ opus-1.5.1/configure	2024-03-27 03:07:39.179757214 +0100
@@ -15781,7 +15781,7 @@
 
             { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: yes" >&5
 printf "%s\n" "yes" >&6; }
-            CFLAGS="$CFLAGS -D_FORTIFY_SOURCE=2"
+            #CFLAGS="$CFLAGS -D_FORTIFY_SOURCE=2"
 
 else $as_nop
 
