diff -ur opus-1.1.orig/Makefile.in opus-1.1/Makefile.in
--- opus-1.1.orig/Makefile.in	2013-12-05 00:29:30.000000000 +0100
+++ opus-1.1/Makefile.in	2013-12-05 14:54:37.691716748 +0100
@@ -2670,8 +2670,7 @@
 
 info-am:
 
-install-data-am: install-data-local install-m4dataDATA \
-	install-pkgconfigDATA install-pkgincludeHEADERS
+install-data-am: install-pkgconfigDATA install-pkgincludeHEADERS
 
 install-dvi: install-dvi-recursive
 
