diff -ur opus-1.0.3.orig/Makefile.in opus-1.0.3/Makefile.in
--- opus-1.0.3.orig/Makefile.in	2013-07-11 06:20:59.000000000 +0200
+++ opus-1.0.3/Makefile.in	2013-07-13 02:41:42.882685621 +0200
@@ -2446,8 +2446,7 @@
 
 info-am:
 
-install-data-am: install-data-local install-m4dataDATA \
-	install-pkgconfigDATA install-pkgincludeHEADERS
+install-data-am: install-pkgconfigDATA install-pkgincludeHEADERS
 
 install-dvi: install-dvi-recursive
 
