diff -ur xvidcore.orig/build/generic/Makefile xvidcore/build/generic/Makefile
--- xvidcore.orig/build/generic/Makefile	2017-12-11 21:26:37.000000000 +0100
+++ xvidcore/build/generic/Makefile	2018-03-12 19:39:46.773509944 +0100
@@ -149,6 +149,7 @@
 	@$(INSTALL) -d $(DESTDIR)$(libdir)
 	@echo "  I: $(libdir)/$(STATIC_LIB)"
 	@$(INSTALL) -m 644 $(BUILD_DIR)/$(STATIC_LIB) $(DESTDIR)$(libdir)/$(STATIC_LIB)
+disabled:
 ifeq ($(SHARED_EXTENSION),dll)
 	@echo "  I: $(libdir)/$(SHARED_LIB).a"
 	@$(INSTALL) -m 644 $(BUILD_DIR)/$(SHARED_LIB).a $(DESTDIR)$(libdir)/$(SHARED_LIB).a
