diff -ur xvidcore.orig/build/generic/configure xvidcore/build/generic/configure
--- xvidcore.orig/build/generic/configure	2017-12-11 21:26:37.000000000 +0100
+++ xvidcore/build/generic/configure	2018-03-16 01:57:30.910450337 +0100
@@ -4392,7 +4392,7 @@
      [cC][yY][gG][wW][iI][nN]*|mingw32*|mks*)
 	{ $as_echo "$as_me:${as_lineno-$LINENO}: result: ok" >&5
 $as_echo "ok" >&6; }
-	STATIC_LIB="xvidcore.\$(STATIC_EXTENSION)"
+	STATIC_LIB="libxvidcore.\$(STATIC_EXTENSION)"
 	SHARED_LIB="xvidcore.\$(SHARED_EXTENSION)"
 	SPECIFIC_LDFLAGS="-shared -Wl,--dll,--out-implib,\$@.a libxvidcore.def"
 	SPECIFIC_CFLAGS="-mno-cygwin"
