diff -ur game-music-emu-0.6.3.orig/CMakeLists.txt game-music-emu-0.6.3/CMakeLists.txt
--- game-music-emu-0.6.3.orig/CMakeLists.txt	2020-02-25 02:28:10.000000000 +0100
+++ game-music-emu-0.6.3/CMakeLists.txt	2020-04-19 16:07:14.606702154 +0200
@@ -102,5 +102,5 @@
 add_subdirectory(gme)
 
 # EXCLUDE_FROM_ALL adds build rules but keeps it out of default build
-add_subdirectory(player EXCLUDE_FROM_ALL)
-add_subdirectory(demo EXCLUDE_FROM_ALL)
+#add_subdirectory(player EXCLUDE_FROM_ALL)
+#add_subdirectory(demo EXCLUDE_FROM_ALL)
