diff -ur game-music-emu-0.6.3.orig/gme/CMakeLists.txt game-music-emu-0.6.3/gme/CMakeLists.txt
--- game-music-emu-0.6.3.orig/gme/CMakeLists.txt	2020-02-25 02:28:10.000000000 +0100
+++ game-music-emu-0.6.3/gme/CMakeLists.txt	2020-04-19 16:08:41.314220545 +0200
@@ -164,13 +164,13 @@
 # On some platforms we may need to change headers or whatnot based on whether
 # we're building the library or merely using the library. The following is
 # only defined when building the library to allow us to tell which is which.
-add_definitions(-DBLARGG_BUILD_DLL)
+#add_definitions(-DBLARGG_BUILD_DLL)
 
 # For the gme_types.h
 include_directories(${CMAKE_CURRENT_BINARY_DIR})
 
 # Add library to be compiled.
-add_library(gme ${libgme_SRCS})
+add_library(gme STATIC ${libgme_SRCS})
 
 if(ZLIB_FOUND)
     message(" ** ZLib library located, compressed file formats will be supported")
