diff -ur gmp-6.1.2.orig/Makefile.am gmp-6.1.2/Makefile.am
--- gmp-6.1.2.orig/Makefile.am	2016-12-16 16:45:27.000000000 +0100
+++ gmp-6.1.2/Makefile.am	2019-04-21 17:01:29.104968669 +0200
@@ -110,7 +110,7 @@
 LIBGMPXX_LT_AGE      = 5
 
 
-SUBDIRS = tests mpn mpz mpq mpf printf scanf rand cxx demos tune doc
+SUBDIRS = mpn mpz mpq mpf printf scanf rand cxx tune
 
 EXTRA_DIST = configfsf.guess configfsf.sub .gdbinit INSTALL.autoconf \
 	     COPYING.LESSERv3 COPYINGv2 COPYINGv3
