diff -ur gmp-6.0.0.orig/Makefile.in gmp-6.0.0/Makefile.in
--- gmp-6.0.0.orig/Makefile.in	2014-03-25 15:38:06.000000000 +0100
+++ gmp-6.0.0/Makefile.in	2015-08-22 01:58:54.238653994 +0200
@@ -463,7 +463,7 @@
 LIBGMPXX_LT_CURRENT = 8
 LIBGMPXX_LT_REVISION = 0
 LIBGMPXX_LT_AGE = 4
-SUBDIRS = tests mpn mpz mpq mpf printf scanf rand cxx demos tune doc
+SUBDIRS = mpn mpz mpq mpf printf scanf rand cxx tune
 
 # The "test -f" support for srcdir!=builddir is similar to the automake .c.o
 # etc rules, but with each foo.c explicitly, since $< is not portable
