diff -ur giflib-5.1.2.orig/Makefile.in giflib-5.1.2/Makefile.in
--- giflib-5.1.2.orig/Makefile.in	2016-01-07 13:54:02.000000000 +0100
+++ giflib-5.1.2/Makefile.in	2016-02-26 02:18:42.506601471 +0100
@@ -528,7 +528,7 @@
 top_build_prefix = @top_build_prefix@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
-SUBDIRS = lib util pic $(am__append_1)
+SUBDIRS = lib
 
 # Pacify autoreconf.  We don't actually use this.
 ACLOCAL_AMFLAGS = -I m4
