diff -ur giflib-5.0.6.orig/Makefile.in giflib-5.0.6/Makefile.in
--- giflib-5.0.6.orig/Makefile.in	2014-03-29 10:56:48.000000000 +0100
+++ giflib-5.0.6/Makefile.in	2014-04-13 03:28:13.419163015 +0200
@@ -517,7 +517,7 @@
 top_build_prefix = @top_build_prefix@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
-SUBDIRS = lib util pic $(am__append_1)
+SUBDIRS = lib
 
 # Pacify autoreconf.  We don't actually use this.
 ACLOCAL_AMFLAGS = -I m4
