diff -ur speex-1.2.1.orig/Makefile.in speex-1.2.1/Makefile.in
--- speex-1.2.1.orig/Makefile.in	2022-06-16 19:08:58.000000000 +0200
+++ speex-1.2.1/Makefile.in	2022-11-06 02:47:29.114366877 +0100
@@ -389,7 +389,7 @@
 EXTRA_DIST = Speex.spec Speex.spec.in Speex.kdevelop speex.m4 speex.pc.in README.blackfin README.symbian README.TI-DSP
 
 #Fools KDevelop into including all files
-SUBDIRS = libspeex include doc win32 symbian ti $(am__append_1)
+SUBDIRS = libspeex include
 DIST_SUBDIRS = libspeex include src doc win32 symbian ti
 all: config.h
 	$(MAKE) $(AM_MAKEFLAGS) all-recursive
@@ -861,7 +861,7 @@
 
 info-am:
 
-install-data-am: install-m4dataDATA install-pkgconfigDATA
+install-data-am: install-pkgconfigDATA
 
 install-dvi: install-dvi-recursive
 
