diff -ur speex-1.2.0.orig/Makefile.in speex-1.2.0/Makefile.in
--- speex-1.2.0.orig/Makefile.in	2016-12-08 00:21:03.000000000 +0100
+++ speex-1.2.0/Makefile.in	2017-04-18 18:14:01.217630940 +0200
@@ -388,7 +388,7 @@
 EXTRA_DIST = Speex.spec Speex.spec.in Speex.kdevelop speex.m4 speex.pc.in README.blackfin README.symbian README.TI-DSP
 
 #Fools KDevelop into including all files
-SUBDIRS = libspeex include doc win32 symbian ti $(am__append_1)
+SUBDIRS = libspeex include
 DIST_SUBDIRS = libspeex include src doc win32 symbian ti
 all: config.h
 	$(MAKE) $(AM_MAKEFLAGS) all-recursive
@@ -852,7 +852,7 @@
 
 info-am:
 
-install-data-am: install-m4dataDATA install-pkgconfigDATA
+install-data-am: install-pkgconfigDATA
 
 install-dvi: install-dvi-recursive
 
