diff -ur speex-1.2rc2.orig/Makefile.in speex-1.2rc2/Makefile.in
--- speex-1.2rc2.orig/Makefile.in	2014-12-06 20:58:25.000000000 +0100
+++ speex-1.2rc2/Makefile.in	2015-02-04 04:11:14.725432071 +0100
@@ -374,7 +374,7 @@
 EXTRA_DIST = Speex.spec Speex.spec.in Speex.kdevelop speex.m4 speex.pc.in README.blackfin README.symbian README.TI-DSP
 
 #Fools KDevelop into including all files
-SUBDIRS = libspeex include doc win32 symbian ti $(am__append_1)
+SUBDIRS = libspeex include
 DIST_SUBDIRS = libspeex include src doc win32 symbian ti
 all: config.h
 	$(MAKE) $(AM_MAKEFLAGS) all-recursive
@@ -832,7 +832,7 @@
 
 info-am:
 
-install-data-am: install-m4dataDATA install-pkgconfigDATA
+install-data-am: install-pkgconfigDATA
 
 install-dvi: install-dvi-recursive
 
