From 28ec4a25e96797f136255824f11c3e2f487596f2 Mon Sep 17 00:00:00 2001
From: Ron <ron@debian.org>
Date: Tue, 22 May 2012 06:58:41 +0930
Subject: [PATCH] Check the size of the header packet before we look inside it

---
 libspeex/speex_header.c |   17 +++++++++--------
 1 files changed, 9 insertions(+), 8 deletions(-)

diff --git a/libspeex/speex_header.c b/libspeex/speex_header.c
index b743059..92cc0d4 100644
--- a/libspeex/speex_header.c
+++ b/libspeex/speex_header.c
@@ -147,20 +147,21 @@ EXPORT SpeexHeader *speex_packet_to_header(char *packet, int size)
    int i;
    SpeexHeader *le_header;
    const char *h = "Speex   ";
-   for (i=0;i<8;i++)
-      if (packet[i]!=h[i])
-      {
-         speex_notify("This doesn't look like a Speex file");
-         return NULL;
-      }
-   
+
    /*FIXME: Do we allow larger headers?*/
    if (size < (int)sizeof(SpeexHeader))
    {
       speex_notify("Speex header too small");
       return NULL;
    }
-   
+
+   for (i=0;i<8;i++)
+      if (packet[i]!=h[i])
+      {
+         speex_notify("This doesn't look like a Speex file");
+         return NULL;
+      }
+
    le_header = (SpeexHeader*)speex_alloc(sizeof(SpeexHeader));
    
    SPEEX_COPY(le_header, (SpeexHeader*)packet, 1);
-- 
1.7.2.5

