diff -ur mbedtls-3.0.0.orig/CMakeLists.txt mbedtls-3.0.0/CMakeLists.txt
--- mbedtls-3.0.0.orig/CMakeLists.txt	2021-07-06 14:00:08.000000000 +0200
+++ mbedtls-3.0.0/CMakeLists.txt	2021-07-09 17:24:39.159903594 +0200
@@ -314,21 +314,21 @@
         COMPATIBILITY SameMajorVersion
         VERSION 3.0.0)
 
-install(
-    FILES "${CMAKE_CURRENT_BINARY_DIR}/cmake/MbedTLSConfig.cmake"
-          "${CMAKE_CURRENT_BINARY_DIR}/cmake/MbedTLSConfigVersion.cmake"
-    DESTINATION "cmake")
+#install(
+#    FILES "${CMAKE_CURRENT_BINARY_DIR}/cmake/MbedTLSConfig.cmake"
+#          "${CMAKE_CURRENT_BINARY_DIR}/cmake/MbedTLSConfigVersion.cmake"
+#    DESTINATION "cmake")
 
 export(
     EXPORT MbedTLSTargets
     NAMESPACE MbedTLS::
     FILE "cmake/MbedTLSTargets.cmake")
 
-install(
-    EXPORT MbedTLSTargets
-    NAMESPACE MbedTLS::
-    DESTINATION "cmake"
-    FILE "MbedTLSTargets.cmake")
+#install(
+#    EXPORT MbedTLSTargets
+#    NAMESPACE MbedTLS::
+#    DESTINATION "cmake"
+#    FILE "MbedTLSTargets.cmake")
 
 if(CMAKE_VERSION VERSION_GREATER 3.14)
     # Do not export the package by default
