diff -ur mbedtls-2.28.0.orig/include/mbedtls/platform.h mbedtls-2.28.0/include/mbedtls/platform.h
--- mbedtls-2.28.0.orig/include/mbedtls/platform.h	2021-12-15 14:47:54.000000000 +0100
+++ mbedtls-2.28.0/include/mbedtls/platform.h	2021-12-30 08:36:13.512116236 +0100
@@ -62,7 +62,7 @@
  * implementations of some standard library functions, including snprintf
  * and vsnprintf. This affects MSVC and MinGW builds.
  */
-#if defined(__MINGW32__) || (defined(_MSC_VER) && _MSC_VER <= 1900)
+#if !__USE_MINGW_ANSI_STDIO && (defined(__MINGW32__) || (defined(_MSC_VER) && _MSC_VER <= 1900))
 #define MBEDTLS_PLATFORM_HAS_NON_CONFORMING_SNPRINTF
 #define MBEDTLS_PLATFORM_HAS_NON_CONFORMING_VSNPRINTF
 #endif
