diff -ur openjpeg-2.5.0.orig/src/lib/openjp2/openjpeg.h openjpeg-2.5.0/src/lib/openjp2/openjpeg.h
--- openjpeg-2.5.0.orig/src/lib/openjp2/openjpeg.h	2022-05-13 18:54:29.000000000 +0200
+++ openjpeg-2.5.0/src/lib/openjp2/openjpeg.h	2022-11-06 02:45:34.554578664 +0100
@@ -82,7 +82,7 @@
 #define OPJ_DEPRECATED_STRUCT_MEMBER(memb, msg) memb
 #endif
 
-#if defined(OPJ_STATIC) || !defined(_WIN32)
+#if 1 //defined(OPJ_STATIC) || !defined(_WIN32)
 /* http://gcc.gnu.org/wiki/Visibility */
 #   if !defined(_WIN32) && __GNUC__ >= 4
 #       if defined(OPJ_STATIC) /* static library uses "hidden" */
