diff -ur openjpeg-1.5.2.orig/libopenjpeg/openjpeg.h openjpeg-1.5.2/libopenjpeg/openjpeg.h
--- openjpeg-1.5.2.orig/libopenjpeg/openjpeg.h	2014-03-27 11:58:08.000000000 +0100
+++ openjpeg-1.5.2/libopenjpeg/openjpeg.h	2016-12-04 03:28:50.381865152 +0100
@@ -40,7 +40,7 @@
 ==========================================================
 */
 
-#if defined(OPJ_STATIC) || !defined(_WIN32)
+#if 1 //defined(OPJ_STATIC) || !defined(_WIN32)
 #define OPJ_API
 #define OPJ_CALLCONV
 #else
