diff -ur openjpeg-1.5.1.orig/libopenjpeg/opj_malloc.h openjpeg-1.5.1/libopenjpeg/opj_malloc.h
--- openjpeg-1.5.1.orig/libopenjpeg/opj_malloc.h	2012-09-13 09:58:39.000000000 +0200
+++ openjpeg-1.5.1/libopenjpeg/opj_malloc.h	2013-03-15 03:09:33.184407565 +0100
@@ -72,8 +72,13 @@
 #ifdef _WIN32
 	/* Someone should tell the mingw people that their malloc.h ought to provide _mm_malloc() */
 	#ifdef __GNUC__
-		#include <mm_malloc.h>
-		#define HAVE_MM_MALLOC
+		#ifdef _WIN64
+			#include <mm_malloc.h>
+			#define HAVE_MM_MALLOC
+		#else /* msvcrt6 doesn't provides aligned_malloc/free */
+			#include <malloc.h>
+			#define HAVE_MINGW_ALIGNED_MALLOC
+		#endif
 	#else /* MSVC, Intel C++ */
 		#include <malloc.h>
 		#ifdef _mm_malloc
@@ -102,6 +107,13 @@
 	#define opj_aligned_free(m) _mm_free(m)
 #endif
 
+#ifdef HAVE_MINGW_ALIGNED_MALLOC
+	#undef opj_aligned_malloc
+	#define opj_aligned_malloc(size) __mingw_aligned_malloc(size, 16)
+	#undef opj_aligned_free
+	#define opj_aligned_free(m) __mingw_aligned_free(m)
+#endif
+
 #ifdef HAVE_MEMALIGN
 	extern void* memalign(size_t, size_t);
 	#undef opj_aligned_malloc
