diff -ur libxml2-2.9.7.orig/Makefile.in libxml2-2.9.7/Makefile.in
--- libxml2-2.9.7.orig/Makefile.in	2017-11-02 21:16:49.000000000 +0100
+++ libxml2-2.9.7/Makefile.in	2017-11-11 02:28:23.564194423 +0100
@@ -690,7 +690,7 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 ACLOCAL_AMFLAGS = -I m4
-SUBDIRS = include . doc example xstc $(PYTHON_SUBDIR)
+SUBDIRS = include .
 DIST_SUBDIRS = include . doc example python xstc
 AM_CPPFLAGS = -I$(top_builddir)/include -I$(srcdir)/include
 AM_CFLAGS = $(THREAD_CFLAGS) $(Z_CFLAGS) $(LZMA_CFLAGS)
@@ -1800,15 +1800,13 @@
 
 info-am:
 
-install-data-am: install-cmakeDATA install-data-local \
-	install-m4dataDATA install-man install-pkgconfigDATA
+install-data-am: install-pkgconfigDATA
 
 install-dvi: install-dvi-recursive
 
 install-dvi-am:
 
-install-exec-am: install-binPROGRAMS install-binSCRIPTS \
-	install-confexecDATA install-libLTLIBRARIES
+install-exec-am: install-libLTLIBRARIES
 
 install-html: install-html-recursive
 
