diff -ur libxml2-2.9.1.orig/Makefile.in libxml2-2.9.1/Makefile.in
--- libxml2-2.9.1.orig/Makefile.in	2013-04-19 09:36:12.000000000 +0200
+++ libxml2-2.9.1/Makefile.in	2014-06-09 06:45:50.128715748 +0200
@@ -155,7 +155,7 @@
 testdso_la_LINK = $(LIBTOOL) $(AM_V_lt) --tag=CC $(AM_LIBTOOLFLAGS) \
 	$(LIBTOOLFLAGS) --mode=link $(CCLD) $(AM_CFLAGS) $(CFLAGS) \
 	$(testdso_la_LDFLAGS) $(LDFLAGS) -o $@
-PROGRAMS = $(bin_PROGRAMS) $(noinst_PROGRAMS)
+PROGRAMS = 
 am_runsuite_OBJECTS = runsuite.$(OBJEXT)
 runsuite_OBJECTS = $(am_runsuite_OBJECTS)
 am__DEPENDENCIES_2 = $(am__DEPENDENCIES_1) $(top_builddir)/libxml2.la \
@@ -611,7 +611,7 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 ACLOCAL_AMFLAGS = -I m4
-SUBDIRS = include . doc example xstc $(PYTHON_SUBDIR)
+SUBDIRS = include .
 DIST_SUBDIRS = include . doc example python xstc
 AM_CPPFLAGS = -I$(top_builddir)/include -I$(srcdir)/include
 AM_CFLAGS = $(THREAD_CFLAGS) $(Z_CFLAGS) $(LZMA_CFLAGS)
@@ -1703,15 +1703,13 @@
 
 info-am:
 
-install-data-am: install-data-local install-m4dataDATA install-man \
-	install-pkgconfigDATA
+install-data-am: install-pkgconfigDATA
 
 install-dvi: install-dvi-recursive
 
 install-dvi-am:
 
-install-exec-am: install-binPROGRAMS install-binSCRIPTS \
-	install-confexecDATA install-libLTLIBRARIES
+install-exec-am: install-libLTLIBRARIES
 
 install-html: install-html-recursive
 
