diff -ur libxml2-2.9.10.orig/parser.c libxml2-2.9.10/parser.c
--- libxml2-2.9.10.orig/parser.c	2019-10-30 19:52:27.000000000 +0100
+++ libxml2-2.9.10/parser.c	2019-12-08 04:03:39.866988549 +0100
@@ -8160,6 +8160,15 @@
 	    xmlFatalErrMsgStr(ctxt, XML_ERR_UNDECLARED_ENTITY,
 		 "PEReference: %%%s; not found\n", name);
 	} else {
+	    if ((entity->etype == XML_EXTERNAL_PARAMETER_ENTITY) &&
+	        ((ctxt->options & XML_PARSE_NOENT) == 0) &&
+		((ctxt->options & XML_PARSE_DTDVALID) == 0) &&
+		((ctxt->options & XML_PARSE_DTDLOAD) == 0) &&
+		((ctxt->options & XML_PARSE_DTDATTR) == 0) &&
+		(ctxt->replaceEntities == 0) &&
+		(ctxt->validate == 0))
+		return;
+
 	    /*
 	     * [ VC: Entity Declared ]
 	     * In a document with an external subset or external
