diff -ur libxml2-2.9.12.orig/parser.c libxml2-2.9.12/parser.c
--- libxml2-2.9.12.orig/parser.c	2021-05-13 14:54:16.000000000 +0200
+++ libxml2-2.9.12/parser.c	2021-06-26 05:04:06.676178721 +0200
@@ -8225,6 +8225,15 @@
 	    xmlFatalErrMsgStr(ctxt, XML_ERR_UNDECLARED_ENTITY,
 		 "PEReference: %%%s; not found\n", name);
 	} else {
+	    if ((entity->etype == XML_EXTERNAL_PARAMETER_ENTITY) &&
+	        ((ctxt->options & XML_PARSE_NOENT) == 0) &&
+		((ctxt->options & XML_PARSE_DTDVALID) == 0) &&
+		((ctxt->options & XML_PARSE_DTDLOAD) == 0) &&
+		((ctxt->options & XML_PARSE_DTDATTR) == 0) &&
+		(ctxt->replaceEntities == 0) &&
+		(ctxt->validate == 0))
+		return;
+
 	    /*
 	     * [ VC: Entity Declared ]
 	     * In a document with an external subset or external
