diff -ur libxml2-2.9.8.orig/parser.c libxml2-2.9.8/parser.c
--- libxml2-2.9.8.orig/parser.c	2018-01-25 07:44:40.000000000 +0100
+++ libxml2-2.9.8/parser.c	2018-03-07 04:21:59.760802385 +0100
@@ -8146,6 +8146,15 @@
 	    xmlFatalErrMsgStr(ctxt, XML_ERR_UNDECLARED_ENTITY,
 		 "PEReference: %%%s; not found\n", name);
 	} else {
+	    if ((entity->etype == XML_EXTERNAL_PARAMETER_ENTITY) &&
+	        ((ctxt->options & XML_PARSE_NOENT) == 0) &&
+		((ctxt->options & XML_PARSE_DTDVALID) == 0) &&
+		((ctxt->options & XML_PARSE_DTDLOAD) == 0) &&
+		((ctxt->options & XML_PARSE_DTDATTR) == 0) &&
+		(ctxt->replaceEntities == 0) &&
+		(ctxt->validate == 0))
+		return;
+
 	    /*
 	     * [ VC: Entity Declared ]
 	     * In a document with an external subset or external
