diff -ur libxml2-2.9.4.orig/libxml.h libxml2-2.9.4/libxml.h
--- libxml2-2.9.4.orig/libxml.h	2016-05-23 09:25:25.000000000 +0200
+++ libxml2-2.9.4/libxml.h	2016-07-03 06:53:24.582052212 +0200
@@ -109,7 +109,7 @@
 #endif
 #endif
 #endif
-#if !defined(PIC) && !defined(NOLIBTOOL)
+#if !defined(PIC) && !defined(NOLIBTOOL) && !defined(LIBXML_STATIC)
 #  define LIBXML_STATIC
 #endif
 #endif /* ! __XML_LIBXML_H__ */
