diff -ur libxml2-2.9.3.orig/libxml.h libxml2-2.9.3/libxml.h
--- libxml2-2.9.3.orig/libxml.h	2015-05-06 14:04:36.000000000 +0200
+++ libxml2-2.9.3/libxml.h	2016-03-02 05:26:49.331825400 +0100
@@ -106,7 +106,7 @@
 #endif
 #endif
 #endif
-#if !defined(PIC) && !defined(NOLIBTOOL)
+#if !defined(PIC) && !defined(NOLIBTOOL) && !defined(LIBXML_STATIC)
 #  define LIBXML_STATIC
 #endif
 #endif /* ! __XML_LIBXML_H__ */
