diff -ur wavpack-5.1.0.orig/src/wavpack_local.h wavpack-5.1.0/src/wavpack_local.h
--- wavpack-5.1.0.orig/src/wavpack_local.h	2017-01-10 05:02:45.000000000 +0100
+++ wavpack-5.1.0/src/wavpack_local.h	2017-04-18 18:17:00.815905655 +0200
@@ -25,6 +25,10 @@
                             //  (only works on little-endian machines)
 #endif
 
+#if defined(_WIN32)
+#include <intrin.h>
+#endif
+
 #include <sys/types.h>
 
 // This header file contains all the definitions required by WavPack.
