diff --git a/vp9/encoder/vp9_encoder.c b/vp9/encoder/vp9_encoder.c
index 24040ec50..4ef4bd105 100644
--- a/vp9/encoder/vp9_encoder.c
+++ b/vp9/encoder/vp9_encoder.c
@@ -5848,7 +5848,7 @@ static void encode_frame_to_data_rate(
           compute_psnr_stats(cpi->Source, &coded_frame_buf->buf, cm->bit_depth,
                              cpi->oxcf.input_bit_depth);
       fprintf(cpi->ext_ratectrl.log_file,
-              "ENCODE_FRAME_RESULT gop_index %d psnr %f bits %zu\n",
+              "ENCODE_FRAME_RESULT gop_index %d psnr %f bits %" PRIuPTR "\n",
               cpi->twopass.gf_group.index, psnr.psnr[0], (*size) << 3);
     }
 
