From d5ab58b0e2f1f6b21ed3321b63053050d390effb Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Fri, 8 Mar 2013 17:07:30 +0100
Subject: [PATCH 2/7] avfilters: better behavior of frei0r on win32

---
 libavfilter/vf_frei0r.c |   24 ++++++++++++++++++++++++
 1 file changed, 24 insertions(+)

diff --git a/libavfilter/vf_frei0r.c b/libavfilter/vf_frei0r.c
index 345509b..1a50e1c 100644
--- a/libavfilter/vf_frei0r.c
+++ b/libavfilter/vf_frei0r.c
@@ -42,6 +42,10 @@
 #include "internal.h"
 #include "video.h"
 
+#ifdef _WIN32
+#include <windows.h>
+#endif
+
 typedef f0r_instance_t (*f0r_construct_f)(unsigned int width, unsigned int height);
 typedef void (*f0r_destruct_f)(f0r_instance_t instance);
 typedef void (*f0r_deinit_f)(void);
@@ -276,6 +280,25 @@ static av_cold int frei0r_init(AVFilterContext *ctx,
         if (ret < 0)
             return ret;
     }
+#ifdef _WIN32
+    if (!frei0r->dl_handle) {
+        char *ls, prefix[MAX_PATH + 1];
+
+        if (!GetModuleFileNameA(NULL, prefix, MAX_PATH))
+            return AVERROR(EINVAL);
+        prefix[MAX_PATH] = 0;
+
+        if (!(ls = strrchr(prefix, '\\')))
+            return AVERROR(EINVAL);
+
+        *ls = 0;
+        strncat(prefix, "\\frei0r-1\\", sizeof(prefix) - 1 - strlen(prefix));
+
+        ret = load_path(ctx, &frei0r->dl_handle, prefix, dl_name);
+        if (ret < 0)
+            return ret;
+    }
+#else
     if (!frei0r->dl_handle) {
         ret = load_path(ctx, &frei0r->dl_handle, "/usr/local/lib/frei0r-1/", dl_name);
         if (ret < 0)
@@ -286,6 +309,7 @@ static av_cold int frei0r_init(AVFilterContext *ctx,
         if (ret < 0)
             return ret;
     }
+#endif
     if (!frei0r->dl_handle) {
         av_log(ctx, AV_LOG_ERROR, "Could not find module '%s'\n", dl_name);
         return AVERROR(EINVAL);
-- 
1.7.10.4

