From 92f8f866ecb31a8ad56d7ac97e1b16f950af8a5a Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Sat, 18 May 2013 01:27:58 +0200
Subject: [PATCH 2/8] avfilters: better behavior of frei0r on win32

---
 libavfilter/vf_frei0r.c | 24 ++++++++++++++++++++++++
 1 file changed, 24 insertions(+)

diff --git a/libavfilter/vf_frei0r.c b/libavfilter/vf_frei0r.c
index c5922fd..e569a97 100644
--- a/libavfilter/vf_frei0r.c
+++ b/libavfilter/vf_frei0r.c
@@ -40,6 +40,10 @@
 #include "internal.h"
 #include "video.h"
 
+#ifdef _WIN32
+#include <windows.h>
+#endif
+
 typedef f0r_instance_t (*f0r_construct_f)(unsigned int width, unsigned int height);
 typedef void (*f0r_destruct_f)(f0r_instance_t instance);
 typedef void (*f0r_deinit_f)(void);
@@ -274,6 +278,25 @@ static av_cold int frei0r_init(AVFilterContext *ctx,
         if (ret < 0)
             return ret;
     }
+#ifdef _WIN32
+    if (!s->dl_handle) {
+        char *ls, prefix[MAX_PATH + 1];
+
+        if (!GetModuleFileNameA(NULL, prefix, MAX_PATH))
+            return AVERROR(EINVAL);
+        prefix[MAX_PATH] = 0;
+
+        if (!(ls = strrchr(prefix, '\\')))
+            return AVERROR(EINVAL);
+
+        *ls = 0;
+        strncat(prefix, "\\frei0r-1\\", sizeof(prefix) - 1 - strlen(prefix));
+
+        ret = load_path(ctx, &s->dl_handle, prefix, dl_name);
+        if (ret < 0)
+            return ret;
+    }
+#else
     if (!s->dl_handle) {
         ret = load_path(ctx, &s->dl_handle, "/usr/local/lib/frei0r-1/", dl_name);
         if (ret < 0)
@@ -284,6 +307,7 @@ static av_cold int frei0r_init(AVFilterContext *ctx,
         if (ret < 0)
             return ret;
     }
+#endif
 #if ARCH_X86_64
     if (!s->dl_handle) {
         ret = load_path(ctx, &s->dl_handle, "/usr/local/lib64/frei0r-1/", dl_name);
-- 
1.9.0.rc3

