From 1e50bf428ae53fa63417ccffd9d16fa7740bd216 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Fri, 24 Feb 2012 03:08:08 +0100
Subject: [PATCH 2/5] no Werror on implicit-function-declaration and
 missing-prototypes

---
 configure |    2 --
 1 file changed, 2 deletions(-)

diff --git a/configure b/configure
index 9cf5d7d..6e2eb82 100755
--- a/configure
+++ b/configure
@@ -4114,8 +4114,6 @@ elif enabled ccc; then
     add_cflags -msg_disable unsupieee
 elif enabled gcc; then
     check_optflags -fno-tree-vectorize
-    check_cflags -Werror=implicit-function-declaration
-    check_cflags -Werror=missing-prototypes
     check_cflags -Werror=return-type
     check_cflags -Werror=vla
 elif enabled llvm_gcc; then
-- 
1.7.10.4

