From 07ed028e17c6ea2a6841f94b30395792c108bfa0 Mon Sep 17 00:00:00 2001
From: Gianluigi Tiesi <sherpya@netfarm.it>
Date: Sun, 15 Dec 2019 17:18:25 +0100
Subject: [PATCH 3/9] ffplay: allow selecting sdl2 renderer

by setting SDL_VIDEO_RENDER environment variable

diff --git a/fftools/ffplay.c b/fftools/ffplay.c
index ccea0e4578..665ee2f048 100644
--- a/fftools/ffplay.c
+++ b/fftools/ffplay.c
@@ -3736,6 +3736,7 @@ int main(int argc, char **argv)
 
     if (!display_disable) {
         int flags = SDL_WINDOW_HIDDEN;
+        const char *video_renderer = getenv("SDL_VIDEO_RENDER");
         if (alwaysontop)
 #if SDL_VERSION_ATLEAST(2,0,5)
             flags |= SDL_WINDOW_ALWAYS_ON_TOP;
@@ -3748,6 +3749,8 @@ int main(int argc, char **argv)
             flags |= SDL_WINDOW_RESIZABLE;
         window = SDL_CreateWindow(program_name, SDL_WINDOWPOS_UNDEFINED, SDL_WINDOWPOS_UNDEFINED, default_width, default_height, flags);
         SDL_SetHint(SDL_HINT_RENDER_SCALE_QUALITY, "linear");
+        if (video_renderer)
+            SDL_SetHint(SDL_HINT_RENDER_DRIVER, video_renderer);
         if (window) {
             renderer = SDL_CreateRenderer(window, -1, SDL_RENDERER_ACCELERATED | SDL_RENDERER_PRESENTVSYNC);
             if (!renderer) {
-- 
2.33.0

