From ec6ef6cf27b84a4c24a09a8ded51bba9a23dffb8 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Mon, 18 Feb 2013 05:06:47 +0100
Subject: [PATCH 3/7] programs: print version info and configuration only on
 -version

---
 cmdutils.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/cmdutils.c b/cmdutils.c
index fc4b424..0e3f77e 100644
--- a/cmdutils.c
+++ b/cmdutils.c
@@ -1051,7 +1051,7 @@ static void print_program_info(int flags, int level)
     av_log(NULL, level, "%sbuilt on %s %s with %s\n",
            indent, __DATE__, __TIME__, CC_IDENT);
 
-    av_log(NULL, level, "%sconfiguration: " FFMPEG_CONFIGURATION "\n", indent);
+    av_log(NULL, AV_LOG_VERBOSE, "%sconfiguration: " FFMPEG_CONFIGURATION "\n", indent);
 }
 
 static void print_buildconf(int flags, int level)
@@ -1087,8 +1087,8 @@ void show_banner(int argc, char **argv, const OptionDef *options)
         return;
 
     print_program_info (INDENT|SHOW_COPYRIGHT, AV_LOG_INFO);
-    print_all_libs_info(INDENT|SHOW_CONFIG,  AV_LOG_INFO);
-    print_all_libs_info(INDENT|SHOW_VERSION, AV_LOG_INFO);
+    print_all_libs_info(INDENT|SHOW_CONFIG,  AV_LOG_VERBOSE);
+    print_all_libs_info(INDENT|SHOW_VERSION, AV_LOG_VERBOSE);
 }
 
 int show_version(void *optctx, const char *opt, const char *arg)
-- 
1.8.5.1

