From 199f65f23e553a896f4fab67c2517217be50ddd0 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Mon, 18 Feb 2013 05:06:47 +0100
Subject: [PATCH 4/9] programs: print version info and configuration only on
 -version

---
 cmdutils.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/cmdutils.c b/cmdutils.c
index 67bb66e..e3ea61f 100644
--- a/cmdutils.c
+++ b/cmdutils.c
@@ -1077,7 +1077,7 @@ static void print_program_info(int flags, int level)
     av_log(NULL, level, "%sbuilt on %s %s with %s\n",
            indent, __DATE__, __TIME__, CC_IDENT);
 
-    av_log(NULL, level, "%sconfiguration: " FFMPEG_CONFIGURATION "\n", indent);
+    av_log(NULL, AV_LOG_VERBOSE, "%sconfiguration: " FFMPEG_CONFIGURATION "\n", indent);
 }
 
 static void print_buildconf(int flags, int level)
@@ -1113,8 +1113,8 @@ void show_banner(int argc, char **argv, const OptionDef *options)
         return;
 
     print_program_info (INDENT|SHOW_COPYRIGHT, AV_LOG_INFO);
-    print_all_libs_info(INDENT|SHOW_CONFIG,  AV_LOG_INFO);
-    print_all_libs_info(INDENT|SHOW_VERSION, AV_LOG_INFO);
+    print_all_libs_info(INDENT|SHOW_CONFIG,  AV_LOG_VERBOSE);
+    print_all_libs_info(INDENT|SHOW_VERSION, AV_LOG_VERBOSE);
 }
 
 int show_version(void *optctx, const char *opt, const char *arg)
-- 
2.0.1

