From ca1c74f835fe3f732048350e604f1814f54a9237 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Tue, 15 Jul 2014 07:47:11 +0200
Subject: [PATCH 4/6] no Werror on implicit-function-declaration and
 missing-prototypes

---
 configure | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configure b/configure
index 9dae091033..681903527b 100755
--- a/configure
+++ b/configure
@@ -6410,8 +6410,6 @@ if enabled icc; then
 elif enabled gcc; then
     check_optflags -fno-tree-vectorize
     check_cflags -Werror=format-security
-    check_cflags -Werror=implicit-function-declaration
-    check_cflags -Werror=missing-prototypes
     check_cflags -Werror=return-type
     check_cflags -Werror=vla
     check_cflags -Wformat
-- 
2.15.1

