From 8e75c43d132cf0c804e81d5f592e802870373663 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Fri, 24 Feb 2012 03:08:08 +0100
Subject: [PATCH 5/7] no Werror on implicit-function-declaration and
 missing-prototypes

---
 configure |    2 --
 1 file changed, 2 deletions(-)

diff --git a/configure b/configure
index 12b7e43..af11a88 100755
--- a/configure
+++ b/configure
@@ -4169,8 +4169,6 @@ elif enabled ccc; then
     add_cflags -msg_disable unsupieee
 elif enabled gcc; then
     check_optflags -fno-tree-vectorize
-    check_cflags -Werror=implicit-function-declaration
-    check_cflags -Werror=missing-prototypes
     check_cflags -Werror=return-type
     check_cflags -Werror=vla
 elif enabled llvm_gcc; then
-- 
1.7.10.4

