From 564f4c8c5453ab5c66114c4abd4dc7b0be65abcc Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Fri, 24 Feb 2012 03:08:08 +0100
Subject: [PATCH 5/7] no Werror on implicit-function-declaration and
 missing-prototypes

---
 configure | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configure b/configure
index 6cfe2c4..cfe2517 100755
--- a/configure
+++ b/configure
@@ -4686,8 +4686,6 @@ elif enabled ccc; then
     add_cflags -msg_disable unsupieee
 elif enabled gcc; then
     check_optflags -fno-tree-vectorize
-    check_cflags -Werror=implicit-function-declaration
-    check_cflags -Werror=missing-prototypes
     check_cflags -Werror=return-type
     check_cflags -Werror=vla
     enabled extra_warnings || check_disable_warning -Wno-maybe-uninitialized
-- 
1.8.5.1

