From 5a41cd2c5f7ef4ffd317bd70bbafc1386e305032 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Fri, 24 Feb 2012 03:08:08 +0100
Subject: [PATCH 5/7] no Werror on implicit-function-declaration and
 missing-prototypes

---
 configure | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configure b/configure
index f0ee3fe..b6ff37b 100755
--- a/configure
+++ b/configure
@@ -4570,8 +4570,6 @@ elif enabled ccc; then
     add_cflags -msg_disable unsupieee
 elif enabled gcc; then
     check_optflags -fno-tree-vectorize
-    check_cflags -Werror=implicit-function-declaration
-    check_cflags -Werror=missing-prototypes
     check_cflags -Werror=return-type
     check_cflags -Werror=vla
     enabled extra_warnings || check_disable_warning -Wno-maybe-uninitialized
-- 
1.8.4.4

