From f46c74db615be8b2f5bf6878cdef87cabe51f66d Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Sat, 6 Apr 2013 05:05:22 +0200
Subject: [PATCH 6/8] configure: don't add -fPIC on MinGW-w64

---
 configure |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/configure b/configure
index f8e1813..49d8e5f 100755
--- a/configure
+++ b/configure
@@ -3523,7 +3523,10 @@ esc(){
 
 echo "config:$arch:$subarch:$cpu:$target_os:$(esc $cc_ident):$(esc $FFMPEG_CONFIGURATION)" >config.fate
 
-check_cpp_condition stdlib.h "defined(__PIC__) || defined(__pic__) || defined(PIC)" && enable pic
+# MinGW-w64 always defined __PIC__ and __pic__ but noisy warns when passing -fPIC
+if test $target_os != "mingw32"; then
+    check_cpp_condition stdlib.h "defined(__PIC__) || defined(__pic__) || defined(PIC)" && enable pic
+fi
 
 set_default $PATHS_LIST
 set_default nm
-- 
1.7.10.4

