From 298492ece8bd023571f966b132aaaa46a88b6953 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Tue, 15 Jul 2014 07:47:11 +0200
Subject: [PATCH 6/9] no Werror on implicit-function-declaration and
 missing-prototypes

---
 configure | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configure b/configure
index c2cc412..45b5ec8 100755
--- a/configure
+++ b/configure
@@ -5635,8 +5635,6 @@ elif enabled ccc; then
 elif enabled gcc; then
     check_optflags -fno-tree-vectorize
     check_cflags -Werror=format-security
-    check_cflags -Werror=implicit-function-declaration
-    check_cflags -Werror=missing-prototypes
     check_cflags -Werror=return-type
     check_cflags -Werror=vla
     check_cflags -Wformat
-- 
2.5.0

