From 80ad05281495437012fbe82a928095b052efda29 Mon Sep 17 00:00:00 2001
From: Gianluigi Tiesi <sherpya@netfarm.it>
Date: Mon, 18 Feb 2013 05:06:47 +0100
Subject: [PATCH 6/9] programs: print version info and configuration only on
 -version

---
 fftools/opt_common.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/fftools/opt_common.c b/fftools/opt_common.c
index 8a06df82df..5516efde21 100644
--- a/fftools/opt_common.c
+++ b/fftools/opt_common.c
@@ -205,7 +205,7 @@ static void print_program_info(int flags, int level)
     av_log(NULL, level, "\n");
     av_log(NULL, level, "%sbuilt with %s\n", indent, CC_IDENT);
 
-    av_log(NULL, level, "%sconfiguration: " FFMPEG_CONFIGURATION "\n", indent);
+    av_log(NULL, AV_LOG_VERBOSE, "%sconfiguration: " FFMPEG_CONFIGURATION "\n", indent);
 }
 
 static void print_buildconf(int flags, int level)
@@ -241,8 +241,8 @@ void show_banner(int argc, char **argv, const OptionDef *options)
         return;
 
     print_program_info (INDENT|SHOW_COPYRIGHT, AV_LOG_INFO);
-    print_all_libs_info(INDENT|SHOW_CONFIG,  AV_LOG_INFO);
-    print_all_libs_info(INDENT|SHOW_VERSION, AV_LOG_INFO);
+    print_all_libs_info(INDENT|SHOW_CONFIG,  AV_LOG_VERBOSE);
+    print_all_libs_info(INDENT|SHOW_VERSION, AV_LOG_VERBOSE);
 }
 
 int show_version(void *optctx, const char *opt, const char *arg)
-- 
2.39.0

