From c78e7dce1bb0d601881e1849e553a15180a20bad Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Fri, 24 Feb 2012 03:08:08 +0100
Subject: [PATCH 6/8] no Werror on implicit-function-declaration and
 missing-prototypes

---
 configure | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configure b/configure
index 2932694..7d29a87 100755
--- a/configure
+++ b/configure
@@ -4579,8 +4579,6 @@ elif enabled ccc; then
     add_cflags -msg_disable unsupieee
 elif enabled gcc; then
     check_optflags -fno-tree-vectorize
-    check_cflags -Werror=implicit-function-declaration
-    check_cflags -Werror=missing-prototypes
     check_cflags -Werror=return-type
     check_cflags -Werror=vla
     enabled extra_warnings || check_disable_warning -Wno-maybe-uninitialized
-- 
1.8.4.4

