From 61022b855ee6b820e7820f8032d302d8eb7edc4e Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Sat, 9 Aug 2014 04:25:30 +0200
Subject: [PATCH 6/6] use upstream version tag, my repo is always rebased

---
 ffbuild/version.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ffbuild/version.sh b/ffbuild/version.sh
index edc4dd33c5..334051e8ba 100755
--- a/ffbuild/version.sh
+++ b/ffbuild/version.sh
@@ -5,9 +5,9 @@
 # check for git short hash
 if ! test "$revision"; then
     if (cd "$1" && grep git RELEASE 2> /dev/null >/dev/null) ; then
-        revision=$(cd "$1" && git describe --tags --match N 2> /dev/null)
+        revision=$(cd "$1" && git describe --tags --match N upstream/master 2> /dev/null)
     else
-        revision=$(cd "$1" && git describe --tags --always 2> /dev/null)
+        revision=$(cd "$1" && git describe --tags --always upstream/master 2> /dev/null)
     fi
 fi
 
-- 
2.20.1

