From ef905e80aef1dd4ce4bf554e3590d9eec56a6cb2 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Sun, 9 Feb 2014 07:02:48 +0100
Subject: [PATCH 6/8] removed c99 loop var decl

---
 libavcodec/cinepakenc.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/libavcodec/cinepakenc.c b/libavcodec/cinepakenc.c
index 9f990db..6a7a715 100644
--- a/libavcodec/cinepakenc.c
+++ b/libavcodec/cinepakenc.c
@@ -963,7 +963,8 @@ static int rd_strip(CinepakEncContext *s, int y, int h, int keyframe, AVPicture
     for(v1enough = 0, v1_size = SMALLEST_CODEBOOK; v1_size <= CODEBOOK_MAX && !v1enough; v1_size <<= 2) {
         for(v4enough = 0, v4_size = 0; v4_size <= v1_size && !v4enough; v4_size = v4_size ? v4_size << 2 : v1_size >= SMALLEST_CODEBOOK << 2 ? v1_size >> 2 : SMALLEST_CODEBOOK) {
             //try all modes
-            for(CinepakMode mode = 0; mode < MODE_COUNT; mode++) {
+            CinepakMode mode;
+            for(mode = 0; mode < MODE_COUNT; mode++) {
                 //don't allow MODE_MC in intra frames
                 if(keyframe && mode == MODE_MC)
                     continue;
-- 
1.9.0

