From 832991bcf84c76c534f868ea7432c41ac2327f51 Mon Sep 17 00:00:00 2001
From: Gianluigi Tiesi <sherpya@netfarm.it>
Date: Tue, 15 Jul 2014 07:47:11 +0200
Subject: [PATCH 7/9] no Werror on implicit-function-declaration and
 missing-prototypes

---
 configure | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configure b/configure
index c053bfce80..e82ee51ecd 100755
--- a/configure
+++ b/configure
@@ -7132,8 +7132,6 @@ if enabled icc; then
 elif enabled gcc; then
     check_optflags -fno-tree-vectorize
     check_cflags -Werror=format-security
-    check_cflags -Werror=implicit-function-declaration
-    check_cflags -Werror=missing-prototypes
     check_cflags -Werror=return-type
     check_cflags -Werror=vla
     check_cflags -Wformat
-- 
2.35.1

