From 7e9c901ca3a593e71aaefe822e60cffce70fc069 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Mon, 4 Feb 2013 04:03:37 +0100
Subject: [PATCH 9/9] use upstream version tag, my repo is always rebased

---
 version.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/version.sh b/version.sh
index 92edcb9..15729fd 100755
--- a/version.sh
+++ b/version.sh
@@ -4,7 +4,7 @@
 
 # check for git short hash
 if ! test "$revision"; then
-    revision=$(cd "$1" && git describe --tags --match N 2> /dev/null)
+    revision=$(cd "$1" && git describe --tags --match N upstream/master 2> /dev/null)
 fi
 
 # Shallow Git clones (--depth) do not have the N tag:
-- 
2.0.0.rc2

