From 0f15f38154baf93c2f7bbb1c0d963f99a03087f0 Mon Sep 17 00:00:00 2001
From: sherpya <sherpya@netfarm.it>
Date: Sat, 9 Aug 2014 04:25:30 +0200
Subject: [PATCH 9/9] use upstream version tag, my repo is always rebased

---
 version.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/version.sh b/version.sh
index f9754eb..f724110 100755
--- a/version.sh
+++ b/version.sh
@@ -5,9 +5,9 @@
 # check for git short hash
 if ! test "$revision"; then
     if (cd "$1" && grep git RELEASE 2> /dev/null >/dev/null) ; then
-        revision=$(cd "$1" && git describe --tags --match N 2> /dev/null)
+        revision=$(cd "$1" && git describe --tags --match N upstream/master 2> /dev/null)
     else
-        revision=$(cd "$1" && git describe --tags --always 2> /dev/null)
+        revision=$(cd "$1" && git describe --tags --always upstream/master 2> /dev/null)
     fi
 fi
 
-- 
2.5.0

