From: Anand Kumria <wildfire@progsoc.org> 
Subject: Add C++ guard to gsm.h (Closes: #202712)

diff -ur gsm-1.0-pl19.orig/inc/gsm.h gsm-1.0-pl19/inc/gsm.h
--- gsm-1.0-pl19.orig/inc/gsm.h	2020-03-27 22:07:05.000000000 +0100
+++ gsm-1.0-pl19/inc/gsm.h	2022-02-28 03:05:47.416384880 +0100
@@ -58,6 +58,10 @@
 #define	GSM_OPT_FRAME_INDEX	5
 #define	GSM_OPT_FRAME_CHAIN	6
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
 extern gsm  gsm_create 	GSM_P((void));
 extern void gsm_destroy GSM_P((gsm));	
 
@@ -70,6 +74,10 @@
 extern int  gsm_explode GSM_P((gsm, gsm_byte   *, gsm_signal *));
 extern void gsm_implode GSM_P((gsm, gsm_signal *, gsm_byte   *));
 
+#ifdef __cplusplus
+}
+#endif
+
 #undef	GSM_P
 
 #ifdef __cplusplus
