diff -ur gsm-1.0-pl13.orig/Makefile gsm-1.0-pl13/Makefile
--- gsm-1.0-pl13.orig/Makefile	2006-04-26 21:14:26.000000000 +0200
+++ gsm-1.0-pl13/Makefile	2013-02-28 04:47:15.617680664 +0100
@@ -43,8 +43,8 @@
 # CC		= /usr/lang/acc
 # CCFLAGS 	= -c -O
 
-CC		= gcc -ansi -pedantic
-CCFLAGS 	= -c -O2 -DNeedFunctionPrototypes=1
+CC		?= gcc -ansi -pedantic
+CCFLAGS 	:= -c -O2 -DNeedFunctionPrototypes=1 $(CFLAGS)
 
 LD 		= $(CC)
 
@@ -98,14 +98,14 @@
 SHELL		= /bin/sh
 LN		= ln
 BASENAME 	= basename
-AR		= ar
+AR		?= ar
 ARFLAGS		= cr
-RMFLAGS		=
+RMFLAGS		= -f
 FIND		= find
 COMPRESS 	= compress
 COMPRESSFLAGS 	= 
 # RANLIB 	= true
-RANLIB	 	= ranlib
+RANLIB	 	?= ranlib
 
 #
 #    You shouldn't have to configure below this line if you're porting.
