diff -ur libwebp-1.0.2.orig/Makefile.in libwebp-1.0.2/Makefile.in
--- libwebp-1.0.2.orig/Makefile.in	2019-01-15 05:47:26.000000000 +0100
+++ libwebp-1.0.2/Makefile.in	2019-04-21 16:15:05.677837409 +0200
@@ -348,7 +348,7 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 ACLOCAL_AMFLAGS = -I m4
-SUBDIRS = src imageio man $(am__append_1) examples
+SUBDIRS = src
 EXTRA_DIST = COPYING autogen.sh
 all: all-recursive
 
