diff -ur libwebp-1.2.0.orig/Makefile.in libwebp-1.2.0/Makefile.in
--- libwebp-1.2.0.orig/Makefile.in	2021-01-21 04:44:54.000000000 +0100
+++ libwebp-1.2.0/Makefile.in	2021-06-26 05:19:55.646388765 +0200
@@ -350,7 +350,7 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 ACLOCAL_AMFLAGS = -I m4
-SUBDIRS = src imageio man $(am__append_1) examples
+SUBDIRS = src
 EXTRA_DIST = COPYING autogen.sh
 all: all-recursive
 
