diff -ur libwebp-0.6.1.orig/Makefile.in libwebp-0.6.1/Makefile.in
--- libwebp-0.6.1.orig/Makefile.in	2017-11-28 21:15:28.000000000 +0100
+++ libwebp-0.6.1/Makefile.in	2018-03-15 23:53:49.821987441 +0100
@@ -336,7 +336,7 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 ACLOCAL_AMFLAGS = -I m4
-SUBDIRS = src imageio man $(am__append_1) examples
+SUBDIRS = src
 EXTRA_DIST = COPYING autogen.sh
 all: all-recursive
 
