diff -ur libwebp-0.4.0.orig/Makefile.in libwebp-0.4.0/Makefile.in
--- libwebp-0.4.0.orig/Makefile.in	2013-12-21 04:41:00.000000000 +0100
+++ libwebp-0.4.0/Makefile.in	2014-06-09 05:56:08.028811983 +0200
@@ -254,7 +254,7 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 ACLOCAL_AMFLAGS = -I m4
-SUBDIRS = src examples man
+SUBDIRS = src
 EXTRA_DIST = COPYING autogen.sh
 all: config.h
 	$(MAKE) $(AM_MAKEFLAGS) all-recursive
