diff -ur libwebp-1.0.0.orig/Makefile.in libwebp-1.0.0/Makefile.in
--- libwebp-1.0.0.orig/Makefile.in	2018-04-21 05:08:12.000000000 +0200
+++ libwebp-1.0.0/Makefile.in	2018-05-03 01:23:32.327594316 +0200
@@ -349,7 +349,7 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 ACLOCAL_AMFLAGS = -I m4
-SUBDIRS = src imageio man $(am__append_1) examples
+SUBDIRS = src
 EXTRA_DIST = COPYING autogen.sh
 all: all-recursive
 
