diff -Naur a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	2018-05-29 21:48:59.575706804 +0800
+++ b/CMakeLists.txt	2018-05-29 21:51:31.179700310 +0800
@@ -21,7 +21,7 @@
 
 option(USE_OMP "use parallelization use OMP" ON)
 
-add_definitions(-Wall -Wno-pointer-sign -fPIC -std=gnu99)
+add_definitions(-Wall -Wno-pointer-sign -std=gnu99)
 
 ### ORC is not used in any active code at the moment  ###
 # I tried it with 0.4.14
