diff -ur soxr-0.1.3-Source.orig/CMakeLists.txt soxr-0.1.3-Source/CMakeLists.txt
--- soxr-0.1.3-Source.orig/CMakeLists.txt	2018-02-23 22:39:25.000000000 +0100
+++ soxr-0.1.3-Source/CMakeLists.txt	2018-02-25 05:22:41.506173676 +0100
@@ -266,14 +266,6 @@
   enable_testing ()
 endif ()
 
-install (FILES
-  ${CMAKE_CURRENT_SOURCE_DIR}/README
-  ${CMAKE_CURRENT_SOURCE_DIR}/LICENCE
-  ${CMAKE_CURRENT_SOURCE_DIR}/NEWS
-  DESTINATION ${DOC_INSTALL_DIR})
-
-
-
 # Subdirectories:
 
 include_directories (${PROJECT_SOURCE_DIR}/src)
