diff -ur soxr-0.1.2-Source.orig/CMakeLists.txt soxr-0.1.2-Source/CMakeLists.txt
--- soxr-0.1.2-Source.orig/CMakeLists.txt	2015-08-27 21:37:22.000000000 +0200
+++ soxr-0.1.2-Source/CMakeLists.txt	2015-10-07 03:31:42.131910846 +0200
@@ -219,14 +219,6 @@
   enable_testing ()
 endif ()
 
-install (FILES
-  ${CMAKE_CURRENT_SOURCE_DIR}/README
-  ${CMAKE_CURRENT_SOURCE_DIR}/LICENCE
-  ${CMAKE_CURRENT_SOURCE_DIR}/NEWS
-  DESTINATION ${DOC_INSTALL_DIR})
-
-
-
 # Subdirectories:
 
 include_directories (${PROJECT_SOURCE_DIR}/src)
