diff -ur soxr-0.1.1-Source.orig/CMakeLists.txt soxr-0.1.1-Source/CMakeLists.txt
--- soxr-0.1.1-Source.orig/CMakeLists.txt	2013-03-01 12:03:17.000000000 +0100
+++ soxr-0.1.1-Source/CMakeLists.txt	2013-03-10 02:15:07.649624783 +0100
@@ -218,14 +218,6 @@
   enable_testing ()
 endif ()
 
-install (FILES
-  ${CMAKE_CURRENT_SOURCE_DIR}/README
-  ${CMAKE_CURRENT_SOURCE_DIR}/LICENCE
-  ${CMAKE_CURRENT_SOURCE_DIR}/NEWS
-  DESTINATION ${DOC_INSTALL_DIR})
-
-
-
 # Subdirectories:
 
 include_directories (${PROJECT_SOURCE_DIR}/src)
