diff --git a/CMakeLists.txt b/CMakeLists.txt
index 52a12aea4b..6f32a6ff16 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -330,9 +330,9 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/libilbc.pc
         DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
 install(FILES ilbc.h ${CMAKE_CURRENT_BINARY_DIR}/ilbc_export.h
         DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
-install(FILES CONTRIBUTING.md NEWS.md README.md
-        DESTINATION ${CMAKE_INSTALL_DOCDIR})
-install(TARGETS ilbc ilbc_test
+#install(FILES CONTRIBUTING.md NEWS.md README.md
+#        DESTINATION ${CMAKE_INSTALL_DOCDIR})
+install(TARGETS ilbc
         RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
         LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
         ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
