diff --git a/Makefile.am b/Makefile.am
index 8ead0f9f..2e308bf4 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,10 +1,6 @@
 ACLOCAL_AMFLAGS = -I m4
 
-SUBDIRS = src tests
-
-dist_man1_MANS = doc/kvazaar.1
-
-dist_doc_DATA = LICENSE LICENSE.EXT.greatest CREDITS README.md
+SUBDIRS = src
 
 EXTRA_DIST = \
 	build \
diff --git a/src/Makefile.am b/src/Makefile.am
index abc5ffe0..9e9d4707 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -1,4 +1,3 @@
-bin_PROGRAMS = kvazaar
 lib_LTLIBRARIES = libkvazaar.la
 
 
