diff -ur gavl-1.4.0.orig/configure gavl-1.4.0/configure
--- gavl-1.4.0.orig/configure	2012-06-08 13:49:51.000000000 +0200
+++ gavl-1.4.0/configure	2014-06-07 04:06:11.562978181 +0200
@@ -11188,44 +11188,13 @@
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for CLOCK_MONOTONIC" >&5
 $as_echo_n "checking for CLOCK_MONOTONIC... " >&6; }
 
-if test "$cross_compiling" = yes; then :
-  { { $as_echo "$as_me:${as_lineno-$LINENO}: error: in \`$ac_pwd':" >&5
-$as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
-as_fn_error "cannot run test program while cross compiling
-See \`config.log' for more details." "$LINENO" 5; }
-else
-  cat confdefs.h - <<_ACEOF >conftest.$ac_ext
-/* end confdefs.h.  */
-
-  #include <time.h>
-  #include <errno.h>
-main()
-  {
-  struct timespec tp;
-  if(clock_gettime(CLOCK_MONOTONIC, &tp))
-    return -1;
-  else
-    return 0;
-  }
-
-_ACEOF
-if ac_fn_c_try_run "$LINENO"; then :
-
     # program could be run
     { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
 $as_echo "yes" >&6; }
     HAVE_CLOCK_MONOTONIC="true"
 
-
-else
-  # program could not be run
-    { $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
-$as_echo "no" >&6; }
-
-fi
 rm -f core *.core core.conftest.* gmon.out bb.out conftest$ac_exeext \
   conftest.$ac_objext conftest.beam conftest.$ac_ext
-fi
 
 
 if test "x$HAVE_CLOCK_MONOTONIC" = "xtrue"; then
