diff -ur libdca-0.0.7.orig/libdca/Makefile.am libdca-0.0.7/libdca/Makefile.am
--- libdca-0.0.7.orig/libdca/Makefile.am	2020-03-26 15:08:37.000000000 +0100
+++ libdca-0.0.7/libdca/Makefile.am	2020-04-19 16:20:36.386257687 +0200
@@ -10,9 +10,3 @@
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = libdca.pc libdts.pc
-
-install-exec-hook:
-	$(LN_S)f "$(DESTDIR)$(libdir)/libdca.a" "$(DESTDIR)$(libdir)/libdts.a"
-
-uninstall-hook:
-	unlink "$(DESTDIR)$(libdir)/libdts.a"
