diff -ur expat-2.2.6.orig/lib/Makefile.in expat-2.2.6/lib/Makefile.in
--- expat-2.2.6.orig/lib/Makefile.in	2018-08-15 14:55:52.000000000 +0200
+++ expat-2.2.6/lib/Makefile.in	2018-09-03 03:14:43.059912174 +0200
@@ -701,9 +701,8 @@
 
 info-am:
 
-install-data-am: install-docDATA install-includeHEADERS
+install-data-am: install-includeHEADERS
 	@$(NORMAL_INSTALL)
-	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
 install-dvi: install-dvi-am
 
 install-dvi-am:
