diff -ur expat-2.2.10.orig/lib/Makefile.in expat-2.2.10/lib/Makefile.in
--- expat-2.2.10.orig/lib/Makefile.in	2020-10-03 17:37:06.000000000 +0200
+++ expat-2.2.10/lib/Makefile.in	2020-12-05 19:05:53.251778349 +0100
@@ -725,9 +725,8 @@
 
 info-am:
 
-install-data-am: install-docDATA install-includeHEADERS
+install-data-am: install-includeHEADERS
 	@$(NORMAL_INSTALL)
-	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
 install-dvi: install-dvi-am
 
 install-dvi-am:
