diff -ur expat-2.6.2.orig/lib/Makefile.in expat-2.6.2/lib/Makefile.in
--- expat-2.6.2.orig/lib/Makefile.in	2024-03-13 17:49:07.000000000 +0100
+++ expat-2.6.2/lib/Makefile.in	2024-03-27 01:02:46.072150693 +0100
@@ -798,9 +798,8 @@
 
 info-am:
 
-install-data-am: install-docDATA install-includeHEADERS
+install-data-am: install-includeHEADERS
 	@$(NORMAL_INSTALL)
-	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
 install-dvi: install-dvi-am
 
 install-dvi-am:
