diff -ur expat-2.4.6.orig/lib/Makefile.in expat-2.4.6/lib/Makefile.in
--- expat-2.4.6.orig/lib/Makefile.in	2022-02-20 18:04:17.000000000 +0100
+++ expat-2.4.6/lib/Makefile.in	2022-02-28 02:38:28.756693681 +0100
@@ -764,9 +764,8 @@
 
 info-am:
 
-install-data-am: install-docDATA install-includeHEADERS
+install-data-am: install-includeHEADERS
 	@$(NORMAL_INSTALL)
-	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
 install-dvi: install-dvi-am
 
 install-dvi-am:
