diff -ur expat-2.2.9.orig/lib/Makefile.in expat-2.2.9/lib/Makefile.in
--- expat-2.2.9.orig/lib/Makefile.in	2019-09-26 00:03:39.000000000 +0200
+++ expat-2.2.9/lib/Makefile.in	2019-10-14 19:23:05.897829024 +0200
@@ -718,9 +718,8 @@
 
 info-am:
 
-install-data-am: install-docDATA install-includeHEADERS
+install-data-am: install-includeHEADERS
 	@$(NORMAL_INSTALL)
-	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
 install-dvi: install-dvi-am
 
 install-dvi-am:
