diff -ur expat-2.2.0.orig/lib/xmlparse.c expat-2.2.0/lib/xmlparse.c
--- expat-2.2.0.orig/lib/xmlparse.c	2016-06-21 14:58:38.000000000 +0200
+++ expat-2.2.0/lib/xmlparse.c	2016-07-17 01:27:28.395657099 +0200
@@ -6,6 +6,7 @@
 #include <string.h>                     /* memset(), memcpy() */
 #include <assert.h>
 #include <limits.h>                     /* UINT_MAX */
+#include <stdint.h>
 
 #ifdef WIN32
 #define getpid GetCurrentProcessId
@@ -722,7 +723,7 @@
   /* Process ID is 0 bits entropy if attacker has local access
    * XML_Parser address is few bits of entropy if attacker has local access */
   const unsigned long entropy =
-      gather_time_entropy() ^ getpid() ^ (unsigned long)parser;
+      gather_time_entropy() ^ getpid() ^ (uintptr_t)parser;
 
   /* Factors are 2^31-1 and 2^61-1 (Mersenne primes M31 and M61) */
   if (sizeof(unsigned long) == 4) {
